<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=UTF-8">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Mangal;
	panose-1:2 4 5 3 5 2 3 3 2 2;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Times New Roman",serif;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;
	font-style:italic;}
h5
	{mso-style-link:"Heading 5 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoBodyText, li.MsoBodyText, div.MsoBodyText
	{mso-style-link:"Body Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:6.0pt;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
code
	{font-family:"Courier New";}
pre
	{mso-style-link:"HTML Preformatted Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Courier New";}
tt
	{font-family:"Courier New";}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.UnresolvedMention1
	{mso-style-name:"Unresolved Mention1";
	color:#605E5C;
	background:#E1DFDD;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Times New Roman",serif;
	font-weight:bold;}
span.mw-headline
	{mso-style-name:mw-headline;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-style:italic;}
span.a-declarative
	{mso-style-name:a-declarative;}
span.a-icon-alt
	{mso-style-name:a-icon-alt;}
span.a-size-small
	{mso-style-name:a-size-small;}
p.zghrsritem, li.zghrsritem, div.zghrsritem
	{mso-style-name:zg_hrsr_item;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.zghrsrrank
	{mso-style-name:zg_hrsr_rank;}
span.zghrsrladder
	{mso-style-name:zg_hrsr_ladder;}
span.HTMLPreformattedChar
	{mso-style-name:"HTML Preformatted Char";
	mso-style-link:"HTML Preformatted";
	font-family:"Courier New";}
span.BodyTextChar
	{mso-style-name:"Body Text Char";
	mso-style-link:"Body Text";
	font-family:"Times New Roman",serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;}
span.job-id
	{mso-style-name:job-id;}
span.visually-hidden
	{mso-style-name:visually-hidden;}
span.jobsearch-jobinfoheader-bullet
	{mso-style-name:jobsearch-jobinfoheader-bullet;}
span.icl-u-xs-mr--xs
	{mso-style-name:icl-u-xs-mr--xs;}
span.jobsearch-jobmetadataheader-item
	{mso-style-name:jobsearch-jobmetadataheader-item;}
span.Heading5Char
	{mso-style-name:"Heading 5 Char";
	mso-style-link:"Heading 5";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.topiccitationitalics
	{mso-style-name:topiccitationitalics;}
span.bold
	{mso-style-name:bold;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.st
	{mso-style-name:st;}
span.treelabel
	{mso-style-name:tree_label;}
p.msonormal0, li.msonormal0, div.msonormal0
	{mso-style-name:msonormal;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.token
	{mso-style-name:token;}
span.content-section
	{mso-style-name:content-section;}
span.label
	{mso-style-name:label;}
p.chapter-para, li.chapter-para, div.chapter-para
	{mso-style-name:chapter-para;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.underline
	{mso-style-name:underline;}
span.pre
	{mso-style-name:pre;}
span.caps
	{mso-style-name:caps;}
span.Date1
	{mso-style-name:Date1;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:8.5in 11.0in;
	margin:.3in .3in .3in .5in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:12.0pt;font-family:"Times New Roman",serif'>AWS Bioinformatics</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=787 valign=top style='width:8.2in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=787 valign=top style='width:8.2in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Bioconductor in the
    cloud</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.bioconductor.org/help/bioconductor-cloud-ami/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.bioconductor.org/help/bioconductor-cloud-ami/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Bioconductor
    Community maintained Public AMI image (R Packages, RStudio Server, Git, …) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>RStudio
    Server Amazon Machine Image (AMI)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AMI
    specifically targeted at R and RStudio Server</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.louisaslett.com/RStudio_AMI/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.louisaslett.com/RStudio_AMI/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>To
    use the AMIs described on this page, you simply click your chosen AMI ID </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>which
    will take you through to the Amazon web interface and preselect the correct
    region and AMI. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Simply
    ensure that your ‘security group’ settings allow incoming HTTP (port 80)
    traffic and then </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>copy-and-paste
    the ‘Public DNS’ for your running instance to a web browser address bar to
    bring up the login page.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.louisaslett.com/RStudio_AMI/video_guide.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.louisaslett.com/RStudio_AMI/video_guide.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>US
    East, Virginia     ami-0f2290fdad793f863</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fconsole.aws.amazon.com%2Fec2%2Fhome%3Fregion%3Dus-east-1%26state%3DhashArgs%2523launchAmi%253Dami-0f2290fdad793f863%26isauthcode%3Dtrue&amp;client_id=arn%3Aaws%3Aiam%3A%3A015428540659%3Auser%2Fec2&amp;forceMobileApp=0&amp;code_challenge=bEkiXYttLxSfqW-GZo7rCoYMLO0XpaFOtXitDHZkEwI&amp;code_challenge_method=SHA-256"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://signin.aws.amazon.com/signin?redirect_uri=https%3A%2F%2Fconsole.aws.amazon.com%2Fec2%2Fhome%3Fregion%3Dus-east-1%26state%3DhashArgs%2523launchAmi%253Dami-0f2290fdad793f863%26isauthcode%3Dtrue&amp;client_id=arn%3Aaws%3Aiam%3A%3A015428540659%3Auser%2Fec2&amp;forceMobileApp=0&amp;code_challenge=bEkiXYttLxSfqW-GZo7rCoYMLO0XpaFOtXitDHZkEwI&amp;code_challenge_method=SHA-256</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><b><span style='font-size:10.0pt;color:black'>Includes:</span></b></p>
    <ul type=disc>
     <li class=MsoNormal style='line-height:normal'><span class=caps><span
         style='font-size:9.0pt;color:black'>30GB</span></span><span
         style='font-size:9.0pt;color:black'> <span class=caps>EBS</span>
         storage — compact, but enables storage of more sizeable datasets.</span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Defaults to fast <span class=caps>SSD</span>
          storage (faster, zero <span class=caps>IO</span> costs, only $1 per
          month in most&nbsp;regions)</span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Dropbox integration to up/down-load files and data.</span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Setup can be completed entirely through RStudio in
          the web browser by running a single&nbsp;function.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Selective syncing supported so that large Dropbox
          accounts don’t sync&nbsp;everything.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Unlink and relink to new account&nbsp;supported.</span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Full LaTeX support enabling </span><span
         style='color:black'><a href="http://rmarkdown.rstudio.com/"
         target="_blank"><span style='font-size:9.0pt;color:black;text-decoration:
         none'>R Markdown</span></a></span><span style='font-size:9.0pt;
         color:black'>, Sweave and regular document compiles
         within&nbsp;RStudio.</span></li>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Java 8 JRE enabling full support for </span><span
         style='color:black'><a href="https://www.h2o.ai/products/h2o/"
         target="_blank"><span style='font-size:9.0pt;color:black;text-decoration:
         none'>H2O</span></a></span><span style='font-size:9.0pt;color:black'>
         and&nbsp;</span><span style='color:black'><a
         href="https://spark.rstudio.com/" target="_blank"><span
         style='font-size:9.0pt;color:black;text-decoration:none'>Spark</span></a></span><span
         style='font-size:9.0pt;color:black'>.</span></li>
     <li class=MsoNormal style='line-height:normal'><span class=caps><span
         style='font-size:9.0pt;color:black'>GDAL</span></span><span
         style='font-size:9.0pt;color:black'> dependencies for <span
         class=caps>GIS</span>&nbsp;packages.</span></li>
     <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
         href="http://www.gnu.org/software/gsl/" target="_blank"><span
         class=caps><span style='font-size:9.0pt;color:black;text-decoration:
         none'>GSL</span></span></a></span><span style='font-size:9.0pt;
         color:black'> and <span class=caps>CURL</span>&nbsp;libraries.</span></li>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Database support:</span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span class=caps><span
          style='font-size:9.0pt;color:black'>ODBC</span></span><span
          style='font-size:9.0pt;color:black'> drivers&nbsp;installed.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>RMySQL package precompiled and&nbsp;installed.</span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Git support out of the&nbsp;box.</span></li>
     <li class=MsoNormal style='line-height:normal'><span class=caps><span
         style='font-size:9.0pt;color:black'>MCMC</span></span><span
         style='font-size:9.0pt;color:black'> samplers:</span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="http://mc-stan.org/rstan.html" target="_blank"><span
          style='font-size:9.0pt;color:black;text-decoration:none'>Stan</span></a></span><span
          style='font-size:9.0pt;color:black'> (RStan) installed and ready to
          use for Hamiltonian Monte Carlo&nbsp;sampling.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="http://mcmc-jags.sourceforge.net" target="_blank"><span
          class=caps><span style='font-size:9.0pt;color:black;text-decoration:
          none'>JAGS</span></span></a></span><span style='font-size:9.0pt;
          color:black'> (and rjags) installed and ready to use for
          Gibbs&nbsp;sampling.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="https://greta-stats.org" target="_blank"><span
          style='font-size:9.0pt;color:black;text-decoration:none'>Greta</span></a></span><span
          style='font-size:9.0pt;color:black'> supported for GPU MCMC sampling
          via Tensorflow support.</span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>CUDA and cuDNN. </span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Enables immediate use of GPU instances (e.g. p2.*
          instances) without any setup.</span></li>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="https://www.tensorflow.org/" target="_blank"><span
          style='font-size:9.0pt;color:black;text-decoration:none'>Tensorflow</span></a></span><span
          style='font-size:9.0pt;color:black'> and </span><span
          style='color:black'><a href="https://keras.io/" target="_blank"><span
          style='font-size:9.0pt;color:black;text-decoration:none'>Keras</span></a></span><span
          style='font-size:9.0pt;color:black'> deep learning libraries can be
          accelerated using nVidia GPUs on the GPU compute instances, just 3
          lines of R code each to setup: </span></li>
     </ul>
    </ul>
    <pre style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    # Tensorflow</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    install.packages(&quot;tensorflow&quot;)</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    library(&quot;tensorflow&quot;)</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    install_tensorflow(version = &quot;gpu&quot;)</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt'>&nbsp;</span></pre><pre style='margin-left:1.0in;
    text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    # Keras</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    install.packages(&quot;keras&quot;)</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    library(&quot;keras&quot;)</span></pre><pre
    style='margin-left:1.0in;text-indent:-.25in'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span><span
    style='font-size:9.0pt;color:black'>    install_keras(tensorflow = &quot;gpu&quot;)</span></pre>
    <p class=MsoNormal style='margin-left:1.0in;line-height:normal'><span
    style='font-size:9.0pt;color:black'>You will then be prompted to install
    Miniconda: say &quot;Yes&quot; to this option. <br>
    See the </span><span style='color:black'><a
    href="https://keras.rstudio.com/" target="_blank"><span style='font-size:
    9.0pt'>RStudio Keras page</span></a></span><span style='font-size:9.0pt;
    color:black'> and </span><span style='color:black'><a
    href="https://tensorflow.rstudio.com/" target="_blank"><span
    style='font-size:9.0pt'>RStudio Tensorflow page</span></a></span><span
    style='font-size:9.0pt;color:black'> for details on using from R. </span></p>
    <ul type=disc>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='font-size:
          9.0pt;color:black'>Also preinstalled Magma GPU linear algebra
          libraries for accelerated matrix decompositions.</span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Swap space for compiling of large packages on
         constrained memory instances (such as&nbsp;rugarch).</span></li>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Arbitrary precision arithmetic and number theory
         libraries supported out of the box:</span></li>
     <ul type=circle>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="http://gmplib.org/" target="_blank"><span class=caps><span
          style='font-size:9.0pt;color:black;text-decoration:none'>GMP</span></span></a></span></li>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="http://www.mpfr.org/" target="_blank"><span class=caps><span
          style='font-size:9.0pt;color:black;text-decoration:none'>MPFR</span></span></a></span></li>
      <li class=MsoNormal style='line-height:normal'><span style='color:black'><a
          href="http://flintlib.org/" target="_blank"><span class=caps><span
          style='font-size:9.0pt;color:black;text-decoration:none'>FLINT</span></span></a></span></li>
     </ul>
     <li class=MsoNormal style='line-height:normal'><span style='font-size:
         9.0pt;color:black'>Optimised <span class=caps>BLAS</span> for
         automatically faster matrix operations than base R libraries (</span><span
         style='color:black'><a href="http://www.openblas.net/" target="_blank"><span
         style='font-size:9.0pt;color:black;text-decoration:none'>OpenBLAS</span></a></span><span
         style='font-size:9.0pt;color:black'>).</span></li>
    </ul>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AWS
  EC2 AMI Bioconductor.org Version:</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://www.bioconductor.org/help/bioconductor-cloud-ami/#ami_ids"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.bioconductor.org/help/bioconductor-cloud-ami/#ami_ids</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  Bioconductor.org Docker Image:  RStudio Server, Bioconductor Packages, Git</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  </span><span style='color:black'><a
  href="https://hub.docker.com/r/bioconductor/bioconductor_docker"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://hub.docker.com/r/bioconductor/bioconductor_docker</span></a></span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  Creating RStudio projects from GitHub Repositories</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.youtube.com/watch?v=YxZ8J2rqhEM"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=YxZ8J2rqhEM</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  Git with RStudio R Projects</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://www.infoworld.com/video/97367/r-tip-how-to-use-git-and-github-with-r-projects"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.infoworld.com/video/97367/r-tip-how-to-use-git-and-github-with-r-projects</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA-seq
  workflow: gene-level exploratory analysis and differential expression</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Development
  of a cloud-based Bioinformatics Training Platform</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Briefings
  in Bioinformatics, Volume 18, Issue 3, May 2017, Pages 537–544</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://academic.oup.com/bib/article/18/3/537/2453288"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://academic.oup.com/bib/article/18/3/537/2453288</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Running R on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>23
    JUL 2015</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/running-r-on-aws/"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/running-r-on-aws/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>(October
    2017 Update) <br>
    </span><span style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/author/ajfriedm/"
    target="_blank"><strong><span style='font-size:10.0pt;color:blue'>Aaron
    Friedman</span></strong></a></span><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'> is a Healthcare and Life
    Sciences Partner Solutions Architect with AWS.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/author/ajfriedm/"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/author/ajfriedm/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Choosing an
         Amazon Machine Image</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Choosing an
         instance type</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Configuring
         instance details: EC2 user data</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Configuring
         instance details: IAM roles</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Configuring a
         security group<br>
         <br>
         </span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'>Loading data into your R-based environment on AWS</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Storing data
         in S3</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Configuring
         Shiny Server</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-family:"Times New Roman",serif;color:black'>Automating
         deployment <br>
         <br>
         </span></li>
    </ul>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Development
    of a cloud-based Bioinformatics Training Platform</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Briefings
    in Bioinformatics, Volume 18, Issue 3, May 2017, Pages 537–544</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://academic.oup.com/bib/article/18/3/537/2453288"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://academic.oup.com/bib/article/18/3/537/2453288</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    3 day NGS workshop has been organized into seven training modules: </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>introduction
    to the command line, </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>quality
    control of the NGS data and Alignment, </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>ChIPSeq,
    RNASeq, De novo Assembly and Post Workshop. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>All
    the presentations, tools, data sets and tutorials for each training module
    are accessible and maintained on </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GitHub
    </span><span style='color:black'><a
    href="https://github.com/BPA-CSIRO-Workshops"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://github.com/BPA-CSIRO-Workshops</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>BTP has been given a sufficient allocation for running </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>50 instances with specifications of two CPUs and
    8&#8201;GB of RAM each.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><img border=0 width=416 height=268 id="Picture 14"
    src="AWS-Bioinformatics_files/image001.jpg"></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Specialist bioinformatics software tools for analysis
    of data are essential for any bioinformatics hands-on workshop. These
    software tools are packaged and installed into the BTP image on creation.
    To improve the software tool installation process onto the BTP virtual
    machine images, we have used a tool packaging system. <br>
    <br>
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>Table
    2.<br>
    Current list of analysis tools included and maintained on the
    Bioinformatics Training Platform. These tools are automatically configured
    and installed on the BTP images and instances</span></p>
    <table class=MsoNormalTable border=0 cellpadding=0>
     <thead>
      <tr>
       <td style='padding:.75pt .75pt .75pt .75pt'>
       <p class=MsoNormal><strong><span style='font-size:9.0pt;line-height:
       107%;font-family:"Calibri",sans-serif'>Tool</span></strong></p>
       </td>
       <td style='padding:.75pt .75pt .75pt .75pt'>
       <p class=MsoNormal><strong><span style='font-size:9.0pt;line-height:
       107%;font-family:"Calibri",sans-serif'>Function</span></strong></p>
       </td>
       <td style='padding:.75pt .75pt .75pt .75pt'>
       <p class=MsoNormal><span class=underline><b><span style='font-size:9.0pt;
       line-height:107%'>Link</span></b></span></p>
       </td>
      </tr>
     </thead>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>AMOS
      Hawkeye&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Genome
      data visualization&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://sourceforge.net/projects/amos/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://sourceforge.net/projects/amos/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>BEDTools&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Genome
      data manipulation&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://bedtools.readthedocs.org/en/latest/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://bedtools.readthedocs.org/en/latest/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>BLAT&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Sequence
      location lookup in the genome&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="https://genome.ucsc.edu/FAQ/FAQblat.html"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://genome.ucsc.edu/FAQ/FAQblat.html</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Bowtie&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Read
      Alignment&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://bowtie-bio.sourceforge.net/index.shtml"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://bowtie-bio.sourceforge.net/index.shtml</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>CummeRbund&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>RNA-Seq
      analysis using R&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a
      href="http://bioconductor.org/packages/release/bioc/html/cummeRbund.html"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://bioconductor.org/packages/release/bioc/html/cummeRbund.html</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Cufflinks&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>RNA-Seq
      analysis&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://cole-trapnell-lab.github.io/cufflinks/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://cole-trapnell-lab.github.io/cufflinks/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>DESeq2&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Differential
      gene expression analysis using R&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a
      href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://bioconductor.org/packages/release/bioc/html/DESeq2.html</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>edgeR&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Empirical
      gene expression analysis using R&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a
      href="https://bioconductor.org/packages/release/bioc/html/edgeR.html"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://bioconductor.org/packages/release/bioc/html/edgeR.html</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>FastQC&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>FastQC&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a
      href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>FASTX&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Toolkit
      for short reads preprocessing&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://hannonlab.cshl.edu/fastx_toolkit/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://hannonlab.cshl.edu/fastx_toolkit/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>IGV&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Interactive
      exploration of genomic data&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="https://www.broadinstitute.org/igv/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://www.broadinstitute.org/igv/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>igvtools&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>For
      preprocessing data before loading to IGV&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="https://www.broadinstitute.org/igv/igvtools"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://www.broadinstitute.org/igv/igvtools</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>MACS&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>ChIP-Seq
      analysis&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://liulab.dfci.harvard.edu/MACS/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://liulab.dfci.harvard.edu/MACS/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>MUMmer&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Rapid
      genome alignment, a dependency for AMOS&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://mummer.sourceforge.net/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://mummer.sourceforge.net/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>PeakAnalyzer&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Multi-peak
      data analysis&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a
      href="http://www.bioinformatics.org/peakanalyzer/wiki/" target="_blank"><span
      style='font-size:9.0pt;line-height:107%'>http://www.bioinformatics.org/peakanalyzer/wiki/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Picard&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Sequence
      data analysis&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://broadinstitute.github.io/picard/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://broadinstitute.github.io/picard/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>SAMtools&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>For
      manipulating alignments in the SAM format&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="http://samtools.sourceforge.net/"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>http://samtools.sourceforge.net/</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
     <tr>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Skewer&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><span style='font-size:9.0pt;line-height:107%'>Adapter
      trimmer for paired-end reads&nbsp;</span></p>
      </td>
      <td style='padding:.75pt .75pt .75pt .75pt'>
      <p class=MsoNormal><a href="https://github.com/relipmoc/skewer"
      target="_blank"><span style='font-size:9.0pt;line-height:107%'>https://github.com/relipmoc/skewer</span></a><span
      style='font-size:9.0pt;line-height:107%'>&nbsp;</span></p>
      </td>
     </tr>
    </table>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><img border=0 width=711 height=444 id="Picture 15"
    src="AWS-Bioinformatics_files/image002.jpg"></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NGS
    Graph Generator on AWS Free Tier</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=&amp;cad=rja&amp;uact=8&amp;ved=2ahUKEwjdn5_JvqrtAhUP2VkKHWdfAM0QFjABegQIBBAC&amp;url=https%3A%2F%2Fwww.biorxiv.org%2Fhighwire%2Ffilestream%2F123738%2Ffield_highwire_adjunct_files%2F0%2F409573-1.docx&amp;usg=AOvVaw1ctSQUOrm8NEQy_lD8FV1p"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/url?sa=t&amp;rct=j&amp;q=&amp;esrc=s&amp;source=web&amp;cd=&amp;cad=rja&amp;uact=8&amp;ved=2ahUKEwjdn5_JvqrtAhUP2VkKHWdfAM0QFjABegQIBBAC&amp;url=https%3A%2F%2Fwww.biorxiv.org%2Fhighwire%2Ffilestream%2F123738%2Ffield_highwire_adjunct_files%2F0%2F409573-1.docx&amp;usg=AOvVaw1ctSQUOrm8NEQy_lD8FV1p</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/systems-immunology-roslin-institute/ngs-graph-generator"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/systems-immunology-roslin-institute/ngs-graph-generator</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Choose
    an Instance Type page</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    page lists all of the available instances type varying combinations of CPU,
    memory, storage and capacity. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    order to use this pipeline, it needs a small memory and low storage, </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>thus
    a user can select for t2.micro (free tier eligible) (1), then click ‘Review
    and Launch’ button (2).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BioLayout
    Express3D or Miru</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:white;border-collapse:collapse;border:none'>
 <tr>
  <td width=767 valign=top style='width:575.6pt;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif;color:black'>AWS EC2 Storage</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=767 valign=top style='width:575.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Practical
  guide for managing large-scale human genome data in research</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> Oct
  2020</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://www.nature.com/articles/s10038-020-00862-1"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s10038-020-00862-1</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>What
  kind of large-scale human genome projects are underway and available from
  data sharing?</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>How
  to store and analyze human genome data efficiently?</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>What
  kind of computational platforms are used to store and analyze human genome
  data?</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>How
  to maintain reproducibility, portability, and scalability in genome data
  analysis, and why is it important?</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Bringing
  data to life</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Data
  management for the biomolecular sciences</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://www.ebi.ac.uk/training-beta/online/courses/bringing-data-life-data-management-biomolecular-sciences/what-is-data-management/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ebi.ac.uk/training-beta/online/courses/bringing-data-life-data-management-biomolecular-sciences/what-is-data-management/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
<br>
</span></b></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>AWS Data<span style='color:black'><br>
  <br>
  </span></span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>What
  type of data transfer is free for Amazon s3?<br>
  <br>
  </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Pick
  the right <span style='background:yellow'>AWS region</span> for your <span
  style='background:yellow'>S3 bucket</span>.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
  main benefit of having S3 and EC2 in the same region is the performance and
  lower transfer cost. <br>
  <br>
  Data transfer is free <br>
  between EC2 and S3 in the same region. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Downloading
  file <br>
  from another AWS region will cost $0.02/GB.  </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>May
  18, 2019</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h1 style='margin-top:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:black'>New AWS public datasets
  available from the National Cancer Institute, Massachusetts Institute of
  Technology, Amazon, the National Renewable Energy Laboratory, and others </span></b></h1>
  <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Posted On: <span
  class=Date1>Jul 15, 2020</span> </span></p>
  <p style='margin:0in'><span style='color:black'><a
  href="https://aws.amazon.com/about-aws/whats-new/2020/07/new-aws-public-datasets/"><span
  style='font-size:10.0pt'>https://aws.amazon.com/about-aws/whats-new/2020/07/new-aws-public-datasets/</span></a></span></p>
  <p style='margin:0in'><span style='font-size:10.0pt;color:black'><br>
  Twenty-three new or updated Amazon Web Services (AWS) public datasets from
  the National Center for Bioinformatics, Johns Hopkins University, University
  of Texas at Southwestern, National Oceanic and Atmospheric Administration
  (NOAA), the National Cancer Institute, National Herbarium of New South Wales,
  and others are now available in the following categories: </span></p>
  <p style='margin:0in'><span style='font-size:10.0pt;color:black'><br>
  COVID-19 response:</span></p>
  <ul style='margin-top:0in' type=disc>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a
       href="https://registry.opendata.aws/molssi-covid19-hub/" target="_blank"><span
       style='font-size:10.0pt;font-family:"Times New Roman",serif'>COVID-19
       Molecular Structure and Therapeutics Hub</span></a></span><span
       style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>
       from the Molecular Sciences Software Institute</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a href="https://registry.opendata.aws/ncbi-covid-19"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>COVID-19
       Genome Sequence Dataset</span></a></span><span style='font-size:10.0pt;
       font-family:"Times New Roman",serif;color:black'> from the National
       Center for Biotechnology Information </span><span style='font-size:10.0pt;
       color:black'>Life sciences:</span></li>
  </ul>
  <ul style='margin-top:0in' type=disc>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a href="https://registry.opendata.aws/jhu-indexes/"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Cloud
       Genomic Indexes</span></a></span><span style='font-size:10.0pt;
       font-family:"Times New Roman",serif;color:black'> from Johns Hopkins
       University and the University of Texas at Southwestern</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a href="https://registry.opendata.aws/refgenie/"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Refgenie
       Genomic Assets</span></a></span><span style='font-size:10.0pt;
       font-family:"Times New Roman",serif;color:black'> from University of
       Virginia</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a href="https://registry.opendata.aws/kids-first/"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Gabriella
       Miller Kids First Pediatric Research Program</span></a></span><span
       style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>
       from the National Cancer Institute</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a href="https://registry.opendata.aws/tcga/"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>The
       Cancer Genome Atlas</span></a></span><span style='font-size:10.0pt;
       font-family:"Times New Roman",serif;color:black'> from the National
       Cancer Institute</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a
       href="https://registry.opendata.aws/ncbi-blast-databases/"
       target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Basic
       Local Alignment Sequence Tool (BLAST) Databases</span></a></span><span
       style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>
       from the National Library of Medicine</span></li>
   <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
       style='color:black'><a
       href="https://registry.opendata.aws/nsw-herbarium/" target="_blank"><span
       style='font-size:10.0pt;font-family:"Times New Roman",serif'>National
       Herbarium of New South Wales</span></a></span><span style='font-size:
       10.0pt;font-family:"Times New Roman",serif;color:black'> from the Royal
       Botanic Gardens and Domain Trust</span></li>
  </ul>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Access
  to Genomic Data for free</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In <b>S3</b>:
  </span><span style='color:black'><a href="http://s3.amazonaws.com/1000genomes"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://s3.amazonaws.com/1000genomes</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGSR:
  The International Genome Sample Resource</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Open
  human variation data</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.internationalgenome.org/home"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/home</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
  Genomes Project FAQ</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>If
  you have any other questions you can’t find the answer to please email </span><span
  style='color:black'><a href="mailto:info@1000genomes.org"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>info@1000genomes.org</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.internationalgenome.org/faq"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/faq</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <h4 style='line-height:normal'><b><span style='font-family:"Times New Roman",serif;
  color:windowtext;font-style:normal'>Google</span></b><b><span
  style='font-family:"Times New Roman",serif;color:black;font-style:normal'>
  Searches:</span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  <br>
  </span></h4>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=813 valign=top style='width:610.1pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <h4 style='line-height:normal'><b><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black;font-style:normal'>Google:</span></b></h4>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><img border=0 width=261 height=111 id="Picture 5"
    src="AWS-Bioinformatics_files/image003.jpg"></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/search?as_q=AWS+AMI+EC2+S3+&amp;as_epq=open+data&amp;as_oq=bioinformatics+bioconductor&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs="><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?as_q=AWS+AMI+EC2+S3+&amp;as_epq=open+data&amp;as_oq=bioinformatics+bioconductor&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs=</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=813 valign=top style='width:610.1pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=1005 valign=top style='width:754.1pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <h4 style='line-height:normal'><b><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black;font-style:normal'>Google:</span></b></h4>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><img border=0 width=263 height=108 id="Picture 1"
    src="AWS-Bioinformatics_files/image004.jpg"></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/search?as_q=AWS+AMI+bioinformatics+&amp;as_epq=&amp;as_oq=%22instance+store-backed%22+%22EBS-backed%22&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs="><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?as_q=AWS+AMI+bioinformatics+&amp;as_epq=&amp;as_oq=%22instance+store-backed%22+%22EBS-backed%22&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs=</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=1005 valign=top style='width:754.1pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=999 valign=top style='width:749.6pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <h4 style='line-height:normal'><b><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black;font-style:normal'>Google: </span></b><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    font-style:normal'>aws genomics tutorial </span></b></h4>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>844,000
    results</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/search?as_q=aws+genomics+tutorial&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs="><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?as_q=aws+genomics+tutorial&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs=</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <h4 style='line-height:normal'><b><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black;font-style:normal'>Google: </span></b><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    font-style:normal'>aws genomics tutorial </span></b></h4>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Videos</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>22,900
    results</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/search?q=aws+genomics+tutorial&amp;lr=&amp;source=lnms&amp;tbm=vid&amp;sa=X&amp;ved=2ahUKEwjOt_zvpvTtAhWJuVkKHXHoDLwQ_AUoAnoECAYQBA"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?q=aws+genomics+tutorial&amp;lr=&amp;source=lnms&amp;tbm=vid&amp;sa=X&amp;ved=2ahUKEwjOt_zvpvTtAhWJuVkKHXHoDLwQ_AUoAnoECAYQBA</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=999 valign=top style='width:749.6pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Big
    Data Analysis in the Cloud</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Oct
    24, 2016</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.youtube.com/watch?v=LdPj5UKWtx0"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=LdPj5UKWtx0</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>How
    to Access SRA Data with Amazon Web Services</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.youtube.com/watch?v=rjjrHnZfymU&amp;list=PLH-TjWpFfWrt5MNqU7Jvsk73QefO3ADwD"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=rjjrHnZfymU&amp;list=PLH-TjWpFfWrt5MNqU7Jvsk73QefO3ADwD</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Accelerate
    Genomics Discovery with SRA in the Cloud</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.youtube.com/watch?v=PoseTXa8LU8&amp;list=PLH-TjWpFfWrt5MNqU7Jvsk73QefO3ADwD&amp;index=3"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=PoseTXa8LU8&amp;list=PLH-TjWpFfWrt5MNqU7Jvsk73QefO3ADwD&amp;index=3</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Accelerating
    Life Sciences with HPC on AWS - AWS Online Tech Talks</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.youtube.com/watch?v=fscMSO_vQ7o"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=fscMSO_vQ7o</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.youtube.com/watch?v=YfCgK1bmCjw"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=YfCgK1bmCjw</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=999 valign=top style='width:749.6pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <h4 style='line-height:normal'><b><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black;font-style:normal'>Google:</span></b></h4>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><img border=0 width=272 height=114 id="Picture 7"
    src="AWS-Bioinformatics_files/image005.jpg"></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.google.com/search?as_q=AWS+Bioinformatics+NGS+&amp;as_epq=&amp;as_oq=BMC+bioRxio+&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs="><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?as_q=AWS+Bioinformatics+NGS+&amp;as_epq=&amp;as_oq=BMC+bioRxio+&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs=</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=999 valign=top style='width:749.6pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
name="_Hlk60221252"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=1019 valign=top style='width:764.6pt;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b>Google:</b><span
  style='color:black'> </span><b><span style='font-family:"Times New Roman",serif;
  color:black'>aws genomics guide</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.google.com/search?q=aws+genomics+guide&amp;lr=&amp;as_qdr=all&amp;ei=6LbrX8LrOMez5gKb6aXYDQ&amp;start=20&amp;sa=N&amp;ved=2ahUKEwjC_-69p_TtAhXHmVkKHZt0Cds4ChDy0wN6BAgGEDg&amp;biw=1094&amp;bih=569"><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?q=aws+genomics+guide&amp;lr=&amp;as_qdr=all&amp;ei=6LbrX8LrOMez5gKb6aXYDQ&amp;start=20&amp;sa=N&amp;ved=2ahUKEwjC_-69p_TtAhXHmVkKHZt0Cds4ChDy0wN6BAgGEDg&amp;biw=1094&amp;bih=569</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
 <tr>
  <td width=1019 valign=top style='width:764.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AWS
    genomics Non-Profit Pubs</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    Biostars</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Tutorial:
    Introduction to AWS Cloud Computing</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://www.biostars.org/p/137477/"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://www.biostars.org/p/137477/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Broad
    Institute.org</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genome
    STRiP AWS cluster as described in Running Genome STRiP on the Amazon cloud</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="http://software.broadinstitute.org/software/genomestrip/cookbook-genotyping-novel-site-1000-genomes-phase-1-using-aws"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://software.broadinstitute.org/software/genomestrip/cookbook-genotyping-novel-site-1000-genomes-phase-1-using-aws</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGSR:
    International Genome Sample Resource</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>open
    human variation data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Using
    1000 Genomes Data in the Amazon Web Service Cloud</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://www.internationalgenome.org/using-1000-genomes-data-amazon-web-service-cloud/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/using-1000-genomes-data-amazon-web-service-cloud/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Bioinformatics
    UC Davis </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Introduction
    to Amazon Web Services EC2 - lecture</span></b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'><br>
    </span><a
    href="https://bioshare.bioinformatics.ucdavis.edu/bioshare/download/kveirzo6fvkl2nb/build/Monday-AWS-Intro.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioshare.bioinformatics.ucdavis.edu/bioshare/download/kveirzo6fvkl2nb/build/Monday-AWS-Intro.html</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Practical
    guide for managing large-scale human genome data in research </span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Oct
    2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://www.nature.com/articles/s10038-020-00862-1"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s10038-020-00862-1</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AWS
    genomics Amazon Pubs</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Building
    High-Throughput Genomics Batch Workflows on AWS: Introduction (Part 1 of 4)
    </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
    color:black'>MAY 2017</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/"><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genomics
    Workflows on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://docs.opendata.aws/genomics-workflows/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://docs.opendata.aws/genomics-workflows/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GitHub:
    aws-samples / aws-genomics-workflows</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://github.com/aws-samples/aws-genomics-workflows"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/aws-samples/aws-genomics-workflows</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>DRAGEN
    reanalysis of the 1000 Genomes Dataset now available on the Registry of
    Open Data</span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
    color:black'> OCT 2020 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://aws.amazon.com/blogs/industries/dragen-reanalysis-of-the-1000-genomes-dataset-now-available-on-the-registry-of-open-data/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/industries/dragen-reanalysis-of-the-1000-genomes-dataset-now-available-on-the-registry-of-open-data/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    release of a comprehensive reanalysis of 3,202 deeply-sequenced samples
    from the 1000 Genomes Project(1kGP) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>using
    the Illumina DRAGEN (Dynamic Read Analysis for GENomics) Bio-IT platform. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    seminal dataset will be freely available for researchers across the world
    to use as part of their genomics analysis. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    alignment files small variant, copy-number, structural, and short-tandem
    repeat variant calls </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>are
    now available at no cost through the Registry of Open Data on AWS.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Project and AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://docs.opendata.aws/1000genomes/readme.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://docs.opendata.aws/1000genomes/readme.html</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    1000 Genomes Project is an international collaboration which has
    established the most detailed catalogue of human genetic variation, </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>including
    SNPs, structural variants, and their haplotype context. The final phase of
    the project sequenced more than 2500 individuals </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>from
    26 different populations around the world and produced an integrated set of
    phased haplotypes with more than 80 million variants </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>for
    these individuals.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Amazon mirror contains the complete data set from the project and the data
    can be found in the </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    background:yellow'>s3://1000genomes bucket in the us-east-1 AWS region</span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>For
    more information </span><a href="http://www.1000genomes.org"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.1000genomes.org</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AWS
    Labs</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://github.com/awslabs"><span style='font-size:9.0pt;font-family:
    "Times New Roman",serif'>https://github.com/awslabs</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AWS
    Labs Machine Learning </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://github.com/awslabs/dgl-ke"><span style='font-size:9.0pt;
    font-family:"Times New Roman",serif'>https://github.com/awslabs/dgl-ke</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon EC2</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Amazon
  Elastic Compute Cloud </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Secure
  and resizable compute capacity to support virtually any workload </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://aws.amazon.com/ec2/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/ec2/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Is a
  web service that provides secure, resizable compute capacity in the cloud. It
  is designed to make web-scale cloud computing easier for developers. Amazon
  EC2’s simple web service interface allows you to obtain and configure
  capacity with minimal friction. It provides you with complete control of your
  computing resources and lets you run on Amazon’s proven computing
  environment.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Amazon
  EC2 offers the broadest and deepest compute platform with choice of
  processor, storage, networking, operating system, and purchase model. We
  offer the fastest processors in the cloud and we are the only cloud with 400
  Gbps ethernet networking.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Free Tier</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&amp;all-free-tier.sort-order=asc&amp;awsf.Free%20Tier%20Types=tier%2312monthsfree&amp;awsf.Free%20Tier%20Categories=categories%23compute%7Ccategories%23analytics"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/free/?all-free-tier.sort-by=item.additionalFields.SortRank&amp;all-free-tier.sort-order=asc&amp;awsf.Free%20Tier%20Types=tier%2312monthsfree&amp;awsf.Free%20Tier%20Categories=categories%23compute%7Ccategories%23analytics</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Gain free,
  hands-on experience with the AWS platform, products, and services</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Explore more
  than 85 products and start building on AWS using the free tier. Three
  different types of free offers are available depending on the product used.
  See below for details on each product.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Pricing
  Calculator</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://calculator.aws/#/addService"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://calculator.aws/#/addService</span></a><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  <br>
  </span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon Elastic
  Block Store (EBS)</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://aws.amazon.com/ebs/"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>https://aws.amazon.com/ebs/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>allows you to
  create persistent block storage volumes and attach them to Amazon EC2
  instances.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon Elastic
  Block Store (EBS) is an easy to use, high performance block storage service
  designed for use with Amazon Elastic Compute Cloud (EC2) for both throughput
  and transaction intensive workloads at any scale. A broad range of workloads,
  such as relational </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>and
  non-relational databases, enterprise applications, containerized applications,
  big data analytics engines, file systems, and media workflows are widely
  deployed on Amazon EBS.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>You can choose
  from six different volume types to balance optimal price and performance. You
  can achieve single digit-millisecond latency for high performance database
  workloads such as SAP HANA or gigabyte per second throughput for large,
  sequential workloads such </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>as Hadoop. You
  can change volume types, tune performance, or increase volume size without
  disrupting your critical applications, so you have cost-effective storage
  when you need it.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Designed for
  mission-critical systems, EBS volumes are replicated within an Availability
  Zone (AZ) and can easily scale to petabytes of data. Also, you can use EBS
  Snapshots with automated lifecycle policies to back up your volumes in Amazon
  S3, while ensuring </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>geographic
  protection of your data and business continuity.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <h1 style='line-height:normal'><a
  name="Introducing_new_Amazon_EBS_general_purpo"><b><span style='font-size:
  11.0pt;font-family:"Times New Roman",serif;color:windowtext'>Introducing new
  Amazon EBS general purpose volumes, gp3 </span></b></a><b><span
  style='font-size:11.0pt;font-family:"Times New Roman",serif;color:windowtext'><br>
  </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
  color:windowtext'>Posted On: <span class=Date1>Dec 1, 2020</span> </span></h1>
  <p><span style='font-size:10.0pt'>Today AWS announced the availability of <span
  style='background:yellow'>gp3</span>, the <span style='background:yellow'>next-generation
  general purpose SSD volumes for Amazon Elastic Block Store (Amazon EBS)</span>
  that enable customers to <br>
  <span style='background:yellow'>provision performance independent of storage
  capacity and provides up to 20% lower price-point per GB than existing gp2
  volumes</span>. <br>
  With gp3 volumes, customers can scale IOPS (input/output operations per
  second) and throughput without needing to provision additional block storage
  capacity, and pay only for the resources they need. &nbsp;</span></p>
  <p><span style='font-size:10.0pt'>General purpose SSD volumes make it easy
  and cost effective for customers to meet the IOPS and throughput requirements
  for transaction-intensive workloads, such as virtual desktops, test and
  development environments, low-latency interactive applications, and boot
  volumes. With existing general-purpose SSD (gp2) volumes, performance is tied
  to storage capacity, enabling customers to get higher IOPS and throughput for
  their applications by provisioning a larger storage volume size. But
  customers want to scale performance and throughput without paying for storage
  that they don’t need. &nbsp;</span></p>
  <p><span style='font-size:10.0pt'>Next generation gp3 volumes offer the
  ability to independently provision IOPS and throughput, separate from storage
  capacity. This enables customers to scale performance for transaction-intensive
  workloads without needing to provision more capacity, so they only pay for
  the resources they need. The new gp3 volumes also deliver a baseline
  performance of 3,000 IOPS and 125MB/s at any volume size. For use cases,
  where your application needs more performance than the baseline, customers
  can scale up to 16,000 IOPS and 1,000 MB/s for an additional fee. This makes
  the new gp3 volumes ideal for a wide variety of applications that require
  high performance at low cost, including MySQL, Cassandra, virtual desktops,
  and Hadoop analytics clusters. &nbsp;</span></p>
  <p><span style='font-size:10.0pt'>Customers can easily <span
  style='background:yellow'>migrate gp2 volumes to gp3 volumes using Elastic
  Volumes</span>, which is an existing feature of Amazon EBS. <br>
  Elastic Volumes allow customers to <span style='background:yellow'>modify the
  volume type, IOPS, or throughput of their existing EBS volumes without
  interrupting their Amazon EC2 instances</span>. gp3 volumes are available in
  all AWS commercial and gov cloud regions. For more information, please see
  the gp3 announcement on the AWS News blog, and documentation.<br>
  <br>
  </span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Volume Gateway</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Hybrid cloud
  block storage with local caching</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://aws.amazon.com/storagegateway/volume/"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/storagegateway/volume/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Volume Gateway
  presents cloud-backed iSCSI block storage volumes to your on-premises applications.
  Volume Gateway stores and manages on-premises data in Amazon S3 on your
  behalf and operates in either cache mode or stored mode. In the cached Volume
  Gateway mode, your primary data is stored in Amazon S3, while retaining your
  frequently accessed data locally in the cache for low latency access. In the
  stored Volume Gateway mode, your primary data is stored locally and your
  entire dataset is available for low latency access on </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>premises while
  also asynchronously getting backed up to Amazon S3. In either mode, you can
  take point-in-time copies of your volumes using AWS Backup, which are stored
  in AWS as Amazon EBS snapshots. Using Amazon EBS Snapshots enables you to
  make space-</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>efficient
  versioned copies of your volumes for data protection, recovery, migration,
  and various other copy data needs.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon Simple
  Storage Service (S3)</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Object storage
  built to store and retrieve any amount of data from anywhere</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://aws.amazon.com/s3/"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>https://aws.amazon.com/s3/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>(Amazon S3) is
  storage for the internet. You can use Amazon S3 to store and retrieve any
  amount of data at any time, from anywhere on the web.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
  ElastiCache</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>offers fully
  managed Redis and Memcached. Seamlessly deploy, run, and scale popular open
  source compatible in-memory data stores. Build data-intensive apps or improve
  the performance of your existing apps by retrieving data from high throughput
  and low latency in-memory data stores.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Storage
  Gateway</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://aws.amazon.com/storagegateway/"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://aws.amazon.com/storagegateway/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Storage
  Gateway is a hybrid storage service that enables your on-premises
  applications to seamlessly use AWS cloud storage. You can use the service for
  backup and archiving, disaster recovery, cloud data processing, storage
  tiering, and migration.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon Elastic
  Container Registry</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>(ECR) is a
  fully-managed Docker container registry that makes it easy for developers to
  store, manage, and deploy Docker container images. Amazon ECR eliminates the
  need to operate your own container repositories or worry about scaling the
  underlying infrastructure. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>With Amazon ECR,
  there are no upfront fees or commitments. You pay only for the amount of data
  you store in your repositories and data transferred to the Internet.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon RDS for
  PostgreSQL</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon RDS makes
  it easy to set up, operate, and scale PostgreSQL deployments in the cloud.
  With Amazon RDS, you can deploy scalable PostgreSQL deployments in minutes
  with cost-efficient and resizable hardware capacity. <br>
  <br>
  </span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Start
    AWS Computational Genomics Image</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Video</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Aug
    25, 2018</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.youtube.com/watch?v=AiSvnLamE5Y"><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=AiSvnLamE5Y</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Exploratory
    data analysis of genomic datasets using ADAM and Mango with Apache Spark on
    Amazon EMR</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>13
    JUL 2018 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Amazon
    EMR, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/"><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/adam" target="_blank"><b><span
    style='font-size:10.0pt;color:black;text-decoration:none'>ADAM</span></b></a></span><b><span
    style='font-size:10.0pt;color:black'> </span></b></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/adam"><span style='font-size:10.0pt'>https://github.com/bigdatagenomics/adam</span></a></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>and </span></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/mango"><b><span style='font-size:
    10.0pt;color:black;text-decoration:none'>Mango</span></b></a></span><b><span
    style='font-size:10.0pt;color:black'> </span></b></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/mango"><span style='font-size:
    10.0pt'>https://github.com/bigdatagenomics/mango</span></a></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>provide a
    unified environment for processing, filtering, and visualizing large
    genomic datasets on Apache Spark. ADAM allows users to programmatically
    load, process, and select raw genomic and variation data using SparkSQL, an
    SQL interface for aggregating and selecting data in Apache Spark. Mango
    supports visualization of both raw and aggregated genomic data in a Jupyter
    notebook environment, allowing users to draw conclusions from large
    datasets at multiple resolutions. This combined power of ADAM and Mango
    allows users to load, query and explore datasets in a unified environment,
    allowing users to interactively explore genomic data at a scale previously
    impossible using single node bioinformatics tools.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></h2>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>Configuring ADAM and Mango on
    Amazon EMR</span></b></h2>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Loading data from the 1000 Genomes Project</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now that
    we have a working environment, lets use ADAM and Mango to discover
    interesting variants in the child from the genome sequencing data of a trio
    (data from a mother, father, and child). These data are available from the <br>
    </span><span style='color:black'><a
    href="https://aws.amazon.com/1000genomes/" target="_blank"><span
    style='font-size:10.0pt'>1000 Genomes Project AWS Public Dataset</span></a></span><span
    style='font-size:10.0pt;color:black'>. <br>
    </span><span style='color:black'><a
    href="https://registry.opendata.aws/1000-genomes/"><span style='font-size:
    10.0pt'>https://registry.opendata.aws/1000-genomes/</span></a></span></p>
    <p style='margin:0in'><b><span style='font-size:10.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>Resources on AWS</span></b></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Description</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        </span><span style='color:black'><a
    href="http://www.internationalgenome.org/formats"><span style='font-size:
    9.0pt;font-family:"Segoe UI",sans-serif'>http://www.internationalgenome.org/formats</span></a></span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Resource type</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        S3 Bucket</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Amazon Resource Name (ARN)</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        arn:aws:s3:::1000genomes</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    AWS Region</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        us-east-1</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    AWS CLI Access (No AWS account required)</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        aws s3<br>
    <br>
    </span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Import ADAM modules</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Import Mango modules</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Import Spark modules</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Create ADAM Context</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Variant analysis with Spark SQL</span></b></h2>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># repartition genotypes to balance the load across memory</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># cache genotypes and show the schema</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># trio IDs</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Filter by individuals in the trio</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Add ReferenceRegion column and group by referenceRegion</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>#  Register df with Spark SQL</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># filter by alleles. This is a list of variant names that have an alternate allele for the child</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># get parent records and filter by only REF locations for variant names that were found in the child with an ALT</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># collect snp names as a list</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Working with alignment data</span></b></h2>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># load in NA19685 exome from s3a</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># cache child RDD and count records</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># takes about 2 minutes, on 4 c3.4xlarge worker nodes </span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Count reads in the child</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Output should be 95634679</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Quality control of alignment data</span></b></h2>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Calculate read coverage</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Takes 2-3 minutes</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Output should be 51252612</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Calculate coverage distribution</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># You can check the progress in the SparkUI by navigating to </span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># :8088 and clicking on the currently running Spark application.</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>&nbsp;</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=564 height=320 id="Picture 18"
    src="AWS-Bioinformatics_files/image006.png"></span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Viewing sites with missense variants in the proband</span></b></h2>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># missense variant at GNA13: 63052580-63052581 (SNP rs201316886)</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># define alignment summary for child reads</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Takes about 2 minutes to collect data from workers</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=574 height=306 id="Picture 17"
    src="AWS-Bioinformatics_files/image007.png"></span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># define alignment summary for parent reads</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># view missense variant at GNA13: 63052580-63052581 in parent 1</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=583 height=315 id="Picture 16"
    src="AWS-Bioinformatics_files/image008.png"></span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>Summary</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>To
    summarize, this post demonstrated how to set up and run ADAM and Mango in
    Amazon EMR. We demonstrated how to use these tools in an interactive
    notebook environment to explore the 1000 Genomes dataset, a publicly
    available dataset on Amazon S3. <br>
    We used these tools inspect 1000 Genomes data quality, query for
    interesting variants in the genome, and validate results through the
    visualization of raw data.<br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Orchestrating
    analytics jobs by running Amazon EMR Notebooks programmatically</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>23
    NOV 2020 | Amazon EMR, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/orchestrating-analytics-jobs-by-running-amazon-emr-notebooks-programmatically/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/orchestrating-analytics-jobs-by-running-amazon-emr-notebooks-programmatically/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='color:black'><a
    href="http://aws.amazon.com/emr" target="_blank"><span style='font-size:
    10.0pt'>Amazon EMR</span></a></span><span style='font-size:10.0pt;
    color:black'> is a big data service offered by AWS to run Apache Spark and
    other open-source applications on AWS in a cost-effective manner. </span><span
    style='color:black'><a
    href="https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-managed-notebooks.html"
    target="_blank"><span style='font-size:10.0pt'>Amazon EMR Notebooks</span></a></span><span
    style='font-size:10.0pt;color:black'> is a managed environment based on </span><span
    style='color:black'><a
    href="https://en.wikipedia.org/wiki/Project_Jupyter#Jupyter_Notebook"
    target="_blank"><span style='font-size:10.0pt'>Jupyter Notebook</span></a></span><span
    style='font-size:10.0pt;color:black'> that allows data scientists,
    analysts, and developers to prepare and visualize data, collaborate with
    peers, build applications, and perform interactive analysis using EMR
    clusters.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>EMR
    notebook APIs are available on Amazon EMR release version 5.18.0 or later
    and can be used to run EMR notebooks via a script or command line. The
    ability to start, stop, list, and describe EMR notebook runs without the
    Amazon EMR console enables you to programmatically control running an EMR
    notebook. Using a parameterized notebook cell allows you to pass different
    parameter values to a notebook without having to create a copy of the
    notebook for each new set of parameter values. With this feature, you can
    schedule running EMR notebooks with cron scripts, chain multiple EMR
    notebooks, and use orchestration services such as </span><span
    style='color:black'><a href="https://aws.amazon.com/step-functions/"
    target="_blank"><span style='font-size:10.0pt'>AWS Step Functions</span></a></span><span
    style='font-size:10.0pt;color:black'> or Apache Airflow to build pipelines.
    If you want to use EMR notebooks in a non-interactive manner, this enables
    you to run ETL workloads, especially in production.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In this
    post, we show how to orchestrate analytics jobs by running EMR Notebooks
    programmatically with the following two use cases:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Scheduling an EMR notebook run via crontab and the </span><span
         style='color:black'><a href="http://aws.amazon.com/cli" target="_blank"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
         Command Line Interface</span></a></span><span style='font-size:10.0pt;
         font-family:"Times New Roman",serif;color:black'> (AWS CLI)</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Chaining your notebooks with Step Functions triggered by </span><span
         style='color:black'><a
         href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html"
         target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
         CloudWatch Events</span></a></span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>For our
    data source, we use the open-source, real-time COVID-19 US daily case
    reports provided by Johns Hopkins University CSSE in the following </span><span
    style='color:black'><a
    href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports_us"
    target="_blank"><span style='font-size:10.0pt'>GitHub repo</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Prerequisites</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Before
    getting started, you must have the following prerequisites:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An AWS account that provides access to the following AWS
         services at least: </span></li>
     <ul style='margin-top:0in' type=circle>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/cloudformation"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          CloudFormation</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>Amazon CloudWatch</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/ec2"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          Elastic Compute Cloud</span></a></span><span style='font-size:10.0pt;
          font-family:"Times New Roman",serif;color:black'> (Amazon EC2)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>Amazon EMR</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="https://aws.amazon.com/eventbridge/"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          EventBridge</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/iam"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          Identity and Access Management</span></a></span><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'> (IAM)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/lambda"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          Lambda</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/s3" target="_blank"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          Simple Storage Service</span></a></span><span style='font-size:10.0pt;
          font-family:"Times New Roman",serif;color:black'> (Amazon S3)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>AWS Step Functions</span></li>
     </ul>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>AWS CLI Version 1.18.128 or later installed on your work
         station.</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a href="https://jupyter.org/install"
         target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Jupyter</span></a></span><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> installed on your work station (this is used for the
         output visualization part for this post only).</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An EMR cluster running Amazon EMR release 5.18.0 or
         later, with Hadoop, Spark, and Livy installed. Record the value of the
         cluster ID (for example, </span><em><span style='font-size:10.0pt;
         color:red'>&lt;j-*************&gt;</span></em><span style='font-size:
         10.0pt;font-family:"Times New Roman",serif;color:black'>); you use
         this for the examples later.</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An EMR notebook created on the Amazon EMR console, using
         the following two input notebook files: </span></li>
     <ul style='margin-top:0in' type=circle>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a
          href="https://aws-bigdata-blog.s3.amazonaws.com/artifacts/aws-blog-runnable-notebook/demo/notebook/demo_pyspark.ipynb"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>demo_pyspark.pynb</span></a></span><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'> – Used for both use cases in this post.</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a
          href="https://aws-bigdata-blog.s3.amazonaws.com/artifacts/aws-blog-runnable-notebook/demo/notebook/trailing_N_day.ipynb"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>trailing_N
          day.ipynb</span></a></span><span style='font-size:10.0pt;font-family:
          "Times New Roman",serif;color:black'> – Used for the second use case.</span></li>
     </ul>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Record the
    notebook ID (for example, </span><em><span style='font-size:10.0pt;
    color:red'>&lt;e-*************************&gt;</span></em><span
    style='font-size:10.0pt;color:black'>); you use this later for our examples
    later. Organize the notebook files in the Jupyter UI as follows:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>/demo_pyspark.ipynb</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>/experiment/trailing_N_day.ipynb</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>See </span><span
    style='color:black'><a
    href="https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-managed-notebooks-create.html"
    target="_blank"><span style='font-size:10.0pt'>Creating a Notebook</span></a></span><span
    style='font-size:10.0pt;color:black'> for more information on how to create
    an EMR notebook.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Interactive
    Analysis of Genomic Datasets Using Amazon Athena</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Aaron Friedman, PhD | on 07 DEC 2016 | in Amazon Athena, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/interactive-analysis-of-genomic-datasets-using-amazon-athena/"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/interactive-analysis-of-genomic-datasets-using-amazon-athena/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><em><span style='font-size:10.0pt;color:black'>Aaron
    Friedman is a Healthcare and Life Sciences Solutions Architect with Amazon
    Web Services</span></em></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    genomics industry is in the midst of a data explosion. Due to the rapid
    drop in the cost to sequence genomes, genomics is now central to many
    medical advances. When your genome is sequenced and analyzed, raw
    sequencing files are processed in a multi-step workflow to identify where
    your genome differs from a standard reference. Your variations are stored
    in a Variant Call Format (VCF) file, which is then combined with other
    individuals to enable population-scale analyses. Many of these datasets are
    publicly available, and an increasing number are hosted on AWS as part of
    our </span><span style='color:black'><a
    href="https://aws.amazon.com/public-data-sets/" target="_blank"><span
    style='font-size:10.0pt'>Open Data</span></a></span><span style='font-size:
    10.0pt;color:black'> project.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>To mine
    genomic data for new discoveries, researchers in both industry and academia
    build complex models to analyze populations at scale. When building models,
    they first explore the datasets-of-interest to understand what questions
    the data might answer. In this step, interactivity is key, as it allows
    them to move easily from one question to the next.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Recently,
    we launched </span><span style='color:black'><a
    href="https://aws.amazon.com/athena/"><span style='font-size:10.0pt'>Amazon
    Athena</span></a></span><span style='font-size:10.0pt;color:black'> as an
    interactive query service to analyze data on </span><span style='color:
    black'><a href="https://aws.amazon.com/s3/" target="_blank"><span
    style='font-size:10.0pt'>Amazon S3</span></a></span><span style='font-size:
    10.0pt;color:black'>. With Amazon Athena there are no clusters to manage
    and tune, no infrastructure to setup or manage, and customers pay only for
    the queries they run. Athena is able to query many file types straight from
    S3. This flexibility gives you the ability to interact easily with your
    datasets, whether they are in a raw text format (CSV/JSON) or specialized
    formats (e.g. Parquet). By being able to flexibly query different types of
    data sources, researchers can more rapidly progress through the data
    exploration phase for discovery. Additionally, researchers don’t have to
    know nuances of managing and running a big data system. This makes Athena
    an excellent complement to data warehousing on </span><span
    style='color:black'><a href="https://aws.amazon.com/redshift/"
    target="_blank"><span style='font-size:10.0pt'>Amazon Redshift</span></a></span><span
    style='font-size:10.0pt;color:black'> and big data analytics on </span><span
    style='color:black'><a href="https://aws.amazon.com/emr/" target="_blank"><span
    style='font-size:10.0pt'>Amazon EMR</span></a></span><span
    style='font-size:10.0pt;color:black'>.&nbsp; </span><strong><span
    style='font-size:10.0pt;color:black'>&nbsp;</span></strong></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In this
    post, I discuss how to prepare genomic data for analysis with Amazon Athena
    as well as demonstrating how Athena is well-adapted to address common
    genomics query paradigms.&nbsp; I use the </span><span style='color:black'><a
    href="https://aws.amazon.com/1000genomes/" target="_blank"><span
    style='font-size:10.0pt'>Thousand Genomes dataset</span></a></span><span
    style='font-size:10.0pt;color:black'> hosted on Amazon S3, a seminal
    genomics study, to demonstrate these approaches. All code that is used as
    part of this post is available in our GitHub </span><span style='color:
    black'><a
    href="https://github.com/awslabs/aws-big-data-blog/tree/master/aws-blog-athena-genomics/"
    target="_blank"><span style='font-size:10.0pt'>repository</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Although
    this post is focused on genomic analysis, similar approaches can be applied
    to any discipline where large-scale, interactive analysis is required.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>&nbsp;</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Select, aggregate, annotate query
    pattern in genomics</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Genomics
    researchers may ask different questions of their dataset, such as:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What variations in a genome may increase the risk of
         developing disease?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What positions in the genome have abnormal levels of
         variation, suggesting issues in quality of sequencing or errors in the
         genomic reference?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What variations in a genome influence how an individual
         may respond to a specific drug treatment?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Does a group of individuals contain a higher frequency of
         a genomic variant known to alter response to a drug relative to the
         general population?</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>All these
    questions, and more, can be generalized under a common query pattern I like
    to call “Select, Aggregate, Annotate”. Some of our genomics customers, such
    as Human Longevity, Inc., routinely use this query pattern </span><span
    style='color:black'><a href="https://www.youtube.com/watch?v=CGbWEkszAlQ"
    target="_blank"><span style='font-size:10.0pt'>in their work</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In each of
    the above queries, you execute the following steps:</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>SELECT:
    </span></strong><span style='font-size:10.0pt;color:black'>Specify the
    cohort of individuals meeting certain criteria (disease, drug response,
    age, BMI, entire population, etc.).</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>AGGREGATE:
    </span></strong><span style='font-size:10.0pt;color:black'>Generate summary
    statistics of genomic variants across the cohort that you selected.</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>ANNOTATE:</span></strong><span
    style='font-size:10.0pt;color:black'> Assign meaning to each of the
    variants by joining on known information about each variant.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Dataset preparation</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Properly
    organizing your dataset is one of the most critical decisions for enabling
    fast, interactive analyses. Based on the query pattern I just described,
    the table representing your population needs to have the following
    information:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>A unique sample ID corresponding to each sample in your
         population</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Information about each variant, specifically its location
         in the genome as well as the specific deviation from the reference</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Information about how many times in a sample a variant
         occurs (0, 1, or 2 times) as well as if there are multiple variants in
         the same site. This is known as a genotype.</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    extract, transform, load (ETL) process to generate the appropriate data
    representation has two main steps. First, you use ADAM, a genomics analysis
    platform built on top of Spark, to convert the variant information residing
    a VCF file to Parquet for easier downstream analytics, in a process similar
    to the one described in the </span><span style='color:black'><a
    href="https://blogs.aws.amazon.com/bigdata/post/Tx1GE3J0NATVJ39/Will-Spark-Power-the-Data-behind-Precision-Medicine"
    target="_blank"><span style='font-size:10.0pt'>Will Spark Power the Data
    behind Precision Medicine?</span></a></span><span style='font-size:10.0pt;
    color:black'> post. Then, you use custom Python code to massage the data
    and select only the appropriate fields that you need for analysis with
    Athena.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First, </span><span
    style='color:black'><a
    href="http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-gs.html"
    target="_blank"><span style='font-size:10.0pt'>spin up an EMR cluster</span></a></span><span
    style='font-size:10.0pt;color:black'> (version 5.0.3) for the ETL process.
    I used a c4.xlarge for my master node and m4.4xlarges with 1 TB of scratch
    for my core nodes.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>After you
    SSH into your master node, clone the </span><span style='color:black'><a
    href="https://github.com/awslabs/aws-big-data-blog.git" target="_blank"><span
    style='font-size:10.0pt'>git repository</span></a></span><span
    style='font-size:10.0pt;color:black'>. You can also put this in as a
    bootstrap action when spinning up your cluster.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Building
    High-Throughput Genomics Batch Workflows on AWS: Introduction (Part 1 of 4)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Andy Katz | on 30 MAY 2017 |</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genomics
    Research on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/aws-samples/aws-batch-genomics/tree/v1.0.0"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/aws-samples/aws-batch-genomics/tree/v1.0.0</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    tutorial on how to package and deploy a bioinformatics workflow on AWS
    using AWS Batch</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    tutorial will cover the material presented within the &quot;Genomics
    Workflows on AWS&quot; blog post series </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>(Part
    1, Part 2, Part 3, Part 4) that covers the basics of bootstrapping a
    bioinformatics analysis pipeline on AWS. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>We
    break down the tutorial roughly as follows:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Setting up your AWS account (if you do not already have one)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Package a set of bioinformatics applications using Docker</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Create a AWS Batch environment for analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Define and deploy AWS Step Functions to control the data processing steps</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Initiate a workflow</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Optimizing
    for cost, availability and throughput by selecting your AWS Batch
    allocation strategy</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Bala Thekkedath | on 24 OCT 2019 | in Advanced (300), Amazon EC2, AWS Batch</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>AWS offers
    a broad range of instances that are advantageous for batch workloads. The
    scale and provisioning speed of AWS’ compute instances allow you to get up
    and running at peak capacity in minutes without paying for downtime. Today,
    I’m pleased to introduce allocation strategies: a significant new
    capability in </span><span style='color:black'><a
    href="https://docs.aws.amazon.com/batch/latest/userguide/allocation-strategies.html"><span
    style='font-size:10.0pt'>AWS Batch</span></a></span><span style='font-size:
    10.0pt;color:black'> that makes provisioning compute resources flexible and
    simple. In this blog post, I explain how the AWS Batch allocation
    strategies work, when you should use them for your workload, and provide an
    example CloudFormation script. This blog helps you get started on building
    your personalized Compute Environment (CE) most appropriate to your
    workloads.</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'><br>
    Overview</span></strong></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>AWS Batch
    is a fully managed, cloud-native batch scheduler. It manages the queuing
    and scheduling of your batch jobs, and the resources required to run your
    jobs. One of AWS Batch’s great strengths is the ability to manage instance
    provisioning as your workload requirements and budget needs change. AWS
    Batch takes advantage of AWS’s broad base of compute types. For example,
    you can launch compute-based instances and memory instances that can handle
    different workload types, without having to worry about building a cluster
    to meet peak demand.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Previously,
    AWS Batch had a cost-controlling approach to manage compute instances for
    your workloads. The service chose an instance that was the best fit for
    your jobs based on vCPU, memory, and GPU requirements, at the lowest cost.
    Now, the newly added allocation strategies provide flexibility. They allow
    AWS Batch to consider capacity and throughput in addition to cost when
    provisioning your instances. This allows you to leverage different
    priorities when launching instances depending on your workloads’ needs,
    such as: controlling cost, maximizing throughput, or minimizing </span><span
    style='color:black'><a href="https://aws.amazon.com/ec2/spot/"><span
    style='font-size:10.0pt'>Amazon EC2 Spot</span></a></span><span
    style='font-size:10.0pt;color:black'> instances interruption rates.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>There are
    now three instance allocation strategies from which to choose when creating
    an AWS Batch </span><span style='color:black'><a
    href="https://docs.aws.amazon.com/batch/latest/userguide/compute_environments.html"><span
    style='font-size:10.0pt'>Compute Environment</span></a></span><span
    style='font-size:10.0pt;color:black'> (CE). They are:</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Spot Capacity Optimized</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit Progressive</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:12.0pt;font-family:"Times New Roman",serif'>Registry of Open
  Data on AWS</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://registry.opendata.aws/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://registry.opendata.aws/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>All
  of this is done with open data on S3 buckets</span></b></p>
  <h4 style='line-height:normal'><b>currently 205 datasets</b></h4>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif;color:black;background:yellow'>See</span></b><b><span
  style='font-family:"Times New Roman",serif;color:black'> </span></b><span
  style='color:black'><a href="https://registry.opendata.aws/usage-examples"><b><span
  style='font-family:"Times New Roman",serif;color:black;background:yellow;
  text-decoration:none'> Usage Examples</span></b><b><span style='font-family:
  "Times New Roman",serif;color:black;text-decoration:none'>/<span
  style='background:yellow'>Tutorials</span> for datasets listed in this
  registry</span></b></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Encyclopedia of DNA
    Elements (ENCODE)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://registry.opendata.aws/encode-project/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://registry.opendata.aws/encode-project/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Encyclopedia of DNA Elements (ENCODE) Consortium is an international
    collaboration of research groups funded by the National Human Genome
    Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive
    parts list of functional elements in the human genome, including elements
    that act at the protein and RNA levels, and regulatory elements that
    control cells and circumstances in which a gene is active. ENCODE
    investigators employ a variety of assays and methods to identify functional
    elements. The discovery and annotation of gene elements is accomplished
    primarily by sequencing a diverse range of RNA sources, comparative
    genomics, integrative bioinformatic methods, and human curation. Regulatory
    elements are typically investigated through DNA hypersensitivity assays,
    assays of DNA methylation, and immunoprecipitation (IP) of proteins that
    interact with DNA and RNA, i.e., modified histones, transcription factors,
    chromatin regulators, and RNA-binding proteins, followed by sequencing.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black;background:yellow'>Usage
    Examples</span></b><b><span style='font-family:"Times New Roman",serif;
    color:black'>/<span style='background:yellow'>Tutorials</span></span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black;background:yellow'>Exploring
    ENCODE data from EC2 with Jupyter notebook</span></b><b><span
    style='font-family:"Times New Roman",serif;color:black'>  </span></b><span
    style='font-family:"Times New Roman",serif;color:black;background:yellow'>Excellent
    Detailed instructions</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/<b><span
    style='background:yellow'>mount_s3_bucket_and_run_jupyter_on_ec2.ipynb</span></b></span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    notebook demonstrates how to mount s3://encode-public on an EC2 instance
    using Goofys. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Goofys
    makes an S3 bucket appear as a typical UNIX file system, and is useful for
    tools that expect a local file path. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Once
    the bucket is mounted we can launch a Jupyter notebook on the instance and
    connect to it remotely. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    benefit of using EC2 is that the compute is scalable to the analysis you
    would like to perform, and you don't have to download anything locally.<br>
    <br>
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ingesting
    ENCODE data into TileDB with S3 backend </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/<b><span
    style='background:yellow'>ingest_encode_data_tile_db_with_s3_backend.ipynb</span></b></span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Related
    Publication:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black;background:yellow'>New
    developments on the Encyclopedia of DNA Elements (ENCODE) data portal</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nucleic
    Acids Research, Volume 48, Issue D1, 08 January 2020, Pages D882–D889</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>12
    November 2019</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Genome Aggregation
    Database (gnomAD)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://registry.opendata.aws/broad-gnomad/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://registry.opendata.aws/broad-gnomad/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Description</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Genome Aggregation Database (gnomAD) is a resource developed by an
    international coalition of investigators that aggregates and harmonizes
    both exome and genome data from a wide range of large-scale human
    sequencing projects. The summary data provided here are released for the
    benefit of the wider scientific community without restriction on use. The
    v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome
    sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702
    genomes, selected as in v2. Sign up for the gnomAD mailing list here.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Resources
    on AWS</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Description</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    gnomAD summary data aggregated from large-scale human genome and exome
    sequencing projects.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Resource type</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    S3 Bucket</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    <b>Amazon Resource Name (ARN)</b></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    <span style='background:yellow'>arn:aws:s3:::gnomad-public-us-east-1</span></span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    AWS Region</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    us-east-1</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    AWS CLI Access (No AWS account required)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    aws s3 ls s3://gnomad-public-us-east-1/ --no-sign-request</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>gnomAD browser</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Downloads</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gnomad.broadinstitute.org/downloads/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    data is available for download through Google Cloud Public Datasets, the
    Registry of Open Data on AWS, and Azure Open Datasets.  We recommend using
    Hail and our Hail utilities for gnomAD to work with the data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    addition to the files listed below, gnomAD variants are available as a
    BigQuery dataset and Terra has a demo workspace for working with gnomAD
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Registry
    of Open Data on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Files
    can be browsed and downloaded using the AWS Command Line Interface.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      <b><span
    style='background:yellow'>aws s3 ls s3://gnomad-public-us-east-1/release/</span></b></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Example
    use of the big data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    structural variation reference for medical and population genetics</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages444–451(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>31
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nature.com/articles/s41586-020-2287-8"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s41586-020-2287-8</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif;color:black'>Data availability</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>All
    gnomAD-SV site-frequency data for appropriately consented samples (<i>n</i>&nbsp;=&nbsp;10,847)
    have been distributed in VCF and BED format via the gnomAD browser (</span><span
    style='color:black'><a href="https://gnomad.broadinstitute.org/downloads/"><span
    style='font-size:10.0pt'>https://gnomad.broadinstitute.org/downloads/</span></a></span><span
    style='font-size:10.0pt;color:black'>), as well as from NCBI dbVar under
    accession nstd166. Furthermore, these SVs have been integrated directly
    into the gnomAD browser</span><span style='color:black'><a
    href="https://www.nature.com/articles/s41586-020-2287-8#ref-CR8"
    title="Karczewski, K. J. et al. The ExAC browser: displaying reference data information from over 60 000 exomes. Nucleic Acids Res. 45 (D1), D840–D845 (2017)."><sup><span
    style='font-size:10.0pt'>8</span></sup></a></span><span style='font-size:
    10.0pt;color:black'>. The architecture of the gnomAD browser is described
    in the main gnomAD study</span><span style='color:black'><a
    href="https://www.nature.com/articles/s41586-020-2287-8#ref-CR4"
    title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature &#10;https://doi.org/10.1038/s41586-020-2308-7&#10;&#10; (2020)."><sup><span
    style='font-size:10.0pt'>4</span></sup></a></span><span style='font-size:
    10.0pt;color:black'>, as well as instructions for how to access and query
    the data hosted therein.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt'>&nbsp;</span></p>
    <p style='margin:0in'><b><span style='font-size:10.0pt;color:black'>Code
    availability</span></b></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    gnomAD-SV discovery pipeline is publicly available via a series of methods
    configured for the FireCloud/Terra platform (</span><span style='color:
    black'><a href="https://portal.firecloud.org/#methods"><span
    style='font-size:10.0pt'>https://portal.firecloud.org/#methods</span></a></span><span
    style='font-size:10.0pt;color:black'>) under the methods namespace
    ‘Talkowski-SV’. The svtk software package used extensively in the gnomAD-SV
    discovery pipeline is publicly available via GitHub (</span><span
    style='color:black'><a href="https://github.com/talkowski-lab/svtk"><span
    style='font-size:10.0pt'>https://github.com/talkowski-lab/svtk</span></a></span><span
    style='font-size:10.0pt;color:black'>). Most custom scripts used in the
    production and/or analysis of the gnomAD-SV dataset are publicly available
    via GitHub (</span><span style='color:black'><a
    href="https://github.com/talkowski-lab/gnomad-sv-pipeline"><span
    style='font-size:10.0pt'>https://github.com/talkowski-lab/gnomad-sv-pipeline</span></a></span><span
    style='font-size:10.0pt;color:black'>). All code is made available under
    the MIT license, unless stated otherwise.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    browser</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Downloads</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gnomad.broadinstitute.org/downloads/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    data is available for download through Google Cloud Public Datasets, the
    Registry of Open Data on AWS, and Azure Open Datasets.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>We
    recommend using Hail and our Hail utilities for gnomAD to work with the
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    addition to the files listed below, gnomAD variants are available as a
    BigQuery dataset and Terra has a demo workspace for working with gnomAD
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Registry
    of Open Data on AWS</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Files
    can be browsed and downloaded using the AWS Command Line Interface.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    background:yellow'>aws s3 ls s3://gnomad-public-us-east-1/release/</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>All
    chromosomes sites VCF</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Download
    Google / Amazon / Microsoft</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Exomes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr1
    sites VCF      5.77 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr2
    sites VCF      4.2 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr3
    sites VCF      3.29 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr4
    sites VCF      2.17 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr5
    sites VCF      2.51 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr6
    sites VCF      2.83 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr7
    sites VCF      2.88 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr8
    sites VCF      2.13 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr9
    sites VCF      2.4 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr10
    sites VCF   2.23 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr11
    sites VCF   3.61 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr12
    sites VCF   3.07 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr13
    sites VCF   976.73 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr14
    sites VCF   2.02 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr15
    sites VCF   2.08 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr16
    sites VCF   3.04 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr17
    sites VCF   3.62 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr18
    sites VCF   882.14 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr19
    sites VCF   4.3 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr20
    sites VCF   1.44 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr21
    sites VCF   652.73 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr22
    sites VCF   1.43 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chrX
    sites VCF     1.33 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chrY
    sites VCF     15.66 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genomes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Constraint</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-constraint"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-constraint</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    mutational constraint spectrum quantified from variation in 141,456 humans.
    Nature 581, 434–443 (2020).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Multi-nucleotide
    variants (MNVs)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-multi-nucleotide-variants"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-multi-nucleotide-variants</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Landscape
    of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes.
    Nature Communications 11, 2539 (2020).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Proportion
    expressed across transcripts (pext)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-pext"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-pext</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Transcript
    expression-aware annotation improves rare variant interpretation. Nature
    581, 452–458 (2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Related
    Publications</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    mutational constraint spectrum quantified from variation in 141,456 humans</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nature.com/articles/s41586-020-2308-7"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s41586-020-2308-7</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages434–443(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>300
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Transcript
    expression-aware annotation improves rare variant interpretation</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nature.com/articles/s41586-020-2329-2"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s41586-020-2329-2</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Thousands
    of human sequences provide deep insight into single genomes </span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    May 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    massive genome-sequencing and analysis effort has produced the most
    comprehensive sets of data and tools for understanding human genetic
    variation so far. The resource will be invaluable to biologists of every
    stripe. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    structural variation reference for medical and population genetics</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://pubmed.ncbi.nlm.nih.gov/32461652/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://pubmed.ncbi.nlm.nih.gov/32461652/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages444–451(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>31
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Using Amazon EC2 to run large
  data analyses <span style='color:black'>cheaply<br>
  </span></span></b><b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
  color:black'>Spot instances on Amazon Elastic Compute Cloud (EC2) </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Jul
  5, 2018 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://gex.netlify.app/post/using-amazon-ec2-to-run-large-data-analysis-cheaply/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gex.netlify.app/post/using-amazon-ec2-to-run-large-data-analysis-cheaply/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Spot instances
  on Amazon Elastic Compute Cloud (EC2)</span></b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'> allow researchers to do high
  performance computing at very low cost. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>For example, a
  64-core workstation with 256 GB of memory can be rented at about $0.7 per
  hour</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Main topics
  covered this tutorial</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Set up Key
  Pairs and Security Groups to enable SHH login,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Request spot
  instance (virtual machine),</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    SHH access
  to the instance via Putty and Filezilla,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Install
  Docker software,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Build a
  Docker image based on the Bioconductor Docker definition files,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Start R and
  compute from within the container,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Create a
  “volume” (a virtual hard drive) and attach it to running instances,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Take a
  snapshot of a volume, copy it across regions, and use it to create a volume,</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Google Compute
  Engine set-up.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Start the Spot
  Advisor and enter your request. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:#C00000'>Change
  the amount required to 1. Otherwise, 20 instances will be requested</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;background:yellow'>A
  64-core instance with 256GB memory costs $0.67 per hour or about $16 per day.</span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'> </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Note that Amazon
  has web servers all across the world. You can switch regions from the top
  right of the screen. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Prices and
  availability of resources vary greatly across the region. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>What is Packer?</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.packer.io/"><b><span
  style='font-family:"Times New Roman",serif'>https://www.packer.io/</span></b></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Packer is an
  open source tool for creating identical machine images for multiple platforms
  from a single source configuration. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Packer is
  lightweight, runs on every major operating system, and is highly performant,
  creating machine images for multiple platforms in parallel. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Packer does not
  replace configuration management like Chef or Puppet. In fact, when building
  images, Packer is able to use tools like Chef or Puppet to install software
  onto the image.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>A machine image
  is a single static unit that contains a pre-configured operating system and
  installed software which is used to quickly create new running machines. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Machine image
  formats change for each platform. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Some examples
  include AMIs for EC2, VMDK/VMX files for VMware, OVF exports for VirtualBox,
  etc.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Cosine
  Similarity support in Amazon Elasticsearch Service </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Posted On: Jul
  29, 2020 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
  Elasticsearch Service now supports cosine similarity distance metric with
  k-Nearest Neighbor (k-NN) to power your similarity search engine. Cosine
  similarity is used to measure similarities between two vectors, irrespective
  of their sizes and is most commonly used in information retrieval, image
  recognition, text similarity, bioinformatics and recommendation systems. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Reproducible and
  robust workshop materials</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Workshops were
  authored using R Markdown, and compiled into a book (PDF and ePub) and
  website using Bookdown R package. Bookdown, in turn, uses the gitbook
  publishing system ( https://www.gitbook.com/) to produce a variety of formats
  from the same source material. R Markdown files intended to be part of a
  Bookdown project do not contain the required front matter of a typical
  stand-alone R Markdown document. To help authors use and test the correct
  format, we seeded each workshop document with the syllabus that had been
  submitted by that author, and successfully built the book of the submitted
  syllabi. Each workshop represented a chapter of a book compiled using the
  Bookdown software. This approach provided several advantages:</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>•     R markdown
  syntax is already familiar to any developer of a Bioconductor package, since
  it is the standard approach to creating the     package “vignette” or prose
  documentation.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  •     R markdown implements “literate programming” by including formatted
  text, runnable code, and output of the code</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  •     Bookdown allows collating chapters as a clean, lightweight online book
  format, and pandoc additionally allows creation of PDF and   ePub formats</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  •     These formats can then be self-published with options to order paper
  copies through companies such as </span><a href="https://leanpub.com"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://leanpub.com</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>bookdown:
Authoring Books and Technical Documents with R Markdown</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://books.google.com/books?hl=en&amp;lr=&amp;id=_LrZDQAAQBAJ&amp;oi=fnd&amp;pg=PT12&amp;dq=+Xie+Y+:+bookdown:+Authoring+Books+and+Technical+Documents+with+R+Markdown+%5BInternet%5D+.+Boca+Raton,+Florida:+Chapman+and+Hall/CRC%3B+2016++10.1201/9781315204963++&amp;ots=tz1AhTAN96&amp;sig=mmTuwDYYrl3sVm8SjtJD8tvnwd0#v=onepage&amp;q&amp;f=false"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://books.google.com/books?hl=en&amp;lr=&amp;id=_LrZDQAAQBAJ&amp;oi=fnd&amp;pg=PT12&amp;dq=+Xie+Y+:+bookdown:+Authoring+Books+and+Technical+Documents+with+R+Markdown+%5BInternet%5D+.+Boca+Raton,+Florida:+Chapman+and+Hall/CRC%3B+2016++10.1201/9781315204963++&amp;ots=tz1AhTAN96&amp;sig=mmTuwDYYrl3sVm8SjtJD8tvnwd0#v=onepage&amp;q&amp;f=false</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
name="_Hlk57394992"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>DevOps Pipeline
  Experience</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>NGC Virtual
  envir intended for DevOps Pipeline </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>GitLab, Docker,
  Nexus, Jenkins --&gt; Deployment Env</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Git Local --&gt;
  GitLab Remote --&gt;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  Docker </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Docker Local
  --&gt; Docker Hub --&gt; Docker Hub Nexus </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Dockerfile YAML</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Jenkins </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Jenkinsfile YAML</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-family:"Times New Roman",serif'>Google Cloud Computing</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Google Cloud
  Platform</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Google
  Kubernetes Engine </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://cloud.google.com/kubernetes-engine/"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>https://cloud.google.com/kubernetes-engine/</span></a><span
  class=MsoHyperlink><span style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  </span></span><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Secured
  and managed Kubernetes service with four-way auto scaling and multi-cluster
  support.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>New customers
  get $300 in free credits to spend on Google Cloud during the first 90 days. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>All customers
  get one zonal cluster per month for free, not charged against your credits.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Types of
  clusters</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://cloud.google.com/kubernetes-engine/docs/concepts/types-of-clusters</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Google Domain
  Name</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>A domain name
  through a domain name registrar. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>You can register
  a domain name through Google Domains or another domain registrar of your
  choice. </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>An IP address to
  point the A record of your zone.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>PuTTy</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>putty-64bit-0.74-installer.msi</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><img border=0
width=510 height=391 id="Picture 21" src="AWS-Bioinformatics_files/image009.png"></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>C:\Program
Files\PuTTY\  putty.exe</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>                                                   puttygen.exe
</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-family:"Times New Roman",serif'>AWS EC2 Free-Tier</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><img border=0
width=702 height=496 id="Picture 23" src="AWS-Bioinformatics_files/image010.jpg"></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

</div>

</body>

</html>
