<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Mangal;
	panose-1:2 4 5 3 5 2 3 3 2 2;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Times New Roman",serif;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;
	font-style:italic;}
h5
	{mso-style-link:"Heading 5 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoBodyText, li.MsoBodyText, div.MsoBodyText
	{mso-style-link:"Body Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:6.0pt;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
code
	{font-family:"Courier New";}
pre
	{mso-style-link:"HTML Preformatted Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Courier New";}
tt
	{font-family:"Courier New";}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.UnresolvedMention1
	{mso-style-name:"Unresolved Mention1";
	color:#605E5C;
	background:#E1DFDD;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Times New Roman",serif;
	font-weight:bold;}
span.mw-headline
	{mso-style-name:mw-headline;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-style:italic;}
span.a-declarative
	{mso-style-name:a-declarative;}
span.a-icon-alt
	{mso-style-name:a-icon-alt;}
span.a-size-small
	{mso-style-name:a-size-small;}
p.zghrsritem, li.zghrsritem, div.zghrsritem
	{mso-style-name:zg_hrsr_item;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.zghrsrrank
	{mso-style-name:zg_hrsr_rank;}
span.zghrsrladder
	{mso-style-name:zg_hrsr_ladder;}
span.HTMLPreformattedChar
	{mso-style-name:"HTML Preformatted Char";
	mso-style-link:"HTML Preformatted";
	font-family:"Courier New";}
span.BodyTextChar
	{mso-style-name:"Body Text Char";
	mso-style-link:"Body Text";
	font-family:"Times New Roman",serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;}
span.job-id
	{mso-style-name:job-id;}
span.visually-hidden
	{mso-style-name:visually-hidden;}
span.jobsearch-jobinfoheader-bullet
	{mso-style-name:jobsearch-jobinfoheader-bullet;}
span.icl-u-xs-mr--xs
	{mso-style-name:icl-u-xs-mr--xs;}
span.jobsearch-jobmetadataheader-item
	{mso-style-name:jobsearch-jobmetadataheader-item;}
span.Heading5Char
	{mso-style-name:"Heading 5 Char";
	mso-style-link:"Heading 5";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.topiccitationitalics
	{mso-style-name:topiccitationitalics;}
span.bold
	{mso-style-name:bold;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.st
	{mso-style-name:st;}
span.treelabel
	{mso-style-name:tree_label;}
p.msonormal0, li.msonormal0, div.msonormal0
	{mso-style-name:msonormal;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.token
	{mso-style-name:token;}
span.content-section
	{mso-style-name:content-section;}
span.label
	{mso-style-name:label;}
p.chapter-para, li.chapter-para, div.chapter-para
	{mso-style-name:chapter-para;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.underline
	{mso-style-name:underline;}
span.pre
	{mso-style-name:pre;}
span.caps
	{mso-style-name:caps;}
span.Date1
	{mso-style-name:Date1;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:8.5in 11.0in;
	margin:.3in .3in .3in .5in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:12.0pt;font-family:"Times New Roman",serif'>Data Resources,
Viewers</span></b><b><span style='font-family:"Times New Roman",serif'> <br>
(navigation, search, content interpretation, data accession)<br>
<br>
</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Publicly Available Genomic Data
  Resources, Viewers</span></b><span style='font-family:"Times New Roman",serif;
  color:black'> <br>
  Explore Content, Search</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (GDV)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.38"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.38</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    NCBI Genome Data Viewer (GDV) is a genome browser supporting the
    exploration and analysis of eukaryotic RefSeq genome assemblies. Genome
    Data Viewer is also used by different NCBI resources, such as GEO and
    dbGaP, to display datasets associated with specified experiments or samples
    in a genome browser context. <br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Tutorials</span></b></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (Tutorial Page Functionality)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/help/#LAYOUT"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genome/gdv/browser/help/#LAYOUT</span></a></span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (Tutorial 11 videos Last updated Sep 23, 2020)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.youtube.com/playlist?list=PLH-TjWpFfWruHgL0WRzZfQwp-MWzhIj16"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/playlist?list=PLH-TjWpFfWruHgL0WRzZfQwp-MWzhIj16</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Gene Expression
    Omnibus (GEO)  <br>
    </span></b><span style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO
    is a public functional genomics data repository supporting MIAME-compliant
    data submissions.<br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>GEO DataSets (GDS)</span></b></p>
    <p class=MsoNormal style='margin-bottom:8.0pt;line-height:107%'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/gds/"><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/gds/</span></a></span><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif;
    color:black'><br>
    This database stores curated gene expression DataSets, as well as original
    Series and Platform records in the Gene Expression Omnibus (GEO)
    repository. Enter search terms to locate experiments of interest. DataSet
    records contain additional resources including cluster tools and
    differential expression queries.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>About
    GEO DataSets<br>
    </span></b><span style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/info/datasets.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/info/datasets.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO
    Profiles</span></b></p>
    <p class=MsoNormal style='margin-bottom:8.0pt;line-height:107%'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/geoprofiles/"><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geoprofiles/</span></a></span><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif;
    color:black'><br>
    This database stores individual gene expression profiles from curated
    DataSets in the Gene Expression Omnibus (GEO) repository. Search for
    specific profiles of interest based on gene annotation or pre-computed
    profile characteristics. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Querying
    GEO DataSets and GEO Profiles</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/info/qqtutorial.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/info/qqtutorial.html</span></a></span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    6.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Related Publications</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Gene Expression Omnibus database<br>
    Methods Mol Biol. 2016; 1418: 93–110.</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944384/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944384/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCBI
    GEO: mining millions of expression profiles database and tools<br>
    Nucleic Acids Res. 2005 Jan 1</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539976/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539976/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCBI
    Sequence Read Archive (SRA) </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi</span></a></span><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>?</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>SRA
    is NIH primary archive of high-throughput sequencing data and is part of
    the international partnership of archives (INSDC).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Stores
    raw sequence data from &quot;next-generation&quot; sequencing technologies
    including <br>
    Illumina, 454, IonTorrent, Complete Genomics, PacBio and OxfordNanopores. <br>
    In addition to raw sequence data, SRA now stores alignment information in
    the form of read placements on a reference sequence. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>SRA
    Toolkit</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/ncbi/sra-tools/wiki/02.-Installing-SRA-Toolkit"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://github.com/ncbi/sra-tools/wiki/02.-Installing-SRA-Toolkit</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    SRA Toolkit provides 64-bit binary installations for Linux distributions,
    Mac OS X, Windows.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif;color:black'><br>
    </span><b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
    color:black'>SRA Explorer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>tool
    aims to make datasets within the Sequence Read Archive more accessible.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://sra-explorer.info/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://sra-explorer.info/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>ENCODE:
    Encyclopedia of DNA Elements<br>
    </span></b><span style='color:black'><a
    href="https://www.encodeproject.org/"><span style='font-size:8.0pt;
    font-family:"Times New Roman",serif'>https://www.encodeproject.org/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Project </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://www.1000genomes.org"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://www.1000genomes.org</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Project FAQ</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>If
    you have any other questions you can’t find the answer to please email </span><span
    style='color:black'><a href="mailto:info@1000genomes.org"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>info@1000genomes.org</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.internationalgenome.org/faq"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/faq</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGSR:
    The International Genome Sample Resource</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Open
    human variation data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.internationalgenome.org/home"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/home</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes 30x on GRCh38</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.internationalgenome.org/data-portal/data-collection/30x-grch38"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/data-portal/data-collection/30x-grch38</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Browser (Phase 3)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ensembl</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://uswest.ensembl.org/Homo_sapiens/Info/Index"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://uswest.ensembl.org/Homo_sapiens/Info/Index</span></a></span><span
    class=MsoHyperlink><span style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
    <br>
    </span></span></p>
    <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
     style='background:white;border-collapse:collapse;border:none'>
     <tr>
      <td width=737 valign=top style='width:552.8pt;border:solid windowtext 1.0pt;
      padding:0in 5.4pt 0in 5.4pt'>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Data
      Portal (Search by Population, Technology, Data Collection)</span></b></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='color:black'><a
      href="https://www.internationalgenome.org/data-portal/sample"><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/data-portal/sample</span></a></span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  
      Technology</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Exome</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Low
      coverage WGS </span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Integrated
      variant call sets</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  
      File Format</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      FASTQ</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      VCF</span></p>
      <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
      style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
      </td>
     </tr>
    </table>
    <p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;
    font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>dbGaP</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>db
    Genotypes and Phenotypes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://dbgap.ncbi.nlm.nih.gov/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://dbgap.ncbi.nlm.nih.gov/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCBI
    dbSNP database</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/snp/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/snp/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>dbSNP
    contains human single nucleotide variations, microsatellites, and
    small-scale insertions and deletions along with publication, population
    frequency, molecular consequence, and genomic and RefSeq mapping
    information for both common variations and clinical mutations.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>dbSNP
    Data Access:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/snp/docs/RefSNP_about/#data-access"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/snp/docs/RefSNP_about/#data-access</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RefSNP
    data, including genotype, frequency and associated metadata, are available
    without restrictions on the web, FTP, and API.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    dbSNP Overview:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/NBK21088/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/NBK21088/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    dbSNP Tutorials on GitHub:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/snp/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/snp/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NIH
    Data Sharing Repositories/</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Open
    Domain-Specific Data Sharing Repositories</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nlm.nih.gov/NIHbmic/domain_specific_repositories.html"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.nlm.nih.gov/NIHbmic/domain_specific_repositories.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GenBank
    </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/genbank/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genbank/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GenBank
    is the NIH genetic sequence database, an annotated collection of all
    publicly available DNA sequences.<br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GenBank
    is part of the <b>International Nucleotide Sequence Database Collaboration</b>,
    which comprises the DNA DataBank of </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Japan
    (DDBJ), European Nucleotide Archive (ENA), GenBank at NCBI. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ways
    to search and retrieve data from GenBank:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>
    Search GenBank for sequence identifiers and annotations with Entrez
    Nucleotide.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>
    Search and align GenBank sequences to a query sequence using BLAST (Basic
    Local Alignment Search Tool). </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  
    BLAST searches CoreNucleotide, dbEST, and dbGSS independently.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> Search,
    link, and download sequences programatically using NCBI e-utilities.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCI
    / Genetic Data Commons (GDC)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gdc.cancer.gov/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gdc.cancer.gov/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://portal.gdc.cancer.gov/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://portal.gdc.cancer.gov/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GDC
    Data Transfer Tool: An Overview </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Getting_Started/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Getting_Started/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Raw
    sequence data, stored as BAM files, make up the bulk of data stored at the
    NCI Genomic Data Commons (GDC). </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GDC
    Exploration</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://docs.gdc.cancer.gov/Data_Portal/Users_Guide/Exploration/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://docs.gdc.cancer.gov/Data_Portal/Users_Guide/Exploration/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Exploration Page allows users to explore data in the GDC using advanced
    filters/facets, which includes those on a gene and mutation level. Users
    choose filters on specific Cases, Genes, and/or Mutations on the left of
    this page and then can visualize these results on the right. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GDC
    Data Transfer Tool </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://docs.gdc.cancer.gov/Data_Transfer_Tool/Users_Guide/Preparing_for_Data_Download_and_Upload/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>is
    intended to be used in conjunction with the GDC Data Portal and the GDC
    Data Submission Portal to transfer data to or from the GDC. The GDC Data
    Portal's interface is used to generate a manifest file or obtain UUID(s)
    and (for Controlled-Access Data) an authentication token. The GDC Data
    Transfer Tool is then used to transfer the data files listed in the
    manifest file or identified by UUID(s).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GDC/TCGA
    - Genomic Data Commons (GDC) / The Cancer Genome Atlas (TCGA)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://tcga-data.nci.nih.gov/tcga/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://tcga-data.nci.nih.gov/tcga/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Harmonized
    Cancer Datasets<br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    order to download data from TCGA data portal:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>1.
    Connect to </span><span style='color:black'><a
    href="https://tcga-data.nci.nih.gov/tcga/"><span style='font-size:8.0pt;
    font-family:"Times New Roman",serif'>https://tcga-data.nci.nih.gov/tcga/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>2.
    Select the cancer subtype you are interested in (i.e breast invasive
    carcinoma)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>3.
    Select mRNA</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>4.
    Now you can see a table where rows are representing different patients.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>5.
    If present select the column (by clicking on header) that referse to RNASeq
    or RNASeqV2 if it is present for that </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>cancer
    subtype and then click BUILD archive.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>6.
    Keep in mind that just below the header there is a number indicating the
    respective data level. Levels 1-4 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://wiki.nci.nih.gov/display/TCGA/Data+level"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://wiki.nci.nih.gov/display/TCGA/Data+level</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>If
    you need RAW data such as FASTQ files you have find level 1 data, but often
    this kind of data is not publicly </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>available
    on TCGA and you might need to ask for permission in order to download it.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Marco
    has listed the steps to access open tier data on TCGA. In case you are
    interested in accessing lower level data, </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>such
    as raw bam files for rna seq samples, you can apply for the access here </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?login=&amp;page=login"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://dbgap.ncbi.nlm.nih.gov/aa/wga.cgi?login=&amp;page=login</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GDC
    for TCGA Data Access Matrix Users</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gdc.cancer.gov/gdc-tcga-data-access-matrix-users"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gdc.cancer.gov/gdc-tcga-data-access-matrix-users</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Resources
    for TCGA Users</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gdc.cancer.gov/resources-tcga-users"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gdc.cancer.gov/resources-tcga-users</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Cancer Genome Atlas (TCGA)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>UCSC
    Genome Bioinformatics Group</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://genome-euro.ucsc.edu/index.html"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://genome-euro.ucsc.edu/index.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGSR</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>:
    The International Genome Sample Resource</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Supporting
    open human variation data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.internationalgenome.org/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>AnVIL</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> <br>
    supports the management, analysis and sharing of human disease data for the
    research community</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://anvilproject.org/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://anvilproject.org/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gen3.theanvil.io/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gen3.theanvil.io/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gen3.theanvil.io/explorer"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gen3.theanvil.io/explorer</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>REST
    APIs<br>
    </span></b><a name="_Hlk52360649"></a><a href="http://MyGene.info"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://</span><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>MyGene.info</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://MyVariant.info"><span style='font-size:
    8.0pt;font-family:"Times New Roman",serif'>http://MyVariant.info</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://BioThings.io"><span style='font-size:
    8.0pt;font-family:"Times New Roman",serif'>http://BioThings.io</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://data.cvisb.org/home"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://data.cvisb.org/home</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://Smart-API.info"><span style='font-size:
    8.0pt;font-family:"Times New Roman",serif'>http://Smart-API.info</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
<br>
</span><a name="_Hlk59196085"></a></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:white;border-collapse:collapse;border:none'>
 <tr>
  <td width=759 valign=top style='width:569.6pt;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Google</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif;color:black'>how to analyze geo
  datasets in r</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.google.com/search?lr=&amp;as_qdr=all&amp;sxsrf=ALeKk03S9orXF4hAyQbXO282dDPFBUHzAA:1609876679459&amp;q=how+to+analyze+geo+datasets+in+r&amp;sa=X&amp;ved=2ahUKEwjyuvH8yYXuAhUOQ80KHZVNBEgQ1QIoAHoECAgQAQ"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?lr=&amp;as_qdr=all&amp;sxsrf=ALeKk03S9orXF4hAyQbXO282dDPFBUHzAA:1609876679459&amp;q=how+to+analyze+geo+datasets+in+r&amp;sa=X&amp;ved=2ahUKEwjyuvH8yYXuAhUOQ80KHZVNBEgQ1QIoAHoECAgQAQ</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
 <tr>
  <td width=759 valign=top style='width:569.6pt;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <b>Exploratory analysis of datasets obtained from GEO</b></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="http://www.biotechworld.it/bioinf/2015/10/22/exploratory-analysis-of-datasets-obtained-from-geo/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.biotechworld.it/bioinf/2015/10/22/exploratory-analysis-of-datasets-obtained-from-geo/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>bioinformatics
  &gt; Exploratory analysis of datasets obtained from GEO</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>October
  22, 2015 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Using
  the GEOquery Package</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bioconductor.org/packages/release/bioc/vignettes/GEOquery/inst/doc/GEOquery.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioconductor.org/packages/release/bioc/vignettes/GEOquery/inst/doc/GEOquery.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>September
  21, 2014</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Retrieve
  and analyze a gene expression data set from NCBI GEO in R</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=gC-WuK4BbJY"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=gC-WuK4BbJY</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nov
  2, 2012</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Analysing
  data from GEO - Work in Progress</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://sbc.shef.ac.uk/geo_tutorial/tutorial.nb.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://sbc.shef.ac.uk/geo_tutorial/tutorial.nb.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>30
  Jun 2020<br>
  <br>
  </span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-family:"Times New Roman",serif;color:white'><br>
<br>
</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:12.0pt;font-family:"Times New Roman",serif'>Registry of Open
  Data on AWS</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://registry.opendata.aws/"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://registry.opendata.aws/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h4 style='line-height:normal'>currently 205 datasets</h4>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif;color:black;background:yellow'>See</span></b><b><span
  style='font-family:"Times New Roman",serif;color:black'> </span></b><a
  href="https://registry.opendata.aws/usage-examples"><b><span
  style='font-family:"Times New Roman",serif;color:black;background:yellow;
  text-decoration:none'> Usage Examples</span></b><b><span style='font-family:
  "Times New Roman",serif;color:black;text-decoration:none'>/<span
  style='background:yellow'>Tutorials</span> for datasets listed in this registry</span></b></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Encyclopedia of DNA
    Elements (ENCODE)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://registry.opendata.aws/encode-project/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://registry.opendata.aws/encode-project/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Description</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration
    of research groups funded by the National Human Genome Research Institute
    (NHGRI). The goal of ENCODE is to build a comprehensive parts list of
    functional elements in the human genome, including elements that act at the
    protein and RNA levels, and regulatory elements that control cells and
    circumstances in which a gene is active. ENCODE investigators employ a
    variety of assays and methods to identify functional elements. The
    discovery and annotation of gene elements is accomplished primarily by
    sequencing a diverse range of RNA sources, comparative genomics,
    integrative bioinformatic methods, and human curation. Regulatory elements
    are typically investigated through DNA hypersensitivity assays, assays of
    DNA methylation, and immunoprecipitation (IP) of proteins that interact
    with DNA and RNA, i.e., modified histones, transcription factors, chromatin
    regulators, and RNA-binding proteins, followed by sequencing.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black;background:yellow'>Usage
    Examples</span></b><b><span style='font-family:"Times New Roman",serif;
    color:black'>/<span style='background:yellow'>Tutorials</span></span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Exploring ENCODE
    data from EC2 with Jupyter notebook  </span></b><span style='font-family:
    "Times New Roman",serif;color:black;background:yellow'>Excellent Detailed
    instructions</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/mount_s3_bucket_and_run_jupyter_on_ec2.ipynb</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    notebook demonstrates how to mount s3://encode-public on an EC2 instance
    using Goofys. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Goofys
    makes an S3 bucket appear as a typical UNIX file system, and is useful for
    tools that expect a local file path. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Once
    the bucket is mounted we can launch a Jupyter notebook on the instance and
    connect to it remotely. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    benefit of using EC2 is that the compute is scalable to the analysis you
    would like to perform, and you don't have to download anything locally.<br>
    <br>
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ingesting
    ENCODE data into TileDB with S3 backend </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/ENCODE-DCC/encode-data-usage-examples/blob/master/ingest_encode_data_tile_db_with_s3_backend.ipynb</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Publications</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>New
    developments on the Encyclopedia of DNA Elements (ENCODE) data portal</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://academic.oup.com/nar/article/48/D1/D882/5622708#190992715</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nucleic
    Acids Research, Volume 48, Issue D1, 08 January 2020, Pages D882–D889</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>12
    November 2019</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Genome Aggregation
    Database (gnomAD)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://registry.opendata.aws/broad-gnomad/"><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>https://registry.opendata.aws/broad-gnomad/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Description</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Genome Aggregation Database (gnomAD) is a resource developed by an
    international coalition of investigators that aggregates and harmonizes
    both exome and genome data from a wide range of large-scale human
    sequencing projects. The summary data provided here are released for the
    benefit of the wider scientific community without restriction on use. The
    v2 data set (GRCh37) spans 125,748 exome sequences and 15,708 whole-genome
    sequences from unrelated individuals. The v3 data set (GRCh38) spans 71,702
    genomes, selected as in v2. Sign up for the gnomAD mailing list here.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Resources
    on AWS</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Description</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    gnomAD summary data aggregated from large-scale human genome and exome
    sequencing projects.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Resource type</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    S3 Bucket</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    <b>Amazon Resource Name (ARN)</b></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    <span style='background:yellow'>arn:aws:s3:::gnomad-public-us-east-1</span></span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    AWS Region</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    us-east-1</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    AWS CLI Access (No AWS account required)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    aws s3 ls s3://gnomad-public-us-east-1/ --no-sign-request</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>gnomAD browser</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Downloads</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://gnomad.broadinstitute.org/downloads/"><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    data is available for download through Google Cloud Public Datasets, the
    Registry of Open Data on AWS, and Azure Open Datasets.  We recommend using
    Hail and our Hail utilities for gnomAD to work with the data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    addition to the files listed below, gnomAD variants are available as a
    BigQuery dataset and Terra has a demo workspace for working with gnomAD
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Registry
    of Open Data on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Files
    can be browsed and downloaded using the AWS Command Line Interface.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      <b><span
    style='background:yellow'>aws s3 ls s3://gnomad-public-us-east-1/release/</span></b></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Example
    use of the big data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    structural variation reference for medical and population genetics</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages444–451(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>31
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://www.nature.com/articles/s41586-020-2287-8"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.nature.com/articles/s41586-020-2287-8</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif;color:black'>Data availability</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>All
    gnomAD-SV site-frequency data for appropriately consented samples (<i>n</i>&nbsp;=&nbsp;10,847)
    have been distributed in VCF and BED format via the gnomAD browser (</span><a
    href="https://gnomad.broadinstitute.org/downloads/"><span style='font-size:
    10.0pt'>https://gnomad.broadinstitute.org/downloads/</span></a><span
    style='font-size:10.0pt;color:black'>), as well as from NCBI dbVar under
    accession nstd166. Furthermore, these SVs have been integrated directly
    into the gnomAD browser</span><a
    href="https://www.nature.com/articles/s41586-020-2287-8#ref-CR8"
    title="Karczewski, K. J. et al. The ExAC browser: displaying reference data information from over 60 000 exomes. Nucleic Acids Res. 45 (D1), D840–D845 (2017)."><sup><span
    style='font-size:10.0pt'>8</span></sup></a><span style='font-size:10.0pt;
    color:black'>. The architecture of the gnomAD browser is described in the
    main gnomAD study</span><a
    href="https://www.nature.com/articles/s41586-020-2287-8#ref-CR4"
    title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature &#10;https://doi.org/10.1038/s41586-020-2308-7&#10;&#10; (2020)."><sup><span
    style='font-size:10.0pt'>4</span></sup></a><span style='font-size:10.0pt;
    color:black'>, as well as instructions for how to access and query the data
    hosted therein.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt'>&nbsp;</span></p>
    <p style='margin:0in'><b><span style='font-size:10.0pt;color:black'>Code
    availability</span></b></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    gnomAD-SV discovery pipeline is publicly available via a series of methods
    configured for the FireCloud/Terra platform (</span><a
    href="https://portal.firecloud.org/#methods"><span style='font-size:10.0pt'>https://portal.firecloud.org/#methods</span></a><span
    style='font-size:10.0pt;color:black'>) under the methods namespace
    ‘Talkowski-SV’. The svtk software package used extensively in the gnomAD-SV
    discovery pipeline is publicly available via GitHub (</span><a
    href="https://github.com/talkowski-lab/svtk"><span style='font-size:10.0pt'>https://github.com/talkowski-lab/svtk</span></a><span
    style='font-size:10.0pt;color:black'>). Most custom scripts used in the
    production and/or analysis of the gnomAD-SV dataset are publicly available
    via GitHub (</span><a
    href="https://github.com/talkowski-lab/gnomad-sv-pipeline"><span
    style='font-size:10.0pt'>https://github.com/talkowski-lab/gnomad-sv-pipeline</span></a><span
    style='font-size:10.0pt;color:black'>). All code is made available under
    the MIT license, unless stated otherwise.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    browser</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Downloads</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://gnomad.broadinstitute.org/downloads/"><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>gnomAD
    data is available for download through Google Cloud Public Datasets, the
    Registry of Open Data on AWS, and Azure Open Datasets.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>We
    recommend using Hail and our Hail utilities for gnomAD to work with the
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>In
    addition to the files listed below, gnomAD variants are available as a
    BigQuery dataset and Terra has a demo workspace for working with gnomAD
    data.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Registry
    of Open Data on AWS</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Files
    can be browsed and downloaded using the AWS Command Line Interface.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    background:yellow'>aws s3 ls s3://gnomad-public-us-east-1/release/</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>All
    chromosomes sites VCF</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Download
    Google / Amazon / Microsoft</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Exomes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr1
    sites VCF      5.77 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr2
    sites VCF      4.2 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr3
    sites VCF      3.29 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr4
    sites VCF      2.17 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr5
    sites VCF      2.51 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr6
    sites VCF      2.83 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr7
    sites VCF      2.88 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr8
    sites VCF      2.13 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr9
    sites VCF      2.4 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr10
    sites VCF   2.23 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr11
    sites VCF   3.61 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr12
    sites VCF   3.07 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr13
    sites VCF   976.73 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr14
    sites VCF   2.02 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr15
    sites VCF   2.08 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr16
    sites VCF   3.04 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr17
    sites VCF   3.62 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr18
    sites VCF   882.14 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr19
    sites VCF   4.3 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr20
    sites VCF   1.44 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr21
    sites VCF   652.73 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chr22
    sites VCF   1.43 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chrX
    sites VCF     1.33 GiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>chrY
    sites VCF     15.66 MiB</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genomes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>-</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Constraint</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-constraint"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-constraint</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    mutational constraint spectrum quantified from variation in 141,456 humans.
    Nature 581, 434–443 (2020).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Multi-nucleotide
    variants (MNVs)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-multi-nucleotide-variants"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-multi-nucleotide-variants</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Landscape
    of multi-nucleotide variants in 125,748 human exomes and 15,708 genomes.
    Nature Communications 11, 2539 (2020).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Proportion
    expressed across transcripts (pext)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
    href="https://gnomad.broadinstitute.org/downloads/#v2-pext"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gnomad.broadinstitute.org/downloads/#v2-pext</span></a></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Transcript
    expression-aware annotation improves rare variant interpretation. Nature
    581, 452–458 (2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Related
    Papers</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    mutational constraint spectrum quantified from variation in 141,456 humans</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>https://www.nature.com/articles/s41586-020-2308-7</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages434–443(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>300
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Transcript
    expression-aware annotation improves rare variant interpretation</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>https://www.nature.com/articles/s41586-020-2329-2</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    May 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Thousands
    of human sequences provide deep insight into single genomes</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    massive genome-sequencing and analysis effort has produced the most
    comprehensive sets of data and tools for understanding human genetic
    variation so far. The resource will be invaluable to biologists of every
    stripe. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    structural variation reference for medical and population genetics</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Nature
    volume 581, pages444–451(2020)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>31
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>COVID-19 Data Lake</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>https://registry.opendata.aws/aws-covid19-lake/</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Labs</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://github.com/awslabs"><span style='font-size:10.0pt;font-family:
"Times New Roman",serif'>https://github.com/awslabs</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS Labs Machine
Learning </span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://github.com/awslabs/dgl-ke"><span style='font-size:10.0pt;
font-family:"Times New Roman",serif'>https://github.com/awslabs/dgl-ke</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Medical Data for Machine Learning</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://github.com/beamandrew/medical-data"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/beamandrew/medical-data</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
  is a curated list of medical data for machine learning.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
  list is provided for informational purposes only, please make sure you
  respect any and all usage restrictions for any of the data listed.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>3. Data derived from Electronic
  Health Records (EHRs)</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Building the
  graph of medicine from millions of clinical narratives</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Co-occurence
  statistics for medical terms extracted from 14 million clinical notes and
  260,000 patients.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="http://www.nature.com/articles/sdata201432"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>http://www.nature.com/articles/sdata201432</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="http://datadryad.org/resource/doi:10.5061/dryad.jp917"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://datadryad.org/resource/doi:10.5061/dryad.jp917</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Learning
  Low-Dimensional Representations of Medical Concept</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Low-dimensional
  embeddings of medical concepts constructed using claims data. Note that this
  paper utilizes data from Building the graph of medicine from millions of
  clinical narratives</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="http://cs.nyu.edu/~dsontag/papers/ChoiChiuSontag_AMIA_CRI16.pdf"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://cs.nyu.edu/~dsontag/papers/ChoiChiuSontag_AMIA_CRI16.pdf</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://github.com/clinicalml/embeddings"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://github.com/clinicalml/embeddings</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>MIMIC-III, a
  freely accessible critical care database</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Anonymized
  critical care EHR database on 38,597 patients and 53,423 ICU admissions.
  Requires registration.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="http://www.nature.com/articles/sdata201635"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>http://www.nature.com/articles/sdata201635</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="http://physionet.org/physiobank/database/mimic3cdb/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://physionet.org/physiobank/database/mimic3cdb/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Clinical Concept
  Embeddings Learned from Massive Sources of Medical Data</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Embeddings for
  108,477 medical concepts learned from 60 million patients, 1.7 million
  journal articles, and clinical notes of 20 million patients</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1804.01486"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1804.01486</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Embeddings:  </span><a
  href="https://figshare.com/s/00d69861786cd0156d81"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>https://figshare.com/s/00d69861786cd0156d81</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Interactive
  tool: </span><a href="http://cui2vec.dbmi.hms.harvard.edu"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://cui2vec.dbmi.hms.harvard.edu</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Evaluation of
  Embeddings of Laboratory Test Codes for Patients at a Cancer Center</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>200 dimensional
  Word2Vec embeddings of 1098 laboratory test codes (LOINCs) trained from
  8,280,238 lab orders for 79,081 patients at City of Hope National Medical
  Center (Los Angeles, CA).</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1907.09600"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1907.09600</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Embeddings and
  Code: </span><a
  href="https://github.com/elleros/DSHealth2019_loinc_embeddings"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/elleros/DSHealth2019_loinc_embeddings</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>4. National Healthcare Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Centers for Disease
  Control and Prevention (CDC)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data from the
  CDC on many areas, including:</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>   
  Biomonitoring</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Child
  Vaccinations</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Flu
  Vaccinations</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Health
  Statistics</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Injury &amp;
  Violence</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    MMWR</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Motor
  Vehicle</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    NCHS</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    NNDSS</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Pregnancy
  &amp; Vaccination</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    STDs</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Smoking
  &amp; Tobacco Use</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Teen
  Vaccinations</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Traumatic
  Brain Injury</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Vaccinations</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Web Metrics</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Landing page: </span><a
  href="https://data.cdc.gov"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://data.cdc.gov</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data Catalog: </span><a
  href="https://data.cdc.gov/browse"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>https://data.cdc.gov/browse</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Medicare Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data from the
  Centers for Medicare &amp; Medicaid Services (CMS) on hospitals, nursing
  homes, physicians, home healthcare, dialysis, and device providers.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Landing page: </span><a
  href="https://data.medicare.gov"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>https://data.medicare.gov</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Explorer: </span><a
  href="https://data.medicare.gov/data"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://data.medicare.gov/data</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Texas Public Use
  Inpatient Data File Data on 11 Million inpatient visits with diagnosis,
  procedure codes and outcomes from Texas </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>between 2006
  &amp; 2009.</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Link: </span><a
  href="https://www.dshs.texas.gov/thcic/hospitals/Inpatientpudf.shtm"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.dshs.texas.gov/thcic/hospitals/Inpatientpudf.shtm</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Dollars for
  Doctors</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Propublica
  investigation of money paid by pharmaceutical companies to doctors.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information: </span><a
  href="https://www.propublica.org/series/dollars-for-docs"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.propublica.org/series/dollars-for-docs</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Search tool: </span><a
  href="https://projects.propublica.org/docdollars/"><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>https://projects.propublica.org/docdollars/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data request: </span><a
  href="https://projects.propublica.org/data-store/sets/health-d4d-national-2"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://projects.propublica.org/data-store/sets/health-d4d-national-2</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>DocGraph
  Physician interaction network obtained through a freedom of information act
  request. Covers nearly 1 million entities.</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Main page: </span><a
  href="http://www.docgraph.com"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>http://www.docgraph.com</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information: </span><a
  href="http://thehealthcareblog.com/blog/2012/11/05/tracking-the-social-doctor-opening-up-physician-referral-data-and-much-more/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://thehealthcareblog.com/blog/2012/11/05/tracking-the-social-doctor-opening-up-physician-referral-data-and-much-more/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="http://linea.docgraph.org"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>http://linea.docgraph.org</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>5. UCI Datasets</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Liver Disorders
  Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data on 345
  patients with and without liver disease. Features are 5 blood biomarkers
  thought to be involved with liver disease.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Liver+Disorders"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Liver+Disorders</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Thyroid Disease
  Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Thyroid+Disease"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Thyroid+Disease</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Breast Cancer
  Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Breast+Cancer"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Breast+Cancer</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Heart Disease
  Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Heart+Disease"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Heart+Disease</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Lymphography
  Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Lymphography"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Lymphography</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Parkinsons Data
  Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/parkinsons"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/parkinsons</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Parkinsons
  Telemonitoring Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Parkinsons+Telemonitoring"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Parkinsons+Telemonitoring</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Parkinson Speech
  Dataset with Multiple Types of Sound Recordings Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Parkinson+Speech+Dataset+with++Multiple+Types+of+Sound+Recordings"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Parkinson+Speech+Dataset+with++Multiple+Types+of+Sound+Recordings</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Parkinson's
  Disease Classification Data Set</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/Parkinson%27s+Disease+Classification"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/Parkinson%27s+Disease+Classification</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Primary Tumor
  Dataset Data: </span><a
  href="https://archive.ics.uci.edu/ml/datasets/primary+tumor"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://archive.ics.uci.edu/ml/datasets/primary+tumor</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>6. Biomedical Literature</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>PMC Open Access
  Subset</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Collection of
  all the full-text, open access articles in Pubmed central.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information: </span><a
  href="http://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.ncbi.nlm.nih.gov/pmc/tools/openftlist/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Archived files: </span><a
  href="http://www.ncbi.nlm.nih.gov/pmc/tools/ftp/#Data_Mining"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.ncbi.nlm.nih.gov/pmc/tools/ftp/#Data_Mining</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>PubMed 200k RCT</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Collection of
  pubmed abstracts from randomized control trials (RCTs). Annotations for each
  sentence in the abstract are available.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1710.06071"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1710.06071</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://github.com/Franck-Dernoncourt/pubmed-rct"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/Franck-Dernoncourt/pubmed-rct</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Web API of
  PubMed Articles</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>NLM also provided
  Web API for accessing biomedical literatures in PubMed.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Instructions for
  getting PubMed articles: </span><a
  href="https://www.ncbi.nlm.nih.gov/research/bionlp/APIs/BioC-PubMed/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/research/bionlp/APIs/BioC-PubMed/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>(not full text,
  just title, abstract, etc.)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>For articles in
  PubMed Central, instructions for getting the whole articles:
  https://www.ncbi.nlm.nih.gov/research/bionlp/APIs/BioC-PMC/</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>EBM NLP</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Collection of
  pubmed abstracts from randomized control trials (RCTs). Annotation of
  Population, Intervention, and Outcomes (PICO elements) are available.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1806.04185"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1806.04185</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://ebm-nlp.herokuapp.com/annotations"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://ebm-nlp.herokuapp.com/annotations</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Website: </span><a
  href="https://ebm-nlp.herokuapp.com/index"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://ebm-nlp.herokuapp.com/index</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Evidence
  Inference</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>A dataset for
  inferring the results of randomized control trials (RCTs). A collection of
  pubmed RCTs from the open access subset. Annotations of (intervention,
  comparison intervention, outcome, significance finding, evidence span) are
  available.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1904.01606"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1904.01606</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://github.com/jayded/evidence-inference/tree/master/annotations"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/jayded/evidence-inference/tree/master/annotations</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Website: </span><a
  href="http://evidence-inference.ebm-nlp.com/"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>http://evidence-inference.ebm-nlp.com/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>PubMedQA</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>A dataset for
  biomedical research question answering. Task is to use yes/no/maybe to answer
  naturally occuring questions in PubMed titles.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Paper: </span><a
  href="https://arxiv.org/abs/1909.06146"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://arxiv.org/abs/1909.06146</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data: </span><a
  href="https://github.com/pubmedqa/pubmedqa"><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>https://github.com/pubmedqa/pubmedqa</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Website: </span><a
  href="https://pubmedqa.github.io/"><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>https://pubmedqa.github.io/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>7. TREC Precision Medicine /
  Clinical Decision Support Track</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Text REtrieval
  Conference (TREC) is running a track on Precision Medicine / Clinical
  Decision Support from 2014.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2014 Clinical
  Decision Support Track</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Focus: Retrieval
  of biomedical articles relevant for answering generic clinical questions
  about medical records.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information and
  Data: </span><a href="http://www.trec-cds.org/2014.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.trec-cds.org/2014.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2015 Clinical
  Decision Support Track</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Focus: Retrieval
  of biomedical articles relevant for answering generic clinical questions
  about medical records.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information and
  Data: </span><a href="http://www.trec-cds.org/2015.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.trec-cds.org/2015.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2016 Clinical
  Decision Support Track</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Focus: Retrieval
  of biomedical articles relevant for answering generic clinical questions
  about medical records. Actual electronic health record (EHR) patient records
  are be used instead of synthetic cases.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information and
  Data: </span><a href="http://www.trec-cds.org/2016.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.trec-cds.org/2016.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2017 Clinical
  Decision Support Track</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Focus: Retrieve
  useful precision medicine-related information to clinicians treating cancer
  patients.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Information and
  Data: </span><a href="http://www.trec-cds.org/2017.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://www.trec-cds.org/2017.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>NCI / Genetic Data
Commons (GDC)</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://gdc.cancer.gov/"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://gdc.cancer.gov/</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://portal.gdc.cancer.gov/"><span style='font-size:10.0pt;font-family:
"Times New Roman",serif'>https://portal.gdc.cancer.gov/</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Genome Data Viewer</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.39"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.39</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Variation Viewer</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://www.ncbi.nlm.nih.gov/variation/view/"><span style='font-size:
10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/variation/view/</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>NCBI Sequence
Viewer </span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://www.ncbi.nlm.nih.gov/projects/sviewer/"><span style='font-size:
10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/projects/sviewer/</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
<br>
</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:white;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>NCBI Reference
  Sequence Database <span style='color:black'>     (RefSeq) </span></span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.ncbi.nlm.nih.gov/refseq/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/refseq/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Homo_sapiens/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Homo_sapiens/</span></a></span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Publicly available
data:</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>GEO NCBI, ENA, SRA
</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>other useful data
that can be used in bioinformatic analysis namely; reference genomes,
annotations.</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Testing
differential gene expression</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Whole-genome
bisulfite sequencing data analysis</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Whole-exome
sequencing data analysis</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Whole-genome
sequencing data analysis</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>    Microbiome
data analysis</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

</div>

</body>

</html>
