<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Mangal;
	panose-1:2 4 5 3 5 2 3 3 2 2;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.treelabel
	{mso-style-name:tree_label;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:8.5in 11.0in;
	margin:.3in .3in .3in .5in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:12.0pt;font-family:"Times New Roman",serif'>Data Analysis,
Modeling, Visualization (Software Tools Overview)</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><b><span style='font-family:"Times New Roman",serif'>Categorized
  <span style='color:black'>Bioinformatics Software Tools<br>
  </span></span></b><span style='color:black'><a
  href="https://bioinformaticshome.com/tools/tools-main.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/tools-main.html</span></a></span><span
  class=MsoHyperlink><span style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  <br>
  </span></span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <h2 style='margin-top:6.0pt;line-height:normal'><a
    href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black;
    text-decoration:none'>RNA-seq Pre-analysis Tools</span></b></a></h2>
    <p style='margin:0in'><span style='color:black'><a
    href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html"><span
    style='font-size:8.0pt'>https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html</span></a></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Pre-analysis
    quality control of raw reads includes assessment of tolerable GC and k-mer
    contents, removal of sequence adaptors, PCR artifacts, and contaminations.
    The assessment of duplicates and sequencing errors. In addition, sequencing
    quality tends to decrease towards the 3' end of the reads; Thus, the reads
    must be trimmed to remove the low-quality ends.<br>
    </span><span style='color:black'><a
    href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html"><span
    style='font-size:10.0pt;color:black;text-decoration:none'>Data Quality
    Assessment</span></a></span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a
         href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html#Filtering"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black;text-decoration:none'>Filtering</span></a></span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a
         href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html#Trimming"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black;text-decoration:none'>Trimming</span></a></span><span
         class=treelabel><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> </span></span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a
         href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html#Filtering-and-Trimming"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black;text-decoration:none'>Filtering and Trimming</span></a></span><span
         class=treelabel><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> </span></span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a
         href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html#Reporting-Visualization"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black;text-decoration:none'>Reporting/Visualization</span></a></span><span
         class=treelabel><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> </span></span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a
         href="https://bioinformaticshome.com/tools/rna-seq/pre-analysis.html#Other"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black;text-decoration:none'>Other Pre-analysis RNA-seq Tools</span></a></span><span
         class=treelabel><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> </span></span></li>
    </ul>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <h1 style='margin-top:6.0pt;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif;color:black'>FASTX-Toolkit</span></b></h1>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bioinformaticshome.com/tools/rna-seq/descriptions/FASTX-Toolkit.html"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/rna-seq/descriptions/FASTX-Toolkit.html</span></a></span></p>
    <h1 style='margin-top:6.0pt;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif;color:black'>FastQC</span></b></h1>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bioinformaticshome.com/tools/rna-seq/descriptions/FastQC.html"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/rna-seq/descriptions/FastQC.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>RNA-seq Core Analysis
    Tools </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bioinformaticshome.com/tools/rna-seq/core-analysis.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/rna-seq/core-analysis.html</span></a></span><span
    class=MsoHyperlink><br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.
    Transcriptome Profiling</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1
    Read mapping or assembly</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.1
    De novo (reference free) transcriptome assembly</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.1.1
    Unstranded</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.1.2
    Stranded</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.1.3
    Quality Control</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.2
    Mapping to a reference genome or transcriptome</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.2.1
    Splice Aware</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.2.2
    Splice unaware</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.1.2.3
    Quality Control</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.2
    Expression Quantification</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.2.1
    Union-exon Based</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.2.2
    Transcript Based</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1.2.3
    Bacterial genome</span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>2. Differential Expression
    Analysis</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2.1
    Pre-processing DEA</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2.2
    Parametric</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2.3
    Non-parametric</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2.4
    Power analysis</span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>3. Functional Profiling</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>3.1
    Enrichment Analysis (GSEA), annotation, other</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>3.2
    Comparison with Genome</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    3.0pt;margin-left:0in;line-height:normal'><b><span style='font-family:"Times New Roman",serif;
    color:black'>Whole Genome Assembly (WGA) Analysis Tools</span></b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> -
    Software and Resources<br>
    </span><span style='color:black'><a
    href="https://bioinformaticshome.com/tools/wga/wga.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/wga/wga.html</span></a></span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GAML</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GAML
    is a tool for genome assembly based on maximum likelihood. It implements a
    probabilistic model to take into account sequencing error rates, insert
    lengths and other characteristics to produce a final genome assembly. This
    tool can work on sequenced data generated from multiple sequencing
    platforms (e.g. Illumina, 454, PacBio).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Operation:
    Genome assembly</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Software
    interface: Command-line user interface</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Language:
    -</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Operating
    system: Linux</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>License:
    Not stated</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Cost:
    Free </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bioinformaticshome.com/tools/wga/descriptions/GAML.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/wga/descriptions/GAML.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GAML:
    genome assembly by maximum likelihood</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4454275/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4454275/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Google <span style='color:black'>Searches<br>
  <br>
  </span></span></b></p>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='color:black'><img
  border=0 width=306 height=123 id="Picture 7"
  src="DataAnalysis-Modeling-Visualization-(Software-Tools-Overview)_files/image001.jpg"></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><img border=0 width=305 height=124 id="Picture 8"
  src="DataAnalysis-Modeling-Visualization-(Software-Tools-Overview)_files/image002.jpg"></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><img border=0 width=305 height=120 id="Picture 9"
  src="DataAnalysis-Modeling-Visualization-(Software-Tools-Overview)_files/image003.jpg"></span></p>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><span style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><br>
  </span><b><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
  color:black'> Relevant journal publications </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA-seq
    workflow: gene-level exploratory analysis and differential expression</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2015
    Oct 14</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Abstract</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Here
    we walk through an end-to-end gene-level RNA-Seq differential expression
    workflow using Bioconductor packages. We will start from the FASTQ files,
    show how these were aligned to the reference genome, and prepare a count
    matrix which tallies the number of RNA-seq reads/fragments within each gene
    for each sample. We will perform exploratory data analysis (EDA) for
    quality assessment and to explore the relationship between samples, perform
    differential gene expression analysis, and visually explore the results.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Keywords
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA-seq,
    differential expression, gene expression, Bioconductor, statistical
    analysis, high-throughput sequencing, visualization, genomics</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>RNA-seq: Basic
    Bioinformatics Analysis</span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Curr Protoc Mol Biol. 2018
    Oct</span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6168365/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6168365/</span></a></span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>The workflow includes
    three parts: </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(a) mapping sequencing
    reads to a reference genome or transcriptome; </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(b) quantifying expression
    levels of individual genes and transcripts; </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(c) identifying specific
    genes and transcripts that are differentially expressed between samples.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Alignment/Mapping:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    the sequence of each read to a reference genome, annotation of genes </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>STAR</b> alignment tool </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Quality Assessment:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    After mapping reads to the genome, it is important to survey the quality of
    the RNA-seq data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>Picard</b> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>SAMTools</b> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Quantification of Mapped Reads:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>HTseq</b> to quantify sequencing reads mapped to each gene</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    (a) identify genes that are differentially expressed between conditions
    (sample groups), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    (b) derive gene expression values for each individual transcript</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    Approaches for normalization include </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    CPM (counts per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    RPKM (reads per kilobase per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    FPKM (fragments per kilobase per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    TPM (transcripts per million reads).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Differential Gene Expression Analysis:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    <b>EdgeR</b> (Robinson et al., 2010) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    <b>DESeq2</b> (Love et al. 2014)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    Download and install required tools:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    STAR: </span><span style='color:black'><a
    href="https://github.com/alexdobin/STAR"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://github.com/alexdobin/STAR</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Picard: </span><span style='color:black'><a
    href="https://broadinstitute.github.io/picard/"><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>https://broadinstitute.github.io/picard/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    HTseq: </span><span style='color:black'><a
    href="https://htseq.readthedocs.io/en/release_0.9.1/install.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://htseq.readthedocs.io/en/release_0.9.1/install.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    R: </span><span style='color:black'><a href="https://www.r-project.org"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.r-project.org</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>ngs.plot:
    Quick mining and visualization of next-generation sequencing data by integrating
    genomic databases</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Genomics 2014 volume 15, Article number: 284 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>415
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-284"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-284</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer:
    an R/Bioconductor package for interacting with RNA-seq principal components</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Bioinformatics 2019 volume 20, Article number: 331  </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>17
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2879-1"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2879-1</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer
    User Guide</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    October 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/pcaExplorer.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/pcaExplorer.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Up
    and running with pcaExplorer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    October 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/upandrunning.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/upandrunning.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>data
    component (count matrix, experimental data, dds object, annotation)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    Beginner’s Guide to Analysis of RNA Sequencing Data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096346/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096346/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Am
    J Respir Cell Mol Biol. 2018 Aug</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Abstract:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    general understanding of the principles underlying each step of RNA-seq
    data analysis allows investigators without a background in programming and
    bioinformatics to critically analyze their own datasets as well as
    published data.  Our goals in the present review are to break down the
    steps of a typical RNA-seq analysis and to highlight the pitfalls and
    checkpoints along the way that are vital for bench scientists and
    biomedical researchers performing experiments that use RNA-seq.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Keywords:
    </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA
    sequencing, transcriptomics, bioinformatics, data analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Associated
    Data:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    RNA-seq data reported in this article has been deposited in NCBI’s Gene
    Expression Omnibus (GEO) and are accessible through GEO Series accession
    number GSE116583.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Experimental
    Design and Approach:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    major goal of RNA-seq analysis is to identify differentially expressed and
    coregulated genes and to infer biological meaning for further studies.
    Source material can be cells cultured in vitro, whole-tissue homogenates,
    or sorted cells. The ability to interpret findings depends on appropriate
    experimental design, implementation of controls, and correct analysis.
    Every effort should be made to minimize batch effect, because small and
    uncontrolled changes in an environment can result in identification of
    differentially expressed genes (DEGs) unrelated to the designed experiment.
    Sources of batch effect can occur during the experiment, during the RNA
    library preparation, or during the sequencing run and include but are not
    limited to those listed in Table 1. Once a well-designed and controlled
    experiment is performed, a structured approach to the dataset allows for
    quality control followed by unbiased analysis of the data. In the present
    analysis, we use an approach that includes setting low count filtering,
    establishing a noise threshold, checking for potential outliers, running
    appropriate statistical tests to identify DEGs, clustering of genes by
    expression pattern, and testing for gene ontology (GO) enrichment. For each
    of these analysis components, we aim to highlight important checkpoints and
    quality controls that will streamline and strengthen data analysis, avoid
    bias, and allow investigators to maximally use their datasets.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    two most common clustering methods used for RNA-seq data analysis are
    hierarchical and k-means clustering (see Clustering box). The most common
    form of hierarchical clustering is a bottom-up agglomerative approach that
    organizes the data into a tree structure without user input by starting with
    each data point as its own cluster and iteratively combining them into
    larger clusters or “clades.” In contrast, k-means clustering requires the
    investigator to define the number of clusters (k) a priori, and data are
    then sorted into the cluster with the nearest mean. It is possible to
    assess a range of k-values to decide how to best capture the trends. In
    addition, various tools such as Silhouette exist to help the investigator
    determine the ideal k-value, but some subjectivity remains (21). By adjusting
    the k, the investigator may set the degree of granularity they would like
    to achieve with the data. For either approach, the user must specify the
    distance metric by which data points are considered similar. Typically,
    Pearson’s correlation is used, and this is generally the default in
    software designed for RNA-seq analysis. Both approaches are widely used,
    and both aid the investigator in identifying groups of genes that display
    similar expression patterns, allowing for further downstream analyses. The clusters
    can then be used as input for an analysis of functional enrichment (see
    next section).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Why
    do we use clustering on RNA-seq data?</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Clustering
    of RNA-seq data may be used to identify patterns of gene expression by
    grouping genes based on their distance in an unsupervised manner.
    Clustering RNA-seq data is used as an exploratory tool that allows the user
    to organize and visualize relationships between groups of genes, and to
    select certain genes for further consideration.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Hierarchical
    clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    most commonly used hierarchical clustering approach is a form of
    agglomerative, or bottom-up, clustering that iteratively merges clusters
    (originally consisting of individual data points) into larger clusters or
    “clades”.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>K-means
    clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Data
    points are iteratively partitioned into clusters based on the minimum
    distance to the cluster mean. The number of clusters (k) is set by the
    investigator.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Review
    of RNA-Seq Data Analysis Tools</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>February
    17, 2016</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://rna-seqblog.com/review-of-rna-seq-data-analysis-tools/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://rna-seqblog.com/review-of-rna-seq-data-analysis-tools/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Feature-based
    classification of human transcription factors into hypothetical sub-classes
    related to regulatory function</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Bioinformatics volume 17, Article number: 459 (2016) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>7
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1349-2"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1349-2</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO2RNAseq:
    An easy-to-use R pipeline for complete pre-processing of RNA-seq data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>September
    16, 2019 bioRxiv Preprint</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.biorxiv.org/content/10.1101/771063v1.full"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/771063v1.full</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNAseq
    data analysis in R - Notebook</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://monashbioinformaticsplatform.github.io/RNAseq-DE-analysis-with-R/RNAseq_DE_analysis_with_R.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://monashbioinformaticsplatform.github.io/RNAseq-DE-analysis-with-R/RNAseq_DE_analysis_with_R.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Install and load packages</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Mapping reads to a reference genome</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Count reads for each feature</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    QC and stats</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Differential Expression</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Gene Annotation</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Gene Set Enrichment</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Introduction
    to differential gene expression analysis using RNA-seq</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>September
    2015 updated November 14, 2019</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Genomic Data Viewers</span></b><span
  style='font-family:"Times New Roman",serif;color:black'> </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif;color:black'>     (navigation,
  search, content interpretation, data accession)<b><br>
  <br>
  </b></span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (GDV)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.38"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genome/gdv/browser/genome/?id=GCF_000001405.38</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    NCBI Genome Data Viewer (GDV) is a genome browser supporting the
    exploration and analysis of eukaryotic RefSeq genome assemblies. Genome
    Data Viewer is also used by different NCBI resources, such as GEO and
    dbGaP, to display datasets associated with specified experiments or samples
    in a genome browser context. <br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Tutorials</span></b></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (Tutorial Page Functionality)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/genome/gdv/browser/help/#LAYOUT"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/genome/gdv/browser/help/#LAYOUT</span></a></span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Genome Data Viewer
    (Tutorial 11 videos Last updated Sep 23, 2020)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.youtube.com/playlist?list=PLH-TjWpFfWruHgL0WRzZfQwp-MWzhIj16"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/playlist?list=PLH-TjWpFfWruHgL0WRzZfQwp-MWzhIj16</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>NCBI Gene Expression
    Omnibus (GEO)  <br>
    </span></b><span style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO
    is a public functional genomics data repository supporting MIAME-compliant
    data submissions.<br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>GEO DataSets (GDS)</span></b></p>
    <p class=MsoNormal style='margin-bottom:8.0pt;line-height:107%'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/gds/"><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/gds/</span></a></span><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif;
    color:black'><br>
    This database stores curated gene expression DataSets, as well as original
    Series and Platform records in the Gene Expression Omnibus (GEO)
    repository. Enter search terms to locate experiments of interest. DataSet
    records contain additional resources including cluster tools and
    differential expression queries.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>About
    GEO DataSets<br>
    </span></b><span style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/info/datasets.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/info/datasets.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO
    Profiles</span></b></p>
    <p class=MsoNormal style='margin-bottom:8.0pt;line-height:107%'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/geoprofiles/"><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geoprofiles/</span></a></span><span
    style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif;
    color:black'><br>
    This database stores individual gene expression profiles from curated
    DataSets in the Gene Expression Omnibus (GEO) repository. Search for
    specific profiles of interest based on gene annotation or pre-computed
    profile characteristics. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Querying
    GEO DataSets and GEO Profiles</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/geo/info/qqtutorial.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/geo/info/qqtutorial.html</span></a></span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    6.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Related Publications</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    Gene Expression Omnibus database<br>
    Methods Mol Biol. 2016; 1418: 93–110.</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944384/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944384/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCBI
    GEO: mining millions of expression profiles database and tools<br>
    Nucleic Acids Res. 2005 Jan 1</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539976/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539976/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGV
    (Integrative Genomics Viewer)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="http://broadinstitute.org/software/igv/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://broadinstitute.org/software/igv/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGV-Web
    application</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://igv.org/app"><span style='font-size:
    8.0pt;font-family:"Times New Roman",serif'>https://igv.org/app</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGV
    User Guide</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://broadinstitute.org/software/igv/UserGuide"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://broadinstitute.org/software/igv/UserGuide</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Integrative
    Genomics Viewer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3346182/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3346182/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Video:
    </span><span style='color:black'><a
    href="https://www.youtube.com/channel/UCb5W5WqauDOwubZHb-IA_rA"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/channel/UCb5W5WqauDOwubZHb-IA_rA</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Play
    All: </span><span style='color:black'><a
    href="https://www.youtube.com/watch?v=sFeK25K5PE&amp;list=PLSplvWwdPpSrhPn3V2iuPUzyxVIDYZ1xS"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=sFeK25K5PE&amp;list=PLSplvWwdPpSrhPn3V2iuPUzyxVIDYZ1xS</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoNormalTable border=1 cellspacing=0 cellpadding=0
   style='border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Genome Analysis
    Toolkit (GATK) </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
    color:black'>Java framework</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gatk.broadinstitute.org/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gatk.broadinstitute.org/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://gatk.broadinstitute.org/hc/en-us"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gatk.broadinstitute.org/hc/en-us</span></a></span></p>
    <p class=MsoNormal><span style='color:black'><a
    href="https://www.broadinstitute.org/partnerships/education/broade/best-practices-variant-calling-gatk-1"><span
    style='font-size:8.0pt;line-height:107%;font-family:"Times New Roman",serif'>https://www.broadinstitute.org/partnerships/education/broade/best-practices-variant-calling-gatk-1</span></a></span><span
    class=MsoHyperlink><br>
    </span><span style='font-size:10.0pt;line-height:107%;font-family:"Times New Roman",serif;
    color:black'>A large Java library for variant analysis, discovery and
    genotyping, <br>
    powerful processing engine and high-performance computing features make it
    capable of taking on projects of any size.</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;background:white;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GATK
    Best Practices Workflow for DNA-Seq</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bioinformaticsworkbook.org/dataAnalysis/VariantCalling/gatk-dnaseq-best-practices-workflow.html#gsc.tab=0"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bioinformaticsworkbook.org/dataAnalysis/VariantCalling/gatk-dnaseq-best-practices-workflow.html#gsc.tab=0</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GATK
    Getting Started</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gatk.broadinstitute.org/hc/en-us/categories/360002302312"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gatk.broadinstitute.org/hc/en-us/categories/360002302312</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Best
    Practices Workflows, Tutorials, Computing Platforms</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GATK
    Technical Documentation </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gatk.broadinstitute.org/hc/en-us/categories/360002310591"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gatk.broadinstitute.org/hc/en-us/categories/360002310591</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Troubleshooting,
    Glossary, Algorithms</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GATK
    Community Topics</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://gatk.broadinstitute.org/hc/en-us/community/topics"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://gatk.broadinstitute.org/hc/en-us/community/topics</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Browse
    community discussions</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GATK
    / broadinstitute / gatk </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GitHub:
    </span><span style='color:black'><a
    href="https://github.com/broadinstitute/gatk/releases"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://github.com/broadinstitute/gatk/releases</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Docker
    image: </span><span style='color:black'><a
    href="https://hub.docker.com/r/broadinstitute/gatk/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://hub.docker.com/r/broadinstitute/gatk/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Terra
    Support Quickstart New users overview </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Bioinformatics
    in the cloud on Terra </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://support.terra.bio/hc/en-us/articles/360022714931-Bioinformatics-in-the-cloud-on-Terra"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://support.terra.bio/hc/en-us/articles/360022714931-Bioinformatics-in-the-cloud-on-Terra</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:black'>NCBI Sequence Read Archive (SRA)</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi</span></a></span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>?</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>SRA
  Explorer</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>tool
  aims to make datasets within the Sequence Read Archive more accessible.</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://sra-explorer.info/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://sra-explorer.info/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>1000 Genomes Project<span
  style='color:black'> </span></span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Project FAQ</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>If
    you have any other questions you can’t find the answer to please email </span><span
    style='color:black'><a href="mailto:info@1000genomes.org"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>info@1000genomes.org</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.internationalgenome.org/faq"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/faq</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>IGSR:
    The International Genome Sample Resource</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Open
    human variation data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.internationalgenome.org/home"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/home</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes 30x on GRCh38</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.internationalgenome.org/data-portal/data-collection/30x-grch38"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/data-portal/data-collection/30x-grch38</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>1000
    Genomes Browser (Phase 3)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/variation/tools/1000genomes/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ensembl</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://uswest.ensembl.org/Homo_sapiens/Info/Index"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://uswest.ensembl.org/Homo_sapiens/Info/Index</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Data
    Portal (Search by Population, Technology, Data Collection)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.internationalgenome.org/data-portal/sample"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.internationalgenome.org/data-portal/sample</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  
    Technology</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Exome</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Low
    coverage WGS </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      Integrated
    variant call sets</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>  
    File Format</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      FASTQ</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>      VCF</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

</div>

</body>

</html>
