<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Mangal;
	panose-1:2 4 5 3 5 2 3 3 2 2;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:0in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h2
	{mso-style-link:"Heading 2 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:13.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
h3
	{mso-style-link:"Heading 3 Char";
	margin-right:0in;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Times New Roman",serif;
	font-weight:bold;}
h4
	{mso-style-link:"Heading 4 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;
	font-style:italic;}
h5
	{mso-style-link:"Heading 5 Char";
	margin-top:2.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:107%;
	page-break-after:avoid;
	font-size:11.0pt;
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-weight:normal;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoBodyText, li.MsoBodyText, div.MsoBodyText
	{mso-style-link:"Body Text Char";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:6.0pt;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
a:link, span.MsoHyperlink
	{color:#0563C1;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:#954F72;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
code
	{font-family:"Courier New";}
pre
	{mso-style-link:"HTML Preformatted Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Courier New";}
tt
	{font-family:"Courier New";}
p.MsoNoSpacing, li.MsoNoSpacing, div.MsoNoSpacing
	{margin:0in;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:8.0pt;
	margin-left:.5in;
	line-height:107%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
span.UnresolvedMention1
	{mso-style-name:"Unresolved Mention1";
	color:#605E5C;
	background:#E1DFDD;}
span.Heading3Char
	{mso-style-name:"Heading 3 Char";
	mso-style-link:"Heading 3";
	font-family:"Times New Roman",serif;
	font-weight:bold;}
span.mw-headline
	{mso-style-name:mw-headline;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading2Char
	{mso-style-name:"Heading 2 Char";
	mso-style-link:"Heading 2";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.Heading4Char
	{mso-style-name:"Heading 4 Char";
	mso-style-link:"Heading 4";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;
	font-style:italic;}
span.a-declarative
	{mso-style-name:a-declarative;}
span.a-icon-alt
	{mso-style-name:a-icon-alt;}
span.a-size-small
	{mso-style-name:a-size-small;}
p.zghrsritem, li.zghrsritem, div.zghrsritem
	{mso-style-name:zg_hrsr_item;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.zghrsrrank
	{mso-style-name:zg_hrsr_rank;}
span.zghrsrladder
	{mso-style-name:zg_hrsr_ladder;}
span.HTMLPreformattedChar
	{mso-style-name:"HTML Preformatted Char";
	mso-style-link:"HTML Preformatted";
	font-family:"Courier New";}
span.BodyTextChar
	{mso-style-name:"Body Text Char";
	mso-style-link:"Body Text";
	font-family:"Times New Roman",serif;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;}
span.job-id
	{mso-style-name:job-id;}
span.visually-hidden
	{mso-style-name:visually-hidden;}
span.jobsearch-jobinfoheader-bullet
	{mso-style-name:jobsearch-jobinfoheader-bullet;}
span.icl-u-xs-mr--xs
	{mso-style-name:icl-u-xs-mr--xs;}
span.jobsearch-jobmetadataheader-item
	{mso-style-name:jobsearch-jobmetadataheader-item;}
span.Heading5Char
	{mso-style-name:"Heading 5 Char";
	mso-style-link:"Heading 5";
	font-family:"Calibri Light",sans-serif;
	color:#2F5496;}
span.topiccitationitalics
	{mso-style-name:topiccitationitalics;}
span.bold
	{mso-style-name:bold;}
p.p, li.p, div.p
	{mso-style-name:p;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.st
	{mso-style-name:st;}
span.treelabel
	{mso-style-name:tree_label;}
p.msonormal0, li.msonormal0, div.msonormal0
	{mso-style-name:msonormal;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.token
	{mso-style-name:token;}
span.content-section
	{mso-style-name:content-section;}
span.label
	{mso-style-name:label;}
p.chapter-para, li.chapter-para, div.chapter-para
	{mso-style-name:chapter-para;
	margin-right:0in;
	margin-left:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.underline
	{mso-style-name:underline;}
span.pre
	{mso-style-name:pre;}
span.caps
	{mso-style-name:caps;}
span.Date1
	{mso-style-name:Date1;}
.MsoChpDefault
	{font-family:"Calibri",sans-serif;}
.MsoPapDefault
	{margin-bottom:8.0pt;
	line-height:107%;}
 /* Page Definitions */
 @page WordSection1
	{size:8.5in 11.0in;
	margin:.3in .3in .3in .5in;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>

</head>

<body lang=EN-US link="#0563C1" vlink="#954F72" style='word-wrap:break-word'>

<div class=WordSection1>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;line-height:normal'><b><span style='font-size:12.0pt;
font-family:"Times New Roman",serif'>Project Ideas</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=787 valign=top style='width:8.2in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=787 valign=top style='width:8.2in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Exploratory
    data analysis of genomic datasets using ADAM and Mango with Apache Spark on
    Amazon EMR</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>13
    JUL 2018 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Amazon
    EMR, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/exploratory-data-analysis-of-genomic-datasets-using-adam-and-mango-with-apache-spark-on-amazon-emr/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>As the
    cost of genomic sequencing has rapidly decreased, the amount of publicly
    available genomic data has soared over the past couple years. New cohorts
    and studies have produced massive datasets consisting of over 100,000
    individuals. Simultaneously, these datasets have been processed to extract
    genetic variation across populations, producing mass amounts of variation
    data for each cohort. In this era of big data, tools like Apache Spark have
    provided a user-friendly platform for batch processing large datasets.
    However, in order to use such tools as a sufficient replacement to current
    bioinformatics pipelines, we need more accessible and comprehensive API’s
    for processing genomic data, as well as support for interactive exploration
    of these processed datasets.<br>
    <br>
    </span></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/adam" target="_blank"><b><span
    style='font-size:10.0pt;color:black;text-decoration:none'>ADAM</span></b></a></span><b><span
    style='font-size:10.0pt;color:black'> </span></b></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/adam"><span style='font-size:10.0pt'>https://github.com/bigdatagenomics/adam</span></a></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>and </span></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/mango"><b><span style='font-size:
    10.0pt;color:black;text-decoration:none'>Mango</span></b></a></span><b><span
    style='font-size:10.0pt;color:black'> </span></b></p>
    <p style='margin:0in'><span style='color:black'><a
    href="https://github.com/bigdatagenomics/mango"><span style='font-size:
    10.0pt'>https://github.com/bigdatagenomics/mango</span></a></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>provide a
    unified environment for processing, filtering, and visualizing large
    genomic datasets on Apache Spark. ADAM allows users to programmatically
    load, process, and select raw genomic and variation data using SparkSQL, an
    SQL interface for aggregating and selecting data in Apache Spark. Mango
    supports visualization of both raw and aggregated genomic data in a Jupyter
    notebook environment, allowing users to draw conclusions from large
    datasets at multiple resolutions. This combined power of ADAM and Mango
    allows users to load, query and explore datasets in a unified environment,
    allowing users to interactively explore genomic data at a scale previously
    impossible using single node bioinformatics tools.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></h2>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>Configuring ADAM and Mango on
    Amazon EMR</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First, we
    will launch and configure an EMR cluster. Mango uses Docker containers to
    easily run on Amazon EMR. Upon cluster startup, EMR will use the bootstrap
    action below to install Docker and the required startup scripts. The
    scripts will be available at </span><tt><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>/home/hadoop/mango-scripts</span></tt></p>
    <pre><code><span style='font-family:"Times New Roman",serif;color:black'>aws emr create-cluster </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--release-label emr-5.19.0 </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--name </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'emr-5.19.0 Mango example'</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--applications </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>Name=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>Hadoop </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>Name=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>Hive </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>Name=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>Spark </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--ec2-attributes </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>KeyName=&lt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>your-ec2-key</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&gt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>,InstanceProfile</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>EMR_EC2_DefaultRole </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--service-role EMR_DefaultRole </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>     </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--instance-groups </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>     </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>InstanceGroupType=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>MASTER,InstanceCount</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=1</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>,InstanceType</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>c5.4xlarge </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>     </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>InstanceGroupType=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>CORE,InstanceCount</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=4</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>,InstanceType</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>c5.4xlarge </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   --region </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&lt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>your-aws-region</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&gt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--log-uri s3://</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&lt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>your-s3-bucket</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&gt;</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>/emr-logs/ </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>   </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>--bootstrap-actions </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>\</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>     </span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>Name='Install Mango'</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>, </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>Path=&quot;s3://aws-bigdata-blog/artifacts/mango-emr/install-bdg-mango-docker-emr5.sh&quot;</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>To start
    the Mango notebook, run the following:</span></p>
    <pre><code><span style='font-family:"Times New Roman",serif;color:black'>/home/hadoop/mango-scripts/run-notebook.sh</span></code></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>This file
    will set up all of the environment variables needed to run Mango in Docker
    on EMR. In your terminal, you will see the port and Jupyter notebook token
    for the Mango Notebook session. Navigate to this port on the public DNS URL
    of the master node for your EMR cluster.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></h2>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>Loading data from the 1000
    Genomes Project</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now that
    we have a working environment, lets use ADAM and Mango to discover
    interesting variants in the child from the genome sequencing data of a trio
    (data from a mother, father, and child). These data are available from the <br>
    </span><span style='color:black'><a
    href="https://aws.amazon.com/1000genomes/" target="_blank"><span
    style='font-size:10.0pt'>1000 Genomes Project AWS Public Dataset</span></a></span><span
    style='font-size:10.0pt;color:black'>. <br>
    </span><span style='color:black'><a
    href="https://registry.opendata.aws/1000-genomes/"><span style='font-size:
    10.0pt'>https://registry.opendata.aws/1000-genomes/</span></a></span></p>
    <p style='margin:0in'><b><span style='font-size:10.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>Resources on AWS</span></b></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Description</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        </span><span style='color:black'><a
    href="http://www.internationalgenome.org/formats"><span style='font-size:
    9.0pt;font-family:"Segoe UI",sans-serif'>http://www.internationalgenome.org/formats</span></a></span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Resource type</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        S3 Bucket</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    Amazon Resource Name (ARN)</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        arn:aws:s3:::1000genomes</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    AWS Region</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        us-east-1</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>    AWS CLI Access (No AWS account required)</span></p>
    <p style='margin:0in'><span style='font-size:9.0pt;font-family:"Segoe UI",sans-serif;
    color:black'>        aws s3<br>
    </span><span style='font-size:10.0pt;font-family:"Segoe UI",sans-serif;
    color:black'><br>
    </span><span style='font-size:10.0pt;color:black'>In this analysis, we will
    view a trio (</span><span style='color:black'><a
    href="http://www.internationalgenome.org/data-portal/sample/NA19685"
    target="_blank"><span style='font-size:10.0pt'>NA19685</span></a></span><span
    style='font-size:10.0pt;color:black'>, </span><span style='color:black'><a
    href="http://www.internationalgenome.org/data-portal/sample/NA19661"
    target="_blank"><span style='font-size:10.0pt'>NA19661</span></a></span><span
    style='font-size:10.0pt;color:black'>, and </span><span style='color:black'><a
    href="http://www.internationalgenome.org/data-portal/sample/NA19660"
    target="_blank"><span style='font-size:10.0pt'>NA19660</span></a></span><span
    style='font-size:10.0pt;color:black'>) and search for variants that are
    present in the child but not present in the parents.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In
    particular, we want to identify genetic variants that are found in the
    child but not in the parents, known as <em>de novo variants</em>. These are
    interesting regions, as they may indicate sights of de novo variation that
    may contribute to multiple disorders.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>You can
    find the Jupyter notebook containing these examples in </span><span
    style='color:black'><a
    href="https://github.com/bigdatagenomics/mango/blob/master/example-files/notebooks/aws-1000genomes.ipynb"
    target="_blank"><span style='font-size:10.0pt'>Mango’s GitHub repository</span></a></span><span
    style='font-size:10.0pt;color:black'>, or at </span><tt><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>/opt/cgl-docker-lib/mango/example-files/notebooks/aws-1000genomes.ipynb</span></tt><span
    style='font-size:10.0pt;color:black'> in the running Docker container for
    Mango.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First,
    import the ADAM and Mango modules and any Spark modules that you need:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Import ADAM modules</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bdgenomics</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>adam</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>adamContext </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ADAMContext</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bdgenomics</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>adam</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>rdd </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> AlignmentRecordRDD</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> CoverageRDD</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bdgenomics</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>adam</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>stringency </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> LENIENT</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> _toJava</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Import Mango modules</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bdgenomics</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>mango</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>rdd </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> GenomicVizRDD</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bdgenomics</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>mango</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>QC </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> CoverageDistribution</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Import Spark modules</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>from</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> pyspark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>sql </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>import</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> functions </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>as</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sf</span></code></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Next,
    create a Spark session. You will use this session to run SQL queries on
    variants.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Create ADAM Context</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ac </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ADAMContext</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Variant analysis with Spark SQL</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Load in a
    subset of variant data from chromosome 17:</span></p>
    <pre><code><span style='font-family:"Times New Roman",serif;color:black'>genotypesPath </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'s3://1000genomes/phase1/analysis_results/integrated_call_sets/ALL.chr17.integrated_phase1_v3.20101123.snps_indels_svs.genotypes.vcf.gz'</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>genotypes </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>loadGenotypes</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>genotypesPath</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># repartition genotypes to balance the load across memory</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>genotypes_df  </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> genotypes</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>toDF</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>You can
    take a look at the schema by printing the columns in the dataframe.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># cache genotypes and show the schema</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>genotypes_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>columns</span></code></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>This
    genotypes dataset contains all samples from the 1000 Genomes Project.
    Therefore, you will next filter genotypes to only consider samples that are
    in the NA19685 trio, and cache the results in memory.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># trio IDs</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>IDs </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>['NA19685',</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'NA19661','NA19660']</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Filter by individuals in the trio</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>trio_df </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> genotypes_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.filter(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>genotypes_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>[&quot;sampleId&quot;].</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>isin</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>IDs</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>))</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>trio_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>cache</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>trio_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>count</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Next, add
    a new column to your dataframe that determines the genomic location of each
    variant. This is defined by the chromosome (contigName) and the start and
    end position of the variant.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Add ReferenceRegion column and group by referenceRegion</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>trios_with_referenceRegion </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> trio_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>withColumn</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>('ReferenceRegion',</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>                    sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>concat</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>col</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>('contigName'),</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>lit</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(':'),</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>col</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>('start'),</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>lit</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>('-'),</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sf</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>col</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>('end')))</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now, you
    can query your dataset to find de novo variants. But first, you must
    register your dataframe with Spark SQL.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'>#  Register df with Spark SQL</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>trios_with_referenceRegion</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>createOrReplaceTempView</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;trios&quot;)</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now that
    your dataframe is registered, you can run SQL queries on it. For the first
    query, select the names of variants belonging to sample NA19685 that have
    at least one alternative (ALT) allele.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># filter by alleles. This is a list of variant names that have an alternate allele for the child</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>alternate_variant_sites </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>sql</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>&quot;SELECT variant</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>names</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>[0]</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> AS snp FROM trios \</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>                                    WHERE array_contains</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>alleles</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'ALT')</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> AND sampleId </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>==</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'NA19685'</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>&quot;</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>collected_sites </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>list(map(lambda</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>:</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>snp</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> alternate_variant_sites</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>collect</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>())</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>For your
    next query, filter sites in which the parents have both reference alleles.
    Then filter these variants by the set produced previously from the child.</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># get parent records and filter by only REF locations for variant names that were found in the child with an ALT</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>filtered1 </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>sql</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>&quot;SELECT </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>*</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> FROM trios WHERE sampleId </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>==</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'NA19661'</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>or</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sampleId </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>==</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'NA19660'</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> \</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>            AND !array_contains</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>alleles</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'ALT')</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>&quot;</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>filtered2 </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> filtered1</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.filter(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>filtered1</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>[&quot;variant.names&quot;][0].</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>isin</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>collected_sites</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>))</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>snp_counts </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> filtered2</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>groupBy</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;variant.names&quot;).</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>count</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>().</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>collect</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># collect snp names as a list</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>snp_names </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>map(lambda</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>:</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>names</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> snp_counts</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>denovo_snps </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>[</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>item </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>for</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sublist </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>in</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> snp_names </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>for</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> item </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>in</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> sublist</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>]</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>denovo_snps</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>[:10]</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=423 height=166 id="Picture 19"
    src="Project-Ideas_files/image001.png"></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now that
    you have found some interesting variants, you can unpersist your genotypes
    from memory.</span></p>
    <pre><code><span style='font-family:"Times New Roman",serif;color:black'>trio_df</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>unpersist</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Working with alignment data</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>You have
    found a lot of potential de novo variant sites. Next, you can visually
    verify some of these sites to see if the raw alignments match up with these
    de novo hits.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First,
    load in the alignment data for the NA19685 trio:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># load in NA19685 exome from s3a</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childReadsPath </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'s3a://1000genomes/phase1/data/NA19685/exome_alignment/NA19685.mapped.illumina.mosaik.MXL.exome.20110411.bam'</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent1ReadsPath </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'s3a://1000genomes/phase1/data/NA19685/exome_alignment/NA19660.mapped.illumina.mosaik.MXL.exome.20110411.bam'</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent2ReadsPath </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>'s3a://1000genomes/phase1/data/NA19685/exome_alignment/NA19661.mapped.illumina.mosaik.MXL.exome.20110411.bam'</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childReads </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>loadAlignments</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>childReadsPath</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> stringency</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>LENIENT</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent1Reads </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>loadAlignments</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent1ReadsPath</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> stringency</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>LENIENT</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent2Reads </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>loadAlignments</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent2ReadsPath</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> stringency</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>LENIENT</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Note that
    this example uses </span><tt><span style='font-size:10.0pt;font-family:
    "Times New Roman",serif;color:black'>s3a://</span></tt><span
    style='font-size:10.0pt;color:black'> instead of </span><tt><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>s3://</span></tt><span
    style='font-size:10.0pt;color:black'> style URLs. The reason for this is
    that the ADAM formats use Java NIO to access BAM files. To do this, we are
    using a </span><span style='color:black'><a
    href="https://jcp.org/en/jsr/detail?id=203" target="_blank"><span
    style='font-size:10.0pt'>JSR 203</span></a></span><span style='font-size:
    10.0pt;color:black'> implementation for the Hadoop Distributed File System
    to access these files. This itself requires the s3a:// protocol. You can
    view that implementation in </span><span style='color:black'><a
    href="https://github.com/fnothaft/jsr203-s3a" target="_blank"><span
    style='font-size:10.0pt'>this GitHub repository</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>You now
    have data alignment data for three individuals in your trio. However, the
    data has not yet been loaded into memory. To cache these datasets for fast
    subsequent access to the data, run the cache() function:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># cache child RDD and count records</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># takes about 2 minutes, on 4 c3.4xlarge worker nodes </span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childReads</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>cache</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Count reads in the child</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childReads</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>toDF</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>().</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>count</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Output should be 95634679</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Quality control of alignment data</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>One
    popular analysis to visually re-affirm the quality of genomic alignment
    data is by viewing coverage distribution. Coverage distribution gives you
    an idea of the read coverage that you have across a sample.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Next,
    generate a sample coverage distribution plot for the child alignment data
    on chromosome 17:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Calculate read coverage</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Takes 2-3 minutes</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childCoverage </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> childReads</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>transform</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(lambda</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>:</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.filter(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>x</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>contigName </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>==</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&quot;17&quot;)).</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>toCoverage</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childCoverage</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>cache</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childCoverage</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>toDF</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>().</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>count</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Output should be 51252612</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Now that
    coverage data is calculated and cached, compute the coverage distribution
    of chromosome 17 and plot the coverage distribution:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># Calculate coverage distribution</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># You can check the progress in the SparkUI by navigating to </span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># :8088 and clicking on the currently running Spark application.</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>cd </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> CoverageDistribution</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> childCoverage</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> bin_size </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>1)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ax</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> results </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> cd</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>plotDistributions</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>normalize</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=True,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> cumulative</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=False)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ax</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>set_title</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;Normalized Target Region Coverage&quot;)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ax</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>set_ylabel</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;Fraction&quot;)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ax</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>set_xlabel</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;Coverage Depth&quot;)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>ax</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>set_xscale</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(&quot;log&quot;)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>plt</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>show</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>&nbsp;</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=564 height=320 id="Picture 18"
    src="Project-Ideas_files/image002.png"></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>This looks
    pretty standard because the data you are viewing is exome data. Therefore,
    you can see a high number of sights with low coverage and a smaller number
    of genomic positions with more than 100 reads. Now that you are done with
    coverage, you can unpersist these datasets to clear space in memory for the
    next analysis.</span></p>
    <pre><code><span style='font-family:"Times New Roman",serif;color:black'>childCoverage</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>unpersist</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>()</span></span></pre>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'><br>
    Viewing sites with missense variants in the proband</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>After
    verifying alignment data and filtering variants, you have four genes with
    potential missense mutations in the proband, including YBX2, ZNF286B, KSR1,
    and GNA13. You can visually verify these sites by filtering and viewing the
    raw reads of the child and parents.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First,
    view the child reads. If you zoom in to the location of the GNA13 variant
    (63052580-63052581), you can see a heterozygous T to A call:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># missense variant at GNA13: 63052580-63052581 (SNP rs201316886)</span></span></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># define alignment summary for child reads</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childViz </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> AlignmentSummary</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> childReads</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># Takes about 2 minutes to collect data from workers</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>contig </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&quot;17&quot;</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>start </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>63052180</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>end </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>63052981</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>childViz</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>viewPileup</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>contig</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> start</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> end</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=574 height=306 id="Picture 17"
    src="Project-Ideas_files/image003.png"></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>It looks
    like there indeed is a variant at this position, possibly a heterozygous
    SNP with alternate allele A. Look at the parent data to verify that this
    variant does not appear in the parents:</span></p>
    <pre><span class=token><span style='font-family:"Times New Roman",serif;
    color:black'># define alignment summary for parent reads</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent1Viz </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> AlignmentSummary</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>spark</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> ac</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> parent1Reads</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'># view missense variant at GNA13: 63052580-63052581 in parent 1</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>contig </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>&quot;17&quot;</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>start </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>63052180</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>end </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>=</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> </span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>63052981</span></span></pre><pre><code><span
    style='font-family:"Times New Roman",serif'>&nbsp;</span></code></pre><pre><code><span
    style='font-family:"Times New Roman",serif;color:black'>parent1Viz</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>.</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>viewPileup</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>(</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'>contig</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> start</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>,</span></span><code><span
    style='font-family:"Times New Roman",serif;color:black'> end</span></code><span
    class=token><span style='font-family:"Times New Roman",serif;color:black'>)</span></span></pre>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'><img
    border=0 width=583 height=315 id="Picture 16"
    src="Project-Ideas_files/image004.png"></span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>This
    confirms the filter that this variant is indeed present only in the
    proband, but not the parents.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><b><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>Summary</span></b></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>To summarize,
    this post demonstrated how to set up and run ADAM and Mango in Amazon EMR.
    We demonstrated how to use these tools in an interactive notebook
    environment to explore the 1000 Genomes dataset, a publicly available
    dataset on Amazon S3. We used these tools inspect 1000 Genomes data
    quality, query for interesting variants in the genome, and validate results
    through the visualization of raw data.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>For more
    information about Mango, see the </span><span style='color:black'><a
    href="http://bdg-mango.readthedocs.io/en/latest/" target="_blank"><span
    style='font-size:10.0pt'>Mango User Guide</span></a></span><span
    style='font-size:10.0pt;color:black'>. If you have questions or
    suggestions, please comment below.<br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Orchestrating
    analytics jobs by running Amazon EMR Notebooks programmatically</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>23
    NOV 2020 | Amazon EMR, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/orchestrating-analytics-jobs-by-running-amazon-emr-notebooks-programmatically/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/orchestrating-analytics-jobs-by-running-amazon-emr-notebooks-programmatically/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='color:black'><a
    href="http://aws.amazon.com/emr" target="_blank"><span style='font-size:
    10.0pt'>Amazon EMR</span></a></span><span style='font-size:10.0pt;
    color:black'> is a big data service offered by AWS to run Apache Spark and
    other open-source applications on AWS in a cost-effective manner. </span><span
    style='color:black'><a
    href="https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-managed-notebooks.html"
    target="_blank"><span style='font-size:10.0pt'>Amazon EMR Notebooks</span></a></span><span
    style='font-size:10.0pt;color:black'> is a managed environment based on </span><span
    style='color:black'><a
    href="https://en.wikipedia.org/wiki/Project_Jupyter#Jupyter_Notebook"
    target="_blank"><span style='font-size:10.0pt'>Jupyter Notebook</span></a></span><span
    style='font-size:10.0pt;color:black'> that allows data scientists,
    analysts, and developers to prepare and visualize data, collaborate with
    peers, build applications, and perform interactive analysis using EMR
    clusters.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>EMR
    notebook APIs are available on Amazon EMR release version 5.18.0 or later
    and can be used to run EMR notebooks via a script or command line. The
    ability to start, stop, list, and describe EMR notebook runs without the
    Amazon EMR console enables you to programmatically control running an EMR
    notebook. Using a parameterized notebook cell allows you to pass different
    parameter values to a notebook without having to create a copy of the
    notebook for each new set of parameter values. With this feature, you can
    schedule running EMR notebooks with cron scripts, chain multiple EMR
    notebooks, and use orchestration services such as </span><span
    style='color:black'><a href="https://aws.amazon.com/step-functions/"
    target="_blank"><span style='font-size:10.0pt'>AWS Step Functions</span></a></span><span
    style='font-size:10.0pt;color:black'> or Apache Airflow to build pipelines.
    If you want to use EMR notebooks in a non-interactive manner, this enables
    you to run ETL workloads, especially in production.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In this
    post, we show how to orchestrate analytics jobs by running EMR Notebooks
    programmatically with the following two use cases:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Scheduling an EMR notebook run via crontab and the </span><span
         style='color:black'><a href="http://aws.amazon.com/cli" target="_blank"><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
         Command Line Interface</span></a></span><span style='font-size:10.0pt;
         font-family:"Times New Roman",serif;color:black'> (AWS CLI)</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Chaining your notebooks with Step Functions triggered by </span><span
         style='color:black'><a
         href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/WhatIsCloudWatchEvents.html"
         target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
         CloudWatch Events</span></a></span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>For our
    data source, we use the open-source, real-time COVID-19 US daily case
    reports provided by Johns Hopkins University CSSE in the following </span><span
    style='color:black'><a
    href="https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_daily_reports_us"
    target="_blank"><span style='font-size:10.0pt'>GitHub repo</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Prerequisites</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Before
    getting started, you must have the following prerequisites:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An AWS account that provides access to the following AWS
         services at least: </span></li>
     <ul style='margin-top:0in' type=circle>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/cloudformation"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          CloudFormation</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>Amazon CloudWatch</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/ec2"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          Elastic Compute Cloud</span></a></span><span style='font-size:10.0pt;
          font-family:"Times New Roman",serif;color:black'> (Amazon EC2)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>Amazon EMR</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="https://aws.amazon.com/eventbridge/"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          EventBridge</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/iam"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          Identity and Access Management</span></a></span><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'> (IAM)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/lambda"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>AWS
          Lambda</span></a></span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a href="http://aws.amazon.com/s3" target="_blank"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>Amazon
          Simple Storage Service</span></a></span><span style='font-size:10.0pt;
          font-family:"Times New Roman",serif;color:black'> (Amazon S3)</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'>AWS Step Functions</span></li>
     </ul>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>AWS CLI Version 1.18.128 or later installed on your work
         station.</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='color:black'><a href="https://jupyter.org/install"
         target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Jupyter</span></a></span><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'> installed on your work station (this is used for the
         output visualization part for this post only).</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An EMR cluster running Amazon EMR release 5.18.0 or
         later, with Hadoop, Spark, and Livy installed. Record the value of the
         cluster ID (for example, </span><em><span style='font-size:10.0pt;
         color:red'>&lt;j-*************&gt;</span></em><span style='font-size:
         10.0pt;font-family:"Times New Roman",serif;color:black'>); you use
         this for the examples later.</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>An EMR notebook created on the Amazon EMR console, using
         the following two input notebook files: </span></li>
     <ul style='margin-top:0in' type=circle>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a
          href="https://aws-bigdata-blog.s3.amazonaws.com/artifacts/aws-blog-runnable-notebook/demo/notebook/demo_pyspark.ipynb"
          target="_blank"><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>demo_pyspark.pynb</span></a></span><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif;
          color:black'> – Used for both use cases in this post.</span></li>
      <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
          style='color:black'><a
          href="https://aws-bigdata-blog.s3.amazonaws.com/artifacts/aws-blog-runnable-notebook/demo/notebook/trailing_N_day.ipynb"><span
          style='font-size:10.0pt;font-family:"Times New Roman",serif'>trailing_N
          day.ipynb</span></a></span><span style='font-size:10.0pt;font-family:
          "Times New Roman",serif;color:black'> – Used for the second use case.</span></li>
     </ul>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Record the
    notebook ID (for example, </span><em><span style='font-size:10.0pt;
    color:red'>&lt;e-*************************&gt;</span></em><span
    style='font-size:10.0pt;color:black'>); you use this later for our examples
    later. Organize the notebook files in the Jupyter UI as follows:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>/demo_pyspark.ipynb</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>/experiment/trailing_N_day.ipynb</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>See </span><span
    style='color:black'><a
    href="https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-managed-notebooks-create.html"
    target="_blank"><span style='font-size:10.0pt'>Creating a Notebook</span></a></span><span
    style='font-size:10.0pt;color:black'> for more information on how to create
    an EMR notebook.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-family:"Times New Roman",serif;color:black'>Interactive
    Analysis of Genomic Datasets Using Amazon Athena</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Aaron Friedman, PhD | on 07 DEC 2016 | in Amazon Athena, AWS Big Data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/big-data/interactive-analysis-of-genomic-datasets-using-amazon-athena/"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/big-data/interactive-analysis-of-genomic-datasets-using-amazon-athena/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><em><span style='font-size:10.0pt;color:black'>Aaron
    Friedman is a Healthcare and Life Sciences Solutions Architect with Amazon
    Web Services</span></em></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    genomics industry is in the midst of a data explosion. Due to the rapid
    drop in the cost to sequence genomes, genomics is now central to many
    medical advances. When your genome is sequenced and analyzed, raw
    sequencing files are processed in a multi-step workflow to identify where
    your genome differs from a standard reference. Your variations are stored
    in a Variant Call Format (VCF) file, which is then combined with other
    individuals to enable population-scale analyses. Many of these datasets are
    publicly available, and an increasing number are hosted on AWS as part of
    our </span><span style='color:black'><a
    href="https://aws.amazon.com/public-data-sets/" target="_blank"><span
    style='font-size:10.0pt'>Open Data</span></a></span><span style='font-size:
    10.0pt;color:black'> project.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>To mine
    genomic data for new discoveries, researchers in both industry and academia
    build complex models to analyze populations at scale. When building models,
    they first explore the datasets-of-interest to understand what questions
    the data might answer. In this step, interactivity is key, as it allows
    them to move easily from one question to the next.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Recently,
    we launched </span><span style='color:black'><a
    href="https://aws.amazon.com/athena/"><span style='font-size:10.0pt'>Amazon
    Athena</span></a></span><span style='font-size:10.0pt;color:black'> as an
    interactive query service to analyze data on </span><span style='color:
    black'><a href="https://aws.amazon.com/s3/" target="_blank"><span
    style='font-size:10.0pt'>Amazon S3</span></a></span><span style='font-size:
    10.0pt;color:black'>. With Amazon Athena there are no clusters to manage
    and tune, no infrastructure to setup or manage, and customers pay only for
    the queries they run. Athena is able to query many file types straight from
    S3. This flexibility gives you the ability to interact easily with your
    datasets, whether they are in a raw text format (CSV/JSON) or specialized
    formats (e.g. Parquet). By being able to flexibly query different types of
    data sources, researchers can more rapidly progress through the data
    exploration phase for discovery. Additionally, researchers don’t have to
    know nuances of managing and running a big data system. This makes Athena
    an excellent complement to data warehousing on </span><span
    style='color:black'><a href="https://aws.amazon.com/redshift/"
    target="_blank"><span style='font-size:10.0pt'>Amazon Redshift</span></a></span><span
    style='font-size:10.0pt;color:black'> and big data analytics on </span><span
    style='color:black'><a href="https://aws.amazon.com/emr/" target="_blank"><span
    style='font-size:10.0pt'>Amazon EMR</span></a></span><span
    style='font-size:10.0pt;color:black'>.&nbsp; </span><strong><span
    style='font-size:10.0pt;color:black'>&nbsp;</span></strong></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In this
    post, I discuss how to prepare genomic data for analysis with Amazon Athena
    as well as demonstrating how Athena is well-adapted to address common
    genomics query paradigms.&nbsp; I use the </span><span style='color:black'><a
    href="https://aws.amazon.com/1000genomes/" target="_blank"><span
    style='font-size:10.0pt'>Thousand Genomes dataset</span></a></span><span
    style='font-size:10.0pt;color:black'> hosted on Amazon S3, a seminal
    genomics study, to demonstrate these approaches. All code that is used as
    part of this post is available in our GitHub </span><span style='color:
    black'><a
    href="https://github.com/awslabs/aws-big-data-blog/tree/master/aws-blog-athena-genomics/"
    target="_blank"><span style='font-size:10.0pt'>repository</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Although
    this post is focused on genomic analysis, similar approaches can be applied
    to any discipline where large-scale, interactive analysis is required.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>&nbsp;</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Select, aggregate, annotate query
    pattern in genomics</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Genomics
    researchers may ask different questions of their dataset, such as:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What variations in a genome may increase the risk of
         developing disease?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What positions in the genome have abnormal levels of
         variation, suggesting issues in quality of sequencing or errors in the
         genomic reference?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>What variations in a genome influence how an individual
         may respond to a specific drug treatment?</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Does a group of individuals contain a higher frequency of
         a genomic variant known to alter response to a drug relative to the
         general population?</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>All these
    questions, and more, can be generalized under a common query pattern I like
    to call “Select, Aggregate, Annotate”. Some of our genomics customers, such
    as Human Longevity, Inc., routinely use this query pattern </span><span
    style='color:black'><a href="https://www.youtube.com/watch?v=CGbWEkszAlQ"
    target="_blank"><span style='font-size:10.0pt'>in their work</span></a></span><span
    style='font-size:10.0pt;color:black'>.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>In each of
    the above queries, you execute the following steps:</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>SELECT:
    </span></strong><span style='font-size:10.0pt;color:black'>Specify the
    cohort of individuals meeting certain criteria (disease, drug response,
    age, BMI, entire population, etc.).</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>AGGREGATE:
    </span></strong><span style='font-size:10.0pt;color:black'>Generate summary
    statistics of genomic variants across the cohort that you selected.</span></p>
    <p style='margin:0in'><strong><span style='font-size:10.0pt;color:black'>ANNOTATE:</span></strong><span
    style='font-size:10.0pt;color:black'> Assign meaning to each of the
    variants by joining on known information about each variant.</span></p>
    <h2 style='margin-top:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>Dataset preparation</span></h2>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>Properly
    organizing your dataset is one of the most critical decisions for enabling
    fast, interactive analyses. Based on the query pattern I just described,
    the table representing your population needs to have the following
    information:</span></p>
    <ul style='margin-top:0in' type=disc>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>A unique sample ID corresponding to each sample in your
         population</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Information about each variant, specifically its location
         in the genome as well as the specific deviation from the reference</span></li>
     <li class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
         style='font-size:10.0pt;font-family:"Times New Roman",serif;
         color:black'>Information about how many times in a sample a variant
         occurs (0, 1, or 2 times) as well as if there are multiple variants in
         the same site. This is known as a genotype.</span></li>
    </ul>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>The
    extract, transform, load (ETL) process to generate the appropriate data
    representation has two main steps. First, you use ADAM, a genomics analysis
    platform built on top of Spark, to convert the variant information residing
    a VCF file to Parquet for easier downstream analytics, in a process similar
    to the one described in the </span><span style='color:black'><a
    href="https://blogs.aws.amazon.com/bigdata/post/Tx1GE3J0NATVJ39/Will-Spark-Power-the-Data-behind-Precision-Medicine"
    target="_blank"><span style='font-size:10.0pt'>Will Spark Power the Data
    behind Precision Medicine?</span></a></span><span style='font-size:10.0pt;
    color:black'> post. Then, you use custom Python code to massage the data
    and select only the appropriate fields that you need for analysis with
    Athena.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>First, </span><span
    style='color:black'><a
    href="http://docs.aws.amazon.com/ElasticMapReduce/latest/ManagementGuide/emr-gs.html"
    target="_blank"><span style='font-size:10.0pt'>spin up an EMR cluster</span></a></span><span
    style='font-size:10.0pt;color:black'> (version 5.0.3) for the ETL process.
    I used a c4.xlarge for my master node and m4.4xlarges with 1 TB of scratch
    for my core nodes.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt;color:black'>After you
    SSH into your master node, clone the </span><span style='color:black'><a
    href="https://github.com/awslabs/aws-big-data-blog.git" target="_blank"><span
    style='font-size:10.0pt'>git repository</span></a></span><span
    style='font-size:10.0pt;color:black'>. You can also put this in as a
    bootstrap action when spinning up your cluster.</span></p>
    <p style='margin:0in'><span style='font-size:10.0pt'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Building
    High-Throughput Genomics Batch Workflows on AWS: Introduction (Part 1 of 4)</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/building-high-throughput-genomics-batch-workflows-on-aws-introduction-part-1-of-4/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Andy Katz | on 30 MAY 2017 |</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Genomics
    Research on AWS</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://github.com/aws-samples/aws-batch-genomics/tree/v1.0.0"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://github.com/aws-samples/aws-batch-genomics/tree/v1.0.0</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    tutorial on how to package and deploy a bioinformatics workflow on AWS
    using AWS Batch</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>This
    tutorial will cover the material presented within the &quot;Genomics
    Workflows on AWS&quot; blog post series </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>(Part
    1, Part 2, Part 3, Part 4) that covers the basics of bootstrapping a
    bioinformatics analysis pipeline on AWS. </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>We
    break down the tutorial roughly as follows:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Setting up your AWS account (if you do not already have one)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Package a set of bioinformatics applications using Docker</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Create a AWS Batch environment for analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Define and deploy AWS Step Functions to control the data processing steps</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Initiate a workflow</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Optimizing
    for cost, availability and throughput by selecting your AWS Batch
    allocation strategy</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Bala Thekkedath | on 24 OCT 2019 | in Advanced (300), Amazon EC2, AWS Batch</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p style='margin:0in'><span style='color:black'>AWS offers a broad range of
    instances that are advantageous for batch workloads. The scale and
    provisioning speed of AWS’ compute instances allow you to get up and
    running at peak capacity in minutes without paying for downtime. Today, I’m
    pleased to introduce allocation strategies: a significant new capability in
    <a
    href="https://docs.aws.amazon.com/batch/latest/userguide/allocation-strategies.html">AWS
    Batch</a> that &nbsp;makes provisioning compute resources flexible and
    simple. In this blog post, I explain how the AWS Batch allocation
    strategies work, when you should use them for your workload, and provide an
    example CloudFormation script. This blog helps you get started on building
    your personalized Compute Environment (CE) most appropriate to your workloads.</span></p>
    <p style='margin:0in'><strong><span style='color:black'><br>
    Overview</span></strong></p>
    <p style='margin:0in'><span style='color:black'>AWS Batch is a fully
    managed, cloud-native batch scheduler. It manages the queuing and
    scheduling of your batch jobs, and the resources required to run your jobs.
    One of AWS Batch’s great strengths is the ability to manage instance
    provisioning as your workload requirements and budget needs change. AWS
    Batch takes advantage of AWS’s broad base of compute types. For example,
    you can launch compute based instances and memory instances that can handle
    different workload types, without having to worry about building a cluster
    to meet peak demand.</span></p>
    <p style='margin:0in'><span style='color:black'>Previously, AWS Batch had a
    cost-controlling approach to manage compute instances for your workloads.
    The service chose an instance that was the best fit for your jobs based on
    vCPU, memory, and GPU requirements, at the lowest cost. Now, the newly
    added allocation strategies provide flexibility. They allow AWS Batch to
    consider capacity and throughput in addition to cost when provisioning your
    instances. This allows you to leverage different priorities when launching
    instances depending on your workloads’ needs, such as: controlling cost,
    maximizing throughput, or minimizing <a
    href="https://aws.amazon.com/ec2/spot/">Amazon EC2 Spot</a> instances
    interruption rates.</span></p>
    <p style='margin:0in'><span style='color:black'>There are now three
    instance allocation strategies from which to choose when creating an AWS
    Batch <a
    href="https://docs.aws.amazon.com/batch/latest/userguide/compute_environments.html">Compute
    Environment</a> (CE). They are:</span></p>
    <p style='margin:0in'><span style='color:black'>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Spot Capacity Optimized</span></p>
    <p style='margin:0in'><span style='color:black'>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit Progressive</span></p>
    <p style='margin:0in'><span style='color:black'>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BioContainers</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://biocontainers-edu.readthedocs.io/en/latest/"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://biocontainers-edu.readthedocs.io/en/latest/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Integration
    with BioConda</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://biocontainers-edu.readthedocs.io/en/latest/conda_integration.html"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://biocontainers-edu.readthedocs.io/en/latest/conda_integration.html</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    </span></b><b><span style='font-family:"Times New Roman",serif;color:black'>bioconda
    / packages <br>
    View all (8203) <br>
    </span></b><span style='color:black'><a
    href="https://anaconda.org/bioconda/"><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://anaconda.org/bioconda/repo"><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/repo</span></b></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><strong><span
    style='font-family:"Calibri",sans-serif;color:black'>Bioconda</span></strong></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://bioconda.github.io/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioconda.github.io/</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    </span><strong><span style='font-family:"Calibri",sans-serif;color:black'><br>
    </span></strong><strong><span style='font-family:"Calibri",sans-serif;
    color:black;font-weight:normal'>Bioconda</span></strong><span
    style='color:black'> is a channel for the <a
    href="https://conda.io/en/latest/index.html">conda</a> package manager
    specializing in bioinformatics software. Bioconda consists of:</span></p>
    <p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:
    .5in;text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
    style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    </span></span><span style='color:black'>a <a
    href="https://github.com/bioconda/bioconda-recipes">repository of recipes</a>
    hosted on GitHub</span></p>
    <p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:
    .5in;text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
    style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    </span></span><span style='color:black'>a <a
    href="https://github.com/bioconda/bioconda-utils">build system</a> turning
    these recipes into conda packages</span></p>
    <p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:
    .5in;text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
    style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    </span></span><span style='color:black'>a <a
    href="https://anaconda.org/bioconda/">repository of packages</a> containing
    over 7000 bioinformatics packages ready to use with </span><span class=pre><span
    style='font-size:10.0pt;font-family:"Courier New";color:black'>conda</span></span><code><span
    style='font-size:10.0pt;color:black'> </span></code><span class=pre><span
    style='font-size:10.0pt;font-family:"Courier New";color:black'>install</span></span></p>
    <p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:
    .5in;text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
    style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    </span></span><span style='color:black'>over 850 contributors and 570
    members who add, modify, update and maintain the recipes</span></p>
    <p style='margin:0in'><span style='color:black'>The conda package manager
    makes installing software a vastly more streamlined process. Conda is a
    combination of other package managers you may have encountered, such as
    pip, CPAN, CRAN, Bioconductor, apt-get, and homebrew. Conda is both
    language- and OS-agnostic, and can be used to install C/C++, Fortran, Go,
    R, Python, Java etc programs on Linux, Mac OSX, and Windows.</span></p>
    <p style='margin:0in'><span style='color:black'>Conda allows separation of
    packages into repositories, or </span><span class=pre><span
    style='font-size:10.0pt;font-family:"Courier New";color:black'>channels</span></span><span
    style='color:black'>. The main </span><span class=pre><span
    style='font-size:10.0pt;font-family:"Courier New";color:black'>defaults</span></span><span
    style='color:black'> channel has a large number of common packages. Users
    can add additional channels from which to install software packages not
    available in the defaults channel. Bioconda is one such channel
    specializing in bioinformatics software.</span></p>
    <p style='margin:0in'><strong><span style='color:black'>Browse packages in
    the Bioconda channel:</span></strong><span style='color:black'> <a
    href="https://bioconda.github.io/conda-package_index.html">Package Index</a></span></p>
    <p style='margin:0in'><span style='color:black'>Each package added to
    Bioconda also has a corresponding Docker <a href="https://biocontainers.pro">BioContainer</a>
    automatically created and uploaded to <a
    href="https://quay.io/organization/biocontainers">Quay.io</a>. A list of
    these and other containers can be found at the <a
    href="https://biocontainers.pro/#/registry">Biocontainers Registry</a>.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>bioconda
    / packages / samtools 1.11</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://anaconda.org/bioconda/samtools"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/samtools</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Tools
    for dealing with SAM, BAM and CRAM files</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>bioconda
    / packages / igv 2.8.13</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://anaconda.org/bioconda/igv"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/igv</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Integrative
    Genomics Viewer. Fast, efficient, scalable visualization tool for genomics
    data and annotations.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>bioconda
    / packages / bioconductor-deseq2 1.30.0</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://anaconda.org/bioconda/bioconductor-deseq2"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/bioconductor-deseq2</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Differential
    gene expression analysis based on the negative binomial distribution</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>bioconda
    / packages / bioconductor-pcaexplorer 2.16.0</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://anaconda.org/bioconda/bioconductor-pcaexplorer"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://anaconda.org/bioconda/bioconductor-pcaexplorer</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Interactive
    Visualization of RNA-seq Data Using a Principal Components Approach</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='font-size:10.0pt;line-height:107%;
font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:9.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0 align=left
   style='background:white;border-collapse:collapse;border:none;margin-left:
   4.95pt;margin-right:4.95pt;margin-top:4.95pt;margin-bottom:4.95pt'>
   <tr>
    <td width=730 valign=top style='width:7.6in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    3.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Data Science (Python):
    Project Repositories  </span></b></p>
    </td>
   </tr>
   <tr>
    <td width=730 valign=top style='width:7.6in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0 align=left
   style='background:white;border-collapse:collapse;border:none;margin-left:
   4.95pt;margin-right:4.95pt;margin-top:4.95pt;margin-bottom:4.95pt'>
   <tr>
    <td width=730 valign=top style='width:7.6in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Optimizing
    for cost, availability and throughput by selecting your AWS Batch
    allocation strategy</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>by
    Bala Thekkedath | on 24 OCT 2019 | in Advanced (300), Amazon EC2, AWS Batch</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://aws.amazon.com/blogs/compute/optimizing-for-cost-availability-and-throughput-by-selecting-your-aws-batch-allocation-strategy/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=730 valign=top style='width:7.6in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p><span style='color:black'>AWS offers a broad range of instances that are
    advantageous for batch workloads. The scale and provisioning speed of AWS’
    compute instances allow you to get up and running at peak capacity in
    minutes without paying for downtime. Today, I’m pleased to introduce
    allocation strategies: a significant new capability in <a
    href="https://docs.aws.amazon.com/batch/latest/userguide/allocation-strategies.html">AWS
    Batch</a> that &nbsp;makes provisioning compute resources flexible and
    simple. In this blog post, I explain how the AWS Batch allocation
    strategies work, when you should use them for your workload, and provide an
    example CloudFormation script. This blog helps you get started on building
    your personalized Compute Environment (CE) most appropriate to your
    workloads.</span></p>
    <p><strong><span style='color:black'>Overview</span></strong></p>
    <p><span style='color:black'>AWS Batch is a fully managed, cloud-native
    batch scheduler. It manages the queuing and scheduling of your batch jobs,
    and the resources required to run your jobs. One of AWS Batch’s great
    strengths is the ability to manage instance provisioning as your workload
    requirements and budget needs change. AWS Batch takes advantage of AWS’s
    broad base of compute types. For example, you can launch compute based
    instances and memory instances that can handle different workload types,
    without having to worry about building a cluster to meet peak demand.</span></p>
    <p><span style='color:black'>Previously, AWS Batch had a cost-controlling
    approach to manage compute instances for your workloads. The service chose
    an instance that was the best fit for your jobs based on vCPU, memory, and
    GPU requirements, at the lowest cost. Now, the newly added allocation
    strategies provide flexibility. They allow AWS Batch to consider capacity
    and throughput in addition to cost when provisioning your instances. This
    allows you to leverage different priorities when launching instances
    depending on your workloads’ needs, such as: controlling cost, maximizing
    throughput, or minimizing <a href="https://aws.amazon.com/ec2/spot/">Amazon
    EC2 Spot</a> instances interruption rates.</span></p>
    <p><span style='color:black'>There are now three instance allocation
    strategies from which to choose when creating an AWS Batch <a
    href="https://docs.aws.amazon.com/batch/latest/userguide/compute_environments.html">Compute
    Environment</a> (CE). They are:</span></p>
    <p><span style='color:black'>1.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Spot Capacity Optimized</span></p>
    <p><span style='color:black'>2.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit Progressive</span></p>
    <p><span style='color:black'>3.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
    &nbsp;Best Fit</span></p>
    <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=761 valign=top style='width:570.7pt;border:solid windowtext 1.0pt;
  padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>geoquery: a
  bridge between the gene expression omnibus (GEO) and bioconductor</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><em><span
  style='font-size:10.0pt'>Bioinformatics</span></em><span style='font-size:
  10.0pt;font-family:"Times New Roman",serif'>, <b>14</b>, 1846–1847.<br>
  <a
  href="https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html">https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html</a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><a
  href="https://www.bioconductor.org/packages/release/bioc/vignettes/GEOquery/inst/doc/GEOquery.html#platforms">https://www.bioconductor.org/packages/release/bioc/vignettes/GEOquery/inst/doc/GEOquery.html#platforms</a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><b>GOOD BIG PICTURE of Bioinformatics Software Tools</b></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=763 valign=top style='width:7.95in;border:solid windowtext 1.0pt;
  padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Collection of Bioinformatics
  Tools</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><a
  href="https://github.com/jdidion/biotools/blob/main/README.md">https://github.com/jdidion/biotools/blob/main/README.md</a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;background:yellow'>Excellent</span></b><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'> </span></b></p>
  </td>
 </tr>
 <tr>
  <td width=763 valign=top style='width:7.95in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Click [Raw] for
  actual Markdown text which renders as a good hyperlinked page</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Mastering RNA-Seq
(NGS Data Analysis) - A Critical Approach to Transcriptomic Data Analysis</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'><a
href="https://www.youtube.com/watch?v=j4qpJ8sVjT0">https://www.youtube.com/watch?v=j4qpJ8sVjT0</a></span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Video 03:18:55</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Apr 17, 2018</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>This RNA-seq
workshop will address critical issues related to Transcriptomics data, like:</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Processing raw
Next Generation Sequencing (NGS) data</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:white;border-collapse:collapse;border:none'>
 <tr>
  <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
  padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>Relevant <span style='color:black'>Journal
  Publications:</span></span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><br>
  </span><span style='color:black'><a
  href="https://journals.plos.org/ploscompbiol/search"><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://journals.plos.org/ploscompbiol/search</span></b></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>All
  Fields:                 bioconductor  </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Publication
  Date:     2016-01-01 - 2020-10-09   </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Sort
  By:                     Most Bookmarked</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Subject
  Area:           <span style='background:yellow'>Principal component analysis</span></span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>&gt;&gt;
  results</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Ten
  quick tips for effective dimensionality reduction</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>20
  Jun 2019 PLOS Computational Biology</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://doi.org/10.1371/journal.pcbi.1006907"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://doi.org/10.1371/journal.pcbi.1006907</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006907"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006907</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Citations:
  19 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Context
  Specific and Differential Gene Co-expression Networks via Bayesian
  Biclustering</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>28
  Jul 2016 PLOS Computational Biology</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://doi.org/10.1371/journal.pcbi.1004791"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://doi.org/10.1371/journal.pcbi.1004791</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Citations:
  22 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Machine
  learning-based microarray analyses indicate low-expression genes might
  collectively influence PAH disease</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>12
  Aug 2019 PLOS Computational Biology</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://doi.org/10.1371/journal.pcbi.1007264"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://doi.org/10.1371/journal.pcbi.1007264</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Citations:
  2 </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;line-height:normal'><span style='font-family:"Times New Roman",serif'>NGS
RNA-Seq bioinformatics tools, workflows </span></p>

<p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:0in;
margin-left:0in;line-height:normal'><span style='font-family:"Times New Roman",serif'>&quot;differential
expression&quot;<br>
&quot;exploratory data analysis&quot; </span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'><br>
</span><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>Attracted
to the aspects of workflows following the pre-processing, normalization
(Quantification of Mapped Reads, Differential Gene Expression).</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>The Application of
Inferential Statistics, Machine Learning (PCA, Feature/Dimensional Reduction,
Clustering).</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Google Search</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://www.google.com/searchlr=&amp;as_qdr=all&amp;sxsrf=ALeKk00QSjfjp_4g_LC4EyNExgXHpC1w4w:1606327451528&amp;q=enabling+cross+study+analysis+of+rna-sequencing+data&amp;sa=X&amp;ved=2ahUKEwi-jZCJpJ7tAhVkw1kKHXaKCWMQ1QIoAHoECAUQAQ"><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>enabling cross
study analysis of rna-sequencing data</span></b></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
name="_Hlk57370292"></a><a
href="https://www.google.com/searchlr=&amp;as_qdr=all&amp;sxsrf=ALeKk00QSjfjp_4g_LC4EyNExgXHpC1w4w:1606327451528&amp;q=enabling+cross+study+analysis+of+rna-sequencing+data&amp;sa=X&amp;ved=2ahUKEwi-jZCJpJ7tAhVkw1kKHXaKCWMQ1QIoAHoECAUQAQ"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/searchlr=&amp;as_qdr=all&amp;sxsrf=ALeKk00QSjfjp_4g_LC4EyNExgXHpC1w4w:1606327451528&amp;q=enabling+cross+study+analysis+of+rna-sequencing+data&amp;sa=X&amp;ved=2ahUKEwi-jZCJpJ7tAhVkw1kKHXaKCWMQ1QIoAHoECAUQAQ</span></a></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><img border=0
width=346 height=128 id="Picture 1" src="Project-Ideas_files/image005.jpg"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
</span><a
href="https://www.google.com/search?as_q=RNA-seq++data+Normalized+batch-corrected+&amp;as_epq=data+mining+&amp;as_oq=&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs="><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.google.com/search?as_q=RNA-seq++data+Normalized+batch-corrected+&amp;as_epq=data+mining+&amp;as_oq=&amp;as_eq=&amp;as_nlo=&amp;as_nhi=&amp;lr=&amp;cr=&amp;as_qdr=all&amp;as_sitesearch=&amp;as_occt=any&amp;safe=images&amp;as_filetype=&amp;tbs=</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>Keywords:</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>RNA-seq </span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>data mining </span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>cross study
analysis </span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>data normalization</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>batch-corrected RNA-seq
</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>RNA-seq workflow:
gene-level exploratory analysis and differential expression</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><span style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='color:black'><img
  border=0 width=283 height=114 id="Picture 7"
  src="Project-Ideas_files/image006.jpg"></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><img border=0 width=283 height=115 id="Picture 8"
  src="Project-Ideas_files/image007.jpg"></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><img border=0 width=282 height=111 id="Picture 9"
  src="Project-Ideas_files/image008.jpg"></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><br>
  </span><b><span style='font-size:12.0pt;font-family:"Times New Roman",serif;
  color:black'> Relevant journal publications </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA-seq
    workflow: gene-level exploratory analysis and differential expression</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>2015
    Oct 14</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Abstract</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Here
    we walk through an end-to-end gene-level RNA-Seq differential expression
    workflow using Bioconductor packages. We will start from the FASTQ files,
    show how these were aligned to the reference genome, and prepare a count
    matrix which tallies the number of RNA-seq reads/fragments within each gene
    for each sample. We will perform exploratory data analysis (EDA) for
    quality assessment and to explore the relationship between samples, perform
    differential gene expression analysis, and visually explore the results.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Keywords
    </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA-seq,
    differential expression, gene expression, Bioconductor, statistical
    analysis, high-throughput sequencing, visualization, genomics</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>RNA-seq: Basic
    Bioinformatics Analysis</span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>Curr Protoc Mol Biol. 2018
    Oct</span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6168365/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6168365/</a></span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>The workflow includes
    three parts: </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(a) mapping sequencing
    reads to a reference genome or transcriptome; </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(b) quantifying expression
    levels of individual genes and transcripts; </span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif;color:black'>(c) identifying specific
    genes and transcripts that are differentially expressed between samples.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Alignment/Mapping:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    the sequence of each read to a reference genome, annotation of genes </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>STAR</b> alignment tool </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Quality Assessment:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    After mapping reads to the genome, it is important to survey the quality of
    the RNA-seq data</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>Picard</b> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>SAMTools</b> </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Quantification of Mapped Reads:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    - <b>HTseq</b> to quantify sequencing reads mapped to each gene</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    (a) identify genes that are differentially expressed between conditions
    (sample groups), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    (b) derive gene expression values for each individual transcript</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    Approaches for normalization include </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    CPM (counts per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    RPKM (reads per kilobase per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    FPKM (fragments per kilobase per million reads), </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>       
    TPM (transcripts per million reads).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    Differential Gene Expression Analysis:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    <b>EdgeR</b> (Robinson et al., 2010) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>    
    <b>DESeq2</b> (Love et al. 2014)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    Download and install required tools:</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    STAR: </span><span style='color:black'><a
    href="https://github.com/alexdobin/STAR"><span style='font-size:10.0pt;
    font-family:"Times New Roman",serif'>https://github.com/alexdobin/STAR</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Picard: </span><span style='color:black'><a
    href="https://broadinstitute.github.io/picard/"><span style='font-size:
    10.0pt;font-family:"Times New Roman",serif'>https://broadinstitute.github.io/picard/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    HTseq: </span><span style='color:black'><a
    href="https://htseq.readthedocs.io/en/release_0.9.1/install.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://htseq.readthedocs.io/en/release_0.9.1/install.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    R: </span><span style='color:black'><a href="https://www.r-project.org"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.r-project.org</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>ngs.plot:
    Quick mining and visualization of next-generation sequencing data by
    integrating genomic databases</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Genomics 2014 volume 15, Article number: 284 </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>415
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-284"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-15-284</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer:
    an R/Bioconductor package for interacting with RNA-seq principal components</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Bioinformatics 2019 volume 20, Article number: 331  </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>17
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2879-1"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2879-1</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/html/pcaExplorer.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>pcaExplorer
    User Guide</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    October 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/pcaExplorer.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/pcaExplorer.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Up
    and running with pcaExplorer</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>27
    October 2020</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/upandrunning.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/bioc/vignettes/pcaExplorer/inst/doc/upandrunning.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>data
    component (count matrix, experimental data, dds object, annotation)</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    Beginner’s Guide to Analysis of RNA Sequencing Data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096346/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6096346/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Am
    J Respir Cell Mol Biol. 2018 Aug</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Abstract:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    general understanding of the principles underlying each step of RNA-seq
    data analysis allows investigators without a background in programming and
    bioinformatics to critically analyze their own datasets as well as
    published data.  Our goals in the present review are to break down the
    steps of a typical RNA-seq analysis and to highlight the pitfalls and
    checkpoints along the way that are vital for bench scientists and
    biomedical researchers performing experiments that use RNA-seq.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Keywords:
    </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNA
    sequencing, transcriptomics, bioinformatics, data analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Associated
    Data:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    RNA-seq data reported in this article has been deposited in NCBI’s Gene
    Expression Omnibus (GEO) and are accessible through GEO Series accession
    number GSE116583.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Experimental
    Design and Approach:</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>A
    major goal of RNA-seq analysis is to identify differentially expressed and
    coregulated genes and to infer biological meaning for further studies.
    Source material can be cells cultured in vitro, whole-tissue homogenates,
    or sorted cells. The ability to interpret findings depends on appropriate
    experimental design, implementation of controls, and correct analysis.
    Every effort should be made to minimize batch effect, because small and
    uncontrolled changes in an environment can result in identification of
    differentially expressed genes (DEGs) unrelated to the designed experiment.
    Sources of batch effect can occur during the experiment, during the RNA
    library preparation, or during the sequencing run and include but are not
    limited to those listed in Table 1. Once a well-designed and controlled
    experiment is performed, a structured approach to the dataset allows for
    quality control followed by unbiased analysis of the data. In the present
    analysis, we use an approach that includes setting low count filtering,
    establishing a noise threshold, checking for potential outliers, running
    appropriate statistical tests to identify DEGs, clustering of genes by
    expression pattern, and testing for gene ontology (GO) enrichment. For each
    of these analysis components, we aim to highlight important checkpoints and
    quality controls that will streamline and strengthen data analysis, avoid
    bias, and allow investigators to maximally use their datasets.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    two most common clustering methods used for RNA-seq data analysis are
    hierarchical and k-means clustering (see Clustering box). The most common
    form of hierarchical clustering is a bottom-up agglomerative approach that
    organizes the data into a tree structure without user input by starting
    with each data point as its own cluster and iteratively combining them into
    larger clusters or “clades.” In contrast, k-means clustering requires the
    investigator to define the number of clusters (k) a priori, and data are
    then sorted into the cluster with the nearest mean. It is possible to
    assess a range of k-values to decide how to best capture the trends. In
    addition, various tools such as Silhouette exist to help the investigator
    determine the ideal k-value, but some subjectivity remains (21). By
    adjusting the k, the investigator may set the degree of granularity they
    would like to achieve with the data. For either approach, the user must
    specify the distance metric by which data points are considered similar.
    Typically, Pearson’s correlation is used, and this is generally the default
    in software designed for RNA-seq analysis. Both approaches are widely used,
    and both aid the investigator in identifying groups of genes that display
    similar expression patterns, allowing for further downstream analyses. The
    clusters can then be used as input for an analysis of functional enrichment
    (see next section).</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Why
    do we use clustering on RNA-seq data?</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Clustering
    of RNA-seq data may be used to identify patterns of gene expression by
    grouping genes based on their distance in an unsupervised manner.
    Clustering RNA-seq data is used as an exploratory tool that allows the user
    to organize and visualize relationships between groups of genes, and to
    select certain genes for further consideration.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Hierarchical
    clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>The
    most commonly used hierarchical clustering approach is a form of agglomerative,
    or bottom-up, clustering that iteratively merges clusters (originally
    consisting of individual data points) into larger clusters or “clades”.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>K-means
    clustering</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Data
    points are iteratively partitioned into clusters based on the minimum distance
    to the cluster mean. The number of clusters (k) is set by the investigator.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Review
    of RNA-Seq Data Analysis Tools</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>February
    17, 2016</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://rna-seqblog.com/review-of-rna-seq-data-analysis-tools/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://rna-seqblog.com/review-of-rna-seq-data-analysis-tools/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Feature-based
    classification of human transcription factors into hypothetical sub-classes
    related to regulatory function</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BMC
    Bioinformatics volume 17, Article number: 459 (2016) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>7
    Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1349-2"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1349-2</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>GEO2RNAseq:
    An easy-to-use R pipeline for complete pre-processing of RNA-seq data</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>September
    16, 2019 bioRxiv Preprint</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.biorxiv.org/content/10.1101/771063v1.full"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/771063v1.full</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>RNAseq
    data analysis in R - Notebook</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="http://monashbioinformaticsplatform.github.io/RNAseq-DE-analysis-with-R/RNAseq_DE_analysis_with_R.html"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://monashbioinformaticsplatform.github.io/RNAseq-DE-analysis-with-R/RNAseq_DE_analysis_with_R.html</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Install and load packages</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Mapping reads to a reference genome</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Count reads for each feature</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    QC and stats</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Differential Expression</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Gene Annotation</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>   
    Gene Set Enrichment</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Introduction
    to differential gene expression analysis using RNA-seq</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>September
    2015 updated November 14, 2019</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:#FFFAEB;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
  3.0pt;margin-left:0in;line-height:normal'><span style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='color:black'>Google</span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='color:black'>enabling
    cross-study analysis of RNA-Sequencing data</span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><span style='color:black'><a
    href="https://www.google.com/search?lr=&amp;as_qdr=all&amp;sxsrf=ALeKk00QSjfjp_4g_LC4EyNExgXHpC1w4w:1606327451528&amp;q=enabling+cross+study+analysis+of+rna-sequencing+data&amp;sa=X&amp;ved=2ahUKEwi-jZCJpJ7tAhVkw1kKHXaKCWMQ1QIoAHoECAUQAQ">https://www.google.com/search?lr=&amp;as_qdr=all&amp;sxsrf=AleKk00QSjfjp_4g_LC4EyNExgXHpC1w4w:1606327451528&amp;q=enabling+cross+study+analysis+of+rna-sequencing+data&amp;sa=X&amp;ved=2ahUKEwi-jZCJpJ7tAhVkw1kKHXaKCWMQ1QIoAHoECAUQAQ</a></span></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='color:black'>Google</span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'><b><span style='color:black'>RNA-Seq
    data mining </span></b></p>
    <p class=MsoNormal style='margin-top:3.0pt;margin-right:0in;margin-bottom:
    0in;margin-left:0in;line-height:normal'>&nbsp;</p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>A survey of best practices for RNA-seq data analysis</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Genome Biology volume 17, Article number: 13 (2016) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>769 Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8">https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Massive mining of publicly available RNA-seq data from
    human and mouse</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Nature Communications volume 9, Article number: 1366
    (2018) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>96 Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nature.com/articles/s41467-018-03751-6">https://www.nature.com/articles/s41467-018-03751-6</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Omics Playground: Explore Omics Data Freely</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://omicsplayground.readthedocs.io/en/latest/index.html">https://omicsplayground.readthedocs.io/en/latest/index.html</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Omics Playground is a comprehensive self-service
    platform platform for visualization, analytics and exploration of Big Omics
    Data.  It allows users to apply a multitude of state-of-the-art analysis
    tools to their own data to explore and discover underlying biology in a
    short time.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><br>
    The platform offers a unique combination of features that distinguishes it
    from the other analytics platforms currently available.  We believe that
    data preprocessing (primary analysis) and statistical testing (secondary
    analysis) are now well established, and the most challenging task is
    currently data interpretation (tertiary analysis) that often takes the
    longest time but where actual insights can be gained. Therefore, Omics
    Playground focuses strongly on tertiary analysis while providing good
    support for secondary analysis.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Reanalyzing Public Datasets</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>To illustrate the use case of the Omics Playground, we
    reanalyzed different types of publics datasets, including microarray, bulk
    RNA-seq, single-cell RNA-seq and proteomic datasets to recapitulate the
    results.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://omicsplayground.readthedocs.io/en/latest/examples/examples.html">https://omicsplayground.readthedocs.io/en/latest/examples/examples.html</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>comparing rna-seq datasets</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    </td>
   </tr>
   <tr>
    <td width=758 valign=top style='width:7.9in;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Broad Institute Gene Set Enrichment Analysis (GSEA) </span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Using_RNA-seq_Datasets_with_GSEA">https://software.broadinstitute.org/cancer/software/gsea/wiki/index.php/Using_RNA-seq_Datasets_with_GSEA</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>BioStars</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Question: RNA-seq data comparison across experiments</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.biostars.org/p/117451/">https://www.biostars.org/p/117451/</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>BioStars</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Question: Comparing similarity of RNAseq datasets</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.biostars.org/p/334235/">https://www.biostars.org/p/334235/</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='color:black'>Analysis of public RNA-sequencing data reveals
    biological consequences of genetic heterogeneity in cell line populations</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>Scientific Reports volume 8, Article number: 11226
    (2018) Cite this article</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'>7 Citations</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://www.nature.com/articles/s41598-018-29506-3">https://www.nature.com/articles/s41598-018-29506-3</a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-family:"Times New Roman",serif;color:black'>Books:</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>NCBI
  Books Advanced Search</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/advanced/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/advanced/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
   style='background:white;border-collapse:collapse;border:none'>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Computational
    Biology</span></b></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Brisbane
    (AU): Codon Publications; 2019 Nov 21.</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>ISBN-13:
    978-0-9944381-9-5</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/NBK550339/"><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/NBK550339/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'>Creative
    Commons Attribution-NonCommercial 4.0 International (CC BY-NC 4.0) </span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a
    href="https://creativecommons.org/licenses/by-nc/4.0/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://creativecommons.org/licenses/by-nc/4.0/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    </td>
   </tr>
   <tr>
    <td width=753 valign=top style='width:564.4pt;border:solid windowtext 1.0pt;
    border-top:none;padding:0in 5.4pt 0in 5.4pt'>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif;color:black'>Chapter
    4 Biological Sequence Analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/NBK550342/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/NBK550342/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif;color:black'>Chapter
    5 Multivariate Statistical Methods for High-Dimensional Multiset Omics Data
    Analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/NBK550343/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/NBK550343/</span></a></span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='font-size:9.0pt;font-family:"Times New Roman",serif;color:black'>Chapter
    6 Statistical Methods for RNA Sequencing Data Analysis</span></p>
    <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
    style='color:black'><a href="https://www.ncbi.nlm.nih.gov/books/NBK550334/"><span
    style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/books/NBK550334/</span></a></span><span
    style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
    <br>
    </span></p>
    </td>
   </tr>
  </table>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>RNA-seq workflow:
gene-level exploratory analysis and differential expression</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='background:white;border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>NCBI Reference
  Sequence Databas<span style='color:black'>e      (RefSeq) </span></span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.ncbi.nlm.nih.gov/refseq/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/refseq/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Homo_sapiens/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://ftp.ncbi.nlm.nih.gov/genomes/refseq/vertebrate_mammalian/Homo_sapiens/</span></a></span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:6.0pt;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Most Common
  Tools Used for the Analysis of WGS Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h1 style='margin-top:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:windowtext'>Comprehensive Outline
  of Whole Exome Sequencing Data Analysis Tools</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:windowtext'>
  <b>Available in Clinical Oncology</b></span></h1>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Comparative
  analysis of differential gene expression analysis tools for single-cell RNA
  sequencing data   </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>2019</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Most Common
  Tools Used for the Analysis of WGS Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h1 style='margin-top:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:windowtext'>Comprehensive Outline
  of Whole Exome Sequencing Data Analysis Tools Available in Clinical Oncology</span></b></h1>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Comparitive
  Analysis of Differential Gene Expression Analysis Tools for Single-Cell
  Sequencing Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>FASTQC   A
  quality control tool for high throughput sequence data</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>.    2014
  September 29 <br>
  </span><a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Free RNA-seq
  Analysis Tools – Software and Resources</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bioinformaticshome.com/tools/rna-seq/rna-seq.html"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/rna-seq/rna-seq.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif;color:white'><br>
<br>
</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Most Common
  Tools Used for the Analysis of WGS Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h1 style='margin-top:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:windowtext'>Comprehensive Outline
  of Whole Exome Sequencing Data Analysis Tools</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:windowtext'>
  <b>Available in Clinical Oncology</b></span></h1>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Comparative
  analysis of differential gene expression analysis tools for single-cell RNA
  sequencing data   </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>2019</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Most Common
  Tools Used for the Analysis of WGS Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.researchgate.net/figure/NGS-and-analysis-pipelines-Most-common-tools-used-for-the-analysis-of-WGS-data-QC_fig2_317413533</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <h1 style='margin-top:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif;color:windowtext'>Comprehensive Outline
  of Whole Exome Sequencing Data Analysis Tools Available in Clinical Oncology</span></b></h1>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6895801/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Comparitive
  Analysis of Differential Gene Expression Analysis Tools for Single-Cell
  Sequencing Data</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-2599-6</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>FASTQC   A
  quality control tool for high throughput sequence data</span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>.    2014
  September 29 <br>
  </span><a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Free RNA-seq
  Analysis Tools – Software and Resources</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bioinformaticshome.com/tools/rna-seq/rna-seq.html"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://bioinformaticshome.com/tools/rna-seq/rna-seq.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'>&nbsp;</p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-family:"Times New Roman",serif'>Analysis
  of single cell RNA-seq data (</span></b><span style='font-family:"Times New Roman",serif;
  color:black'>University of Cambridge Bioinformatics)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://biocellgen-public.svi.edu.au/mig_2019_scrnaseq-workshop/public/index.html"><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>https://biocellgen-public.svi.edu.au/mig_2019_scrnaseq-workshop/public/index.html</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif;color:black'>quality
  control, visualisation, data normalisation, exploratory data analysis,
  clustering, trajectory (pseudotime) inference, differential expression, batch
  correction, combining datasets, data integration, confounders, latent spaces,
  cell annotation, case studies</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>2-day Course: (16 hours of video) </span></b><span
  class=MsoHyperlink><b><span style='font-size:9.0pt;font-family:"Times New Roman",serif;
  color:windowtext;text-decoration:none'>RStudio</span></b></span><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  <br>
  Day 1:</span><span style='font-size:8.0pt;font-family:"Times New Roman",serif'>
  </span><a
  href="https://www.youtube.com/watch?v=thHgPqQpkE4&amp;feature=emb_err_woyt"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=thHgPqQpkE4&amp;feature=emb_err_woyt</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  class=MsoHyperlink><span style='font-size:9.0pt;font-family:"Times New Roman",serif;
  color:windowtext;text-decoration:none'>Processing Raw scRNA-Seq Data</span></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  class=MsoHyperlink><span style='font-size:9.0pt;font-family:"Times New Roman",serif;
  color:windowtext;text-decoration:none'>Construction of Expression Matrix</span></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  class=MsoHyperlink><span style='font-size:9.0pt;font-family:"Times New Roman",serif;
  color:windowtext;text-decoration:none'>Intro to R/Bioconductor</span></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  class=MsoHyperlink><span style='font-size:9.0pt;font-family:"Times New Roman",serif;
  color:windowtext;text-decoration:none'>Seurat</span></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  Day 2: </span><a
  href="https://www.youtube.com/watch?v=7dQ_pleDO2Y&amp;feature=emb_err_woyt"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=7dQ_pleDO2Y&amp;feature=emb_err_woyt</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Clustering
  example</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Feature Selection</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Pseudotime
  Analysis</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Differental
  Expression Analysis </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>DE Real Dataset</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Comparing/Combining
  scRNA</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Search scRNA-Seq
  Data</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>Seurat</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:9.0pt;font-family:"Times New Roman",serif'>scRNA-Seq
  Pipeline</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal><span style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>Principal Components Analysis / Feature
  Selection</span></b></p>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-family:"Times New Roman",serif;
  color:black'>bioRxiv</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.biorxiv.org/"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="https://www.biorxiv.org/search"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/search</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>mixOmics: an R
  package for omics feature selection and multiple data integration Aug 2017</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.biorxiv.org/content/10.1101/108597v4.full"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/108597v4.full</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Differential
  Principal Components Reveal Patterns of Differentiation in Case/Control
  Studies Feb 2019</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.biorxiv.org/content/10.1101/545798v1.full"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/545798v1.full</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>pathwayPCA: an R
  package for integrative pathway analysis with modern PCA methodology and gene
  selection April 2019</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.biorxiv.org/content/10.1101/615435v1.full"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/615435v1.full</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Accurate and
  Fast feature selection workflow for high-dimensional omics data   June 2017</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.biorxiv.org/content/10.1101/144162v1.full"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.biorxiv.org/content/10.1101/144162v1.full</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
name="_Hlk54534562"><span style='font-family:"Times New Roman",serif'>&nbsp;</span></a></p>

<p class=MsoNormal><a name="_Hlk54540545"><span style='font-size:10.0pt;
line-height:107%;font-family:"Times New Roman",serif'>&nbsp;</span></a></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>SEURAT: R toolkit for single cell
  genomics</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://satijalab.org/seurat/v3.1/pbmc3k_tutorial.html"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://satijalab.org/seurat/v3.1/pbmc3k_tutorial.html</span></a><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif;color:black'><br>
  <br>
  </span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>Installation Instructions for Seurat</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://satijalab.org/seurat/install.html"><span style='font-size:8.0pt;
  font-family:"Times New Roman",serif'>https://satijalab.org/seurat/install.html</span></a></p>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>Vignettes: Guided Analyses </span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://satijalab.org/seurat/vignettes.html"><span style='font-size:
  8.0pt;font-family:"Times New Roman",serif'>https://satijalab.org/seurat/vignettes.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Seurat - Guided
  Clustering Tutorial </span></b><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>2020 April</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Setup the Seurat
  Object</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Standard
  pre-processing workflow</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Normalizing the
  data</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Identification
  of highly variable features (feature selection)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Scaling the data</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Perform linear
  dimensional reduction</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Determine the
  ‘dimensionality’ of the dataset</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Cluster the
  cells</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Run non-linear
  dimensional reduction (UMAP/tSNE)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Finding
  differentially expressed features (cluster biomarkers)</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Assigning cell
  type identity to clusters</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:10.0pt;line-height:107%;font-family:
"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>Statistics for Genomics</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a
  href="https://www.youtube.com/playlist?list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/playlist?list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:6.0pt;margin-right:0in;margin-bottom:
  6.0pt;margin-left:0in;line-height:normal'><b><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>17 Videos: </span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2017 May</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomcs: Distances and Clustering</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=wQhVWUcXM0A&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=2"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=wQhVWUcXM0A&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=2</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomics Lab: Quick Introduction to R and Bioconductor</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=J5h5WxOn3Gw&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=11"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=J5h5WxOn3Gw&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=11</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomics Lab: Distances and Clustering RStudio</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=PArRvqLUP6o&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=7"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=PArRvqLUP6o&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=7</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomics: Introduction to RNAseq</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=C8RNvWu7pAw&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=12"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=C8RNvWu7pAw&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=12</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomics: Advanced Differential Expression</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=QINX3cI7qgk&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=15"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=QINX3cI7qgk&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=15</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Statistics for
  Genomics: Useful plots and bad plots</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://www.youtube.com/watch?v=46-t2jOYsyY&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=17"><span
  style='font-size:8.0pt;font-family:"Times New Roman",serif'>https://www.youtube.com/watch?v=46-t2jOYsyY&amp;list=PLdl4u5ZRDMQQpUcSDRKN3V2vvx3_SmMbr&amp;index=17</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>BioC
  2020 Conference</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='color:black'><a href="http://bioc2020.bioconductor.org/schedule"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioc2020.bioconductor.org/schedule</span></a></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>100: <b>Annotating inter-sample DNA
  methylation and ATAC-seq variation with COCOA</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>100: <b>Human RNA-seq data from recount2
  and related packages</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>100: <b>Introduction to Bioconductor
  annotation resources</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>100: <b>A tidy transcriptomics
  introduction to RNA-Seq analyses</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>Functional enrichment analysis
  of high-throughput omics data</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>Best practices for ATAC-seq QC
  and data analysis</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>Copy number variation analysis
  with Bioconductor</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>Interactive visualization of
  SummarizedExperiment objects with iSEE</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>Integrated ChIP-seq data
  analysis workshop</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>200: <b>An introduction to matrix
  factorization and principal component analysis in R</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>500: <b>Bioconductor toolchain for usage
  and development of reproducible bioinformatics pipelines in CWL</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>500: <b>Effectively Using the
  DelayedArray Framework to Support the Analysis of Large Datasets</b></span></p>
  <p class=MsoNormal style='margin-top:4.0pt;margin-right:0in;margin-bottom:
  0in;margin-left:0in;line-height:normal'><span style='font-size:10.0pt;
  font-family:"Times New Roman",serif'>100: <b>Cloud-based genomics using
  Terra/AnVIL</b></span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<table class=MsoTableGrid border=1 cellspacing=0 cellpadding=0
 style='border-collapse:collapse;border:none'>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  background:#D9E2F3;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif;color:black'>Bioconductor:
  DESeq2<br>
  </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif;
  color:black'>Differential gene expression analysis based on the negative
  binomial distribution</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></b></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Analyzing
  RNA-seq data with DESeq2 </span></b><span style='font-size:10.0pt;font-family:
  "Times New Roman",serif'>2020 Oct</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'><br>
  RNA-seq workflow: gene-level exploratory analysis and differential expression
  </span></b><span style='font-size:10.0pt;font-family:"Times New Roman",serif'>2019
  Oct</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://master.bioconductor.org/packages/release/workflows/vignettes/rnaseqGene/inst/doc/rnaseqGene.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Moderated
  estimation of fold change and dispersion for RNA-seq data with DESeq2 </span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2014 Dec</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0550-8</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>The R package
  Rsubread is easier, faster, cheaper and better for alignment and
  quantification of RNA sequencing reads </span></b></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="https://academic.oup.com/nar/article/47/8/e47/5345150"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://academic.oup.com/nar/article/47/8/e47/5345150</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>Data
  preprocessing and creation of the data objects pasillaGenes and pasillaExons </span></b><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>2020 May</span></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
  href="http://bioconductor.org/packages/release/data/experiment/vignettes/pasilla/inst/doc/create_objects.html"><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>http://bioconductor.org/packages/release/data/experiment/vignettes/pasilla/inst/doc/create_objects.html</span></a></p>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
 <tr>
  <td width=768 valign=top style='width:8.0in;border:solid windowtext 1.0pt;
  border-top:none;padding:0in 5.4pt 0in 5.4pt'>
  <p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
  style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>
  </td>
 </tr>
</table>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<h3 style='margin:0in'><a name="_Hlk57394992"><span style='font-size:10.0pt'>Reproducible
and robust workshop materials<br>
<br>
</span></a></h3>

<p class=p style='margin:0in'><span style='font-size:10.0pt'>Workshops were
authored using R Markdown, and compiled into a book (PDF and ePub) and website
using Bookdown R package. Bookdown, in turn, uses the gitbook publishing system
( </span><a href="https://www.gitbook.com/" target="_blank"><span
style='font-size:10.0pt'>https://www.gitbook.com/</span></a><span
style='font-size:10.0pt'>) to produce a variety of formats from the same source
material. R Markdown files intended to be part of a Bookdown project do not
contain the required front matter of a typical stand-alone R Markdown document.
To help authors use and test the correct format, we seeded each workshop
document with the syllabus that had been submitted by that author, and
successfully built the book of the submitted syllabi. Each workshop represented
a chapter of a book compiled using the Bookdown software. This approach provided
several advantages:<br>
<br>
</span></p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:10.0pt'>R markdown syntax is already
familiar to any developer of a Bioconductor package, since it is the standard
approach to creating the package “vignette” or prose documentation.</span></p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:10.0pt'>R markdown implements “literate
programming” by including formatted text, runnable code, and output of the code</span></p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:10.0pt'>Bookdown allows collating chapters
as a clean, lightweight online book format, and pandoc additionally allows
creation of PDF and ePub formats</span></p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'><span style='font-size:10.0pt;font-family:Symbol'>·<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='font-size:10.0pt'>These formats can then be
self-published with options to order paper copies through companies such as </span><a
href="https://leanpub.com" target="_blank"><span style='font-size:10.0pt'>https://leanpub.com</span></a><span
style='font-size:10.0pt'> <br>
<br>
</span></p>

<p class=p style='margin:0in'><span style='font-size:10.0pt'>This approach
allowed automatic installation of required packages by listing them in the
DESCRIPTION file required by R packages.</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><b><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>bookdown: Authoring
Books and Technical Documents with R Markdown</span></b></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><a
href="https://books.google.com/books?hl=en&amp;lr=&amp;id=_LrZDQAAQBAJ&amp;oi=fnd&amp;pg=PT12&amp;dq=+Xie+Y+:+bookdown:+Authoring+Books+and+Technical+Documents+with+R+Markdown+%5BInternet%5D+.+Boca+Raton,+Florida:+Chapman+and+Hall/CRC%3B+2016++10.1201/9781315204963++&amp;ots=tz1AhTAN96&amp;sig=mmTuwDYYrl3sVm8SjtJD8tvnwd0#v=onepage&amp;q&amp;f=false"><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>https://books.google.com/books?hl=en&amp;lr=&amp;id=_LrZDQAAQBAJ&amp;oi=fnd&amp;pg=PT12&amp;dq=+Xie+Y+:+bookdown:+Authoring+Books+and+Technical+Documents+with+R+Markdown+%5BInternet%5D+.+Boca+Raton,+Florida:+Chapman+and+Hall/CRC%3B+2016++10.1201/9781315204963++&amp;ots=tz1AhTAN96&amp;sig=mmTuwDYYrl3sVm8SjtJD8tvnwd0#v=onepage&amp;q&amp;f=false</span></a></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<p class=MsoNormal style='margin-bottom:0in;line-height:normal'><span
style='font-size:10.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

</div>

</body>

</html>
